// @tanstack/solid-start/dist/esm/plugin/vite.js
import { solidStartDefaultEntryPaths } from "./shared.js";
import { START_ENVIRONMENT_NAMES, tanStackStartVite } from "@tanstack/start-plugin-core/vite";
//#region src/plugin/vite.ts
function tanstackStart(options) {
	return [{
		name: "tanstack-solid-start:config",
		config() {
			return { ssr: { noExternal: [
				"@tanstack/solid-router-ssr-query",
				"@tanstack/solid-query",
				"@tanstack/solid-query-devtools"
			] } };
		},
		configEnvironment(environmentName, options) {
			return { optimizeDeps: environmentName === START_ENVIRONMENT_NAMES.client || environmentName === START_ENVIRONMENT_NAMES.server && options.optimizeDeps?.noDiscovery === false ? { exclude: [
				"@tanstack/solid-start",
				"@tanstack/solid-router",
				"@tanstack/start-static-server-functions"
			] } : void 0 };
		}
	}, tanStackStartVite({
		framework: "solid",
		defaultEntryPaths: solidStartDefaultEntryPaths,
		providerEnvironmentName: START_ENVIRONMENT_NAMES.server,
		ssrIsProvider: true,
		ssrResolverStrategy: { type: "default" }
	}, options)];
}
//#endregion
export { tanstackStart };
