// @tanstack/solid-router/dist/esm/awaited.js
import { TSR_DEFERRED_PROMISE, defer } from "@tanstack/router-core";
import { createComponent } from "solid-js/web";
import * as Solid from "solid-js";
//#region src/awaited.tsx
function useAwaited({ promise: _promise }) {
	const promise = defer(_promise);
	if (promise[TSR_DEFERRED_PROMISE].status === "pending") throw promise;
	if (promise[TSR_DEFERRED_PROMISE].status === "error") throw promise[TSR_DEFERRED_PROMISE].error;
	return [promise[TSR_DEFERRED_PROMISE].data, promise];
}
function Await(props) {
	if (!("fallback" in props)) {
		const [resource] = Solid.createResource(() => defer(props.promise), (p) => p, { deferStream: true });
		return createComponent(Solid.Show, {
			get when() {
				return resource();
			},
			children: (data) => props.children(data())
		});
	}
	return createComponent(Solid.Suspense, {
		get fallback() {
			return props.fallback;
		},
		get children() {
			return createComponent(AwaitInner, props);
		}
	});
}
function AwaitInner(props) {
	const [resource] = Solid.createResource(() => defer(props.promise), (p) => p);
	return props.children(resource());
}
//#endregion
export { Await, useAwaited };
