// @tanstack/solid-router/dist/esm/routerStores.js
import { createNonReactiveMutableStore, createNonReactiveReadonlyStore } from "@tanstack/router-core";
import * as Solid from "solid-js";
import { isServer } from "@tanstack/router-core/isServer";
//#region src/routerStores.ts
function initRouterStores(stores, createReadonlyStore) {
	stores.childMatchIdByRouteId = createReadonlyStore(() => {
		const ids = stores.matchesId.get();
		const obj = {};
		for (let i = 0; i < ids.length - 1; i++) {
			const parentStore = stores.matchStores.get(ids[i]);
			if (parentStore?.routeId) obj[parentStore.routeId] = ids[i + 1];
		}
		return obj;
	});
	stores.pendingRouteIds = createReadonlyStore(() => {
		const ids = stores.pendingIds.get();
		const obj = {};
		for (const id of ids) {
			const store = stores.pendingMatchStores.get(id);
			if (store?.routeId) obj[store.routeId] = true;
		}
		return obj;
	});
}
function createSolidMutableStore(initialValue) {
	const [signal, setSignal] = Solid.createSignal(initialValue);
	return {
		get: signal,
		set: setSignal
	};
}
var finalizationRegistry = null;
if (typeof globalThis !== "undefined" && "FinalizationRegistry" in globalThis) finalizationRegistry = new FinalizationRegistry((cb) => cb());
function createSolidReadonlyStore(read) {
	let dispose;
	const store = { get: Solid.createRoot((d) => {
		dispose = d;
		return Solid.createMemo(read);
	}) };
	finalizationRegistry?.register(store, dispose);
	return store;
}
var getStoreFactory = (opts) => {
	if (isServer ?? opts.isServer) return {
		createMutableStore: createNonReactiveMutableStore,
		createReadonlyStore: createNonReactiveReadonlyStore,
		batch: (fn) => fn(),
		init: (stores) => initRouterStores(stores, createNonReactiveReadonlyStore)
	};
	return {
		createMutableStore: createSolidMutableStore,
		createReadonlyStore: createSolidReadonlyStore,
		batch: Solid.batch,
		init: (stores) => initRouterStores(stores, createSolidReadonlyStore)
	};
};
//#endregion
export { getStoreFactory };
