// @tanstack/solid-router-devtools/dist/cjs/TanStackRouterDevtools.cjs
let solid_js_web = require("solid-js/web");
let _tanstack_solid_router = require("@tanstack/solid-router");
let _tanstack_router_devtools_core = require("@tanstack/router-devtools-core");
let solid_js = require("solid-js");
//#region src/TanStackRouterDevtools.tsx
var _tmpl$ = /* @__PURE__ */ (0, solid_js_web.template)(`<div>`);
var TanStackRouterDevtools = (props) => {
	const activeRouter = props.router ?? (0, _tanstack_solid_router.useRouter)();
	const activeRouterState = (0, _tanstack_solid_router.useRouterState)({ router: activeRouter });
	const usedProps = {
		...props,
		router: activeRouter,
		routerState: activeRouterState
	};
	let devToolRef;
	const [devtools] = (0, solid_js.createSignal)(new _tanstack_router_devtools_core.TanStackRouterDevtoolsCore(usedProps));
	(0, solid_js.createEffect)(() => {
		devtools().setRouter(usedProps.router);
	});
	(0, solid_js.createEffect)(() => {
		devtools().setRouterState(usedProps.routerState);
	});
	(0, solid_js.createEffect)(() => {
		devtools().setOptions({
			initialIsOpen: usedProps.initialIsOpen,
			panelProps: usedProps.panelProps,
			closeButtonProps: usedProps.closeButtonProps,
			toggleButtonProps: usedProps.toggleButtonProps,
			position: usedProps.position,
			containerElement: usedProps.containerElement,
			shadowDOMTarget: usedProps.shadowDOMTarget
		});
	});
	(0, solid_js.onMount)(() => {
		if (devToolRef) {
			devtools().mount(devToolRef);
			(0, solid_js.onCleanup)(() => {
				devtools().unmount();
			});
		}
	});
	return (() => {
		var _el$ = _tmpl$();
		var _ref$ = devToolRef;
		typeof _ref$ === "function" ? (0, solid_js_web.use)(_ref$, _el$) : devToolRef = _el$;
		return _el$;
	})();
};
//#endregion
exports.TanStackRouterDevtools = TanStackRouterDevtools;
