// @tanstack/solid-router/dist/source/index.jsx
export { defer, isMatch, joinPaths, cleanPath, trimPathLeft, trimPathRight, trimPath, resolvePath, interpolatePath, rootRouteId, defaultParseSearch, defaultStringifySearch, parseSearchWith, stringifySearchWith, functionalUpdate, replaceEqualDeep, isPlainObject, isPlainArray, deepEqual, createControlledPromise, retainSearchParams, stripSearchParams, createSerializationAdapter, } from '@tanstack/router-core';
export { createHistory, createBrowserHistory, createHashHistory, createMemoryHistory, } from '@tanstack/history';
export { useAwaited, Await } from './awaited';
export { CatchBoundary, ErrorComponent } from './CatchBoundary';
export { ClientOnly, useHydrated } from './ClientOnly';
export { FileRoute, createFileRoute, FileRouteLoader, LazyRoute, createLazyRoute, createLazyFileRoute, } from './fileRoute';
export * from './history';
export { lazyRouteComponent } from './lazyRouteComponent';
export { useLinkProps, createLink, Link, linkOptions } from './link';
export { Matches, useMatchRoute, MatchRoute, useMatches, useParentMatches, useChildMatches, } from './Matches';
export { Match, Outlet } from './Match';
export { useMatch } from './useMatch';
export { useLoaderDeps } from './useLoaderDeps';
export { useLoaderData } from './useLoaderData';
export { redirect, isRedirect, DEFAULT_PROTOCOL_ALLOWLIST, } from '@tanstack/router-core';
export { RouteApi, getRouteApi, Route, createRoute, RootRoute, rootRouteWithContext, createRootRoute, createRootRouteWithContext, createRouteMask, NotFoundRoute, } from './route';
export { createRouter, Router } from './router';
export { lazyFn, SearchParamError } from '@tanstack/router-core';
export { RouterProvider, RouterContextProvider } from './RouterProvider';
export { useElementScrollRestoration, ScrollRestoration, } from './ScrollRestoration';
export { useBlocker, Block } from './useBlocker';
export { useNavigate, Navigate } from './useNavigate';
export { useParams } from './useParams';
export { useSearch } from './useSearch';
export { useRouteContext } from './useRouteContext';
export { useRouter } from './useRouter';
export { useRouterState } from './useRouterState';
export { useLocation } from './useLocation';
export { useCanGoBack } from './useCanGoBack';
export { CatchNotFound, DefaultGlobalNotFound } from './not-found';
export { notFound, isNotFound } from '@tanstack/router-core';
export { ScriptOnce } from './ScriptOnce';
export { Asset } from './Asset';
export { HeadContent } from './HeadContent';
export { useTags } from './headContentUtils';
export { Scripts } from './Scripts';
export { composeRewrites } from '@tanstack/router-core';
