// lucide-solid/dist/source/aliases/aliases.jsx
/**
* @license lucide-solid v1.11.0 - ISC
*
* This source code is licensed under the ISC license.
* See the LICENSE file in the root directory of this source tree.
*/

// src/aliases/aliases.ts
import {
  default as default2
} from "../icons/fingerprint-pattern";
import {
  default as default3
} from "../icons/alarm-check";
import {
  default as default4
} from "../icons/alarm-minus";
import {
  default as default5
} from "../icons/alarm-plus";
import {
  default as default6
} from "../icons/arrow-down-az";
import {
  default as default7
} from "../icons/sort-desc";
import {
  default as default8
} from "../icons/arrow-down-za";
import {
  default as default9
} from "../icons/arrow-up-az";
import {
  default as default10
} from "../icons/sort-asc";
import {
  default as default11
} from "../icons/arrow-up-za";
import {
  default as default12
} from "../icons/axis-3-d";
import {
  default as default13
} from "../icons/verified";
import {
  default as default14
} from "../icons/badge-help";
import {
  default as default15
} from "../icons/between-horizonal-end";
import {
  default as default16
} from "../icons/between-horizonal-start";
import {
  default as default17
} from "../icons/book-template";
import {
  default as default18
} from "../icons/curly-braces";
import {
  default as default19
} from "../icons/subtitles";
import {
  default as default20
} from "../icons/area-chart";
import {
  default as default21
} from "../icons/bar-chart-horizontal-big";
import {
  default as default22
} from "../icons/bar-chart-horizontal";
import {
  default as default23
} from "../icons/candlestick-chart";
import {
  default as default24
} from "../icons/bar-chart-4";
import {
  default as default25
} from "../icons/bar-chart-big";
import {
  default as default26
} from "../icons/bar-chart-3";
import {
  default as default27
} from "../icons/line-chart";
import {
  default as default28
} from "../icons/bar-chart";
import {
  default as default29
} from "../icons/bar-chart-2";
import {
  default as default30
} from "../icons/gantt-chart";
import {
  default as default31
} from "../icons/pie-chart";
import {
  default as default32
} from "../icons/scatter-chart";
import {
  default as default33
} from "../icons/alert-circle";
import {
  default as default34
} from "../icons/arrow-down-circle";
import {
  default as default35
} from "../icons/arrow-down-left-from-circle";
import {
  default as default36
} from "../icons/arrow-left-circle";
import {
  default as default37
} from "../icons/arrow-down-right-from-circle";
import {
  default as default38
} from "../icons/arrow-up-left-from-circle";
import {
  default as default39
} from "../icons/arrow-up-right-from-circle";
import {
  default as default40
} from "../icons/arrow-right-circle";
import {
  default as default41
} from "../icons/arrow-up-circle";
import {
  default as default42
} from "../icons/check-circle";
import {
  default as default43
} from "../icons/check-circle-2";
import {
  default as default44
} from "../icons/chevron-left-circle";
import {
  default as default45
} from "../icons/chevron-down-circle";
import {
  default as default46
} from "../icons/chevron-right-circle";
import {
  default as default47
} from "../icons/chevron-up-circle";
import {
  default as default48
} from "../icons/divide-circle";
import {
  default as default49
} from "../icons/gauge-circle";
import {
  default as default50
} from "../icons/minus-circle";
import {
  default as default51
} from "../icons/parking-circle-off";
import {
  default as default52
} from "../icons/pause-circle";
import {
  default as default53
} from "../icons/parking-circle";
import {
  default as default54
} from "../icons/percent-circle";
import {
  default as default55
} from "../icons/play-circle";
import {
  default as default56
} from "../icons/plus-circle";
import {
  default as default57
} from "../icons/power-circle";
import {
  default as default58
} from "../icons/help-circle";
import {
  default as default59
} from "../icons/circle-help";
import {
  default as default60
} from "../icons/circle-slashed";
import {
  default as default61
} from "../icons/stop-circle";
import {
  default as default62
} from "../icons/user-circle-2";
import {
  default as default63
} from "../icons/user-circle";
import {
  default as default64
} from "../icons/x-circle";
import {
  default as default65
} from "../icons/clipboard-signature";
import {
  default as default66
} from "../icons/clipboard-edit";
import {
  default as default67
} from "../icons/download-cloud";
import {
  default as default68
} from "../icons/upload-cloud";
import {
  default as default69
} from "../icons/code-2";
import {
  default as default70
} from "../icons/columns";
import {
  default as default71
} from "../icons/columns-settings";
import {
  default as default72
} from "../icons/table-config";
import {
  default as default73
} from "../icons/panels-left-right";
import {
  default as default74
} from "../icons/contact-2";
import {
  default as default75
} from "../icons/percent-diamond";
import {
  default as default76
} from "../icons/globe-2";
import {
  default as default77
} from "../icons/more-vertical";
import {
  default as default78
} from "../icons/more-horizontal";
import {
  default as default79
} from "../icons/file-axis-3-d";
import {
  default as default80
} from "../icons/file-badge-2";
import {
  default as default81
} from "../icons/file-json-2";
import {
  default as default82
} from "../icons/file-json";
import {
  default as default83
} from "../icons/file-bar-chart";
import {
  default as default84
} from "../icons/file-bar-chart-2";
import {
  default as default85
} from "../icons/file-line-chart";
import {
  default as default86
} from "../icons/file-pie-chart";
import {
  default as default87
} from "../icons/file-check-2";
import {
  default as default88
} from "../icons/file-code-2";
import {
  default as default89
} from "../icons/file-cog-2";
import {
  default as default90
} from "../icons/file-warning";
import {
  default as default91
} from "../icons/file-audio";
import {
  default as default92
} from "../icons/file-audio-2";
import {
  default as default93
} from "../icons/file-key-2";
import {
  default as default94
} from "../icons/file-lock-2";
import {
  default as default95
} from "../icons/file-minus-2";
import {
  default as default96
} from "../icons/file-signature";
import {
  default as default97
} from "../icons/file-edit";
import {
  default as default98
} from "../icons/file-video";
import {
  default as default99
} from "../icons/file-plus-2";
import {
  default as default100
} from "../icons/file-search-2";
import {
  default as default101
} from "../icons/file-question";
import {
  default as default102
} from "../icons/file-volume-2";
import {
  default as default103
} from "../icons/file-type-2";
import {
  default as default104
} from "../icons/file-video-2";
import {
  default as default105
} from "../icons/file-x-2";
import {
  default as default106
} from "../icons/folder-cog-2";
import {
  default as default107
} from "../icons/folder-edit";
import {
  default as default108
} from "../icons/filter-x";
import {
  default as default109
} from "../icons/filter";
import {
  default as default110
} from "../icons/git-commit";
import {
  default as default111
} from "../icons/grid-2-x-2-check";
import {
  default as default112
} from "../icons/grid-2-x-2-plus";
import {
  default as default113
} from "../icons/grid-2-x-2";
import {
  default as default114
} from "../icons/grid-2-x-2-x";
import {
  default as default115
} from "../icons/grid";
import {
  default as default116
} from "../icons/grid-3-x-3";
import {
  default as default117
} from "../icons/helping-hand";
import {
  default as default118
} from "../icons/grab";
import {
  default as default119
} from "../icons/home";
import {
  default as default120
} from "../icons/ice-cream-2";
import {
  default as default121
} from "../icons/ice-cream";
import {
  default as default122
} from "../icons/laptop-2";
import {
  default as default123
} from "../icons/layers-3";
import {
  default as default124
} from "../icons/outdent";
import {
  default as default125
} from "../icons/indent-decrease";
import {
  default as default126
} from "../icons/indent";
import {
  default as default127
} from "../icons/indent-increase";
import {
  default as default128
} from "../icons/loader-2";
import {
  default as default129
} from "../icons/unlock-keyhole";
import {
  default as default130
} from "../icons/unlock";
import {
  default as default131
} from "../icons/mail-question";
import {
  default as default132
} from "../icons/location-edit";
import {
  default as default133
} from "../icons/message-circle-question";
import {
  default as default134
} from "../icons/mic-2";
import {
  default as default135
} from "../icons/move-3-d";
import {
  default as default136
} from "../icons/alert-octagon";
import {
  default as default137
} from "../icons/pause-octagon";
import {
  default as default138
} from "../icons/x-octagon";
import {
  default as default139
} from "../icons/paintbrush-2";
import {
  default as default140
} from "../icons/panel-bottom-inactive";
import {
  default as default141
} from "../icons/panel-left-inactive";
import {
  default as default142
} from "../icons/sidebar-close";
import {
  default as default143
} from "../icons/sidebar-open";
import {
  default as default144
} from "../icons/sidebar";
import {
  default as default145
} from "../icons/panel-right-inactive";
import {
  default as default146
} from "../icons/panel-top-inactive";
import {
  default as default147
} from "../icons/layout";
import {
  default as default148
} from "../icons/edit-3";
import {
  default as default149
} from "../icons/edit-2";
import {
  default as default150
} from "../icons/plug-zap-2";
import {
  default as default151
} from "../icons/form-input";
import {
  default as default152
} from "../icons/rotate-3-d";
import {
  default as default153
} from "../icons/rows";
import {
  default as default154
} from "../icons/panels-top-bottom";
import {
  default as default155
} from "../icons/scale-3-d";
import {
  default as default156
} from "../icons/send-horizonal";
import {
  default as default157
} from "../icons/shield-question";
import {
  default as default158
} from "../icons/shield-close";
import {
  default as default159
} from "../icons/sliders";
import {
  default as default160
} from "../icons/stars";
import {
  default as default161
} from "../icons/activity-square";
import {
  default as default162
} from "../icons/arrow-down-left-square";
import {
  default as default163
} from "../icons/arrow-down-right-square";
import {
  default as default164
} from "../icons/arrow-down-square";
import {
  default as default165
} from "../icons/arrow-left-square";
import {
  default as default166
} from "../icons/arrow-down-right-from-square";
import {
  default as default167
} from "../icons/arrow-down-left-from-square";
import {
  default as default168
} from "../icons/arrow-up-left-from-square";
import {
  default as default169
} from "../icons/arrow-up-right-from-square";
import {
  default as default170
} from "../icons/arrow-right-square";
import {
  default as default171
} from "../icons/arrow-up-left-square";
import {
  default as default172
} from "../icons/arrow-up-right-square";
import {
  default as default173
} from "../icons/arrow-up-square";
import {
  default as default174
} from "../icons/asterisk-square";
import {
  default as default175
} from "../icons/scissors-square-dashed-bottom";
import {
  default as default176
} from "../icons/flip-horizontal";
import {
  default as default177
} from "../icons/flip-vertical";
import {
  default as default178
} from "../icons/gantt-chart-square";
import {
  default as default179
} from "../icons/square-gantt-chart";
import {
  default as default180
} from "../icons/check-square";
import {
  default as default181
} from "../icons/chevron-down-square";
import {
  default as default182
} from "../icons/check-square-2";
import {
  default as default183
} from "../icons/chevron-left-square";
import {
  default as default184
} from "../icons/chevron-right-square";
import {
  default as default185
} from "../icons/chevron-up-square";
import {
  default as default186
} from "../icons/code-square";
import {
  default as default187
} from "../icons/kanban-square-dashed";
import {
  default as default188
} from "../icons/mouse-pointer-square-dashed";
import {
  default as default189
} from "../icons/text-selection";
import {
  default as default190
} from "../icons/text-select";
import {
  default as default191
} from "../icons/box-select";
import {
  default as default192
} from "../icons/divide-square";
import {
  default as default193
} from "../icons/dot-square";
import {
  default as default194
} from "../icons/equal-square";
import {
  default as default195
} from "../icons/function-square";
import {
  default as default196
} from "../icons/kanban-square";
import {
  default as default197
} from "../icons/library-square";
import {
  default as default198
} from "../icons/m-square";
import {
  default as default199
} from "../icons/menu-square";
import {
  default as default200
} from "../icons/minus-square";
import {
  default as default201
} from "../icons/inspect";
import {
  default as default202
} from "../icons/parking-square-off";
import {
  default as default203
} from "../icons/parking-square";
import {
  default as default204
} from "../icons/pen-box";
import {
  default as default205
} from "../icons/edit";
import {
  default as default206
} from "../icons/pen-square";
import {
  default as default207
} from "../icons/percent-square";
import {
  default as default208
} from "../icons/pi-square";
import {
  default as default209
} from "../icons/pilcrow-square";
import {
  default as default210
} from "../icons/play-square";
import {
  default as default211
} from "../icons/plus-square";
import {
  default as default212
} from "../icons/power-square";
import {
  default as default213
} from "../icons/scissors-square";
import {
  default as default214
} from "../icons/sigma-square";
import {
  default as default215
} from "../icons/slash-square";
import {
  default as default216
} from "../icons/split-square-horizontal";
import {
  default as default217
} from "../icons/split-square-vertical";
import {
  default as default218
} from "../icons/terminal-square";
import {
  default as default219
} from "../icons/user-square-2";
import {
  default as default220
} from "../icons/user-square";
import {
  default as default221
} from "../icons/x-square";
import {
  default as default222
} from "../icons/test-tube-2";
import {
  default as default223
} from "../icons/align-center";
import {
  default as default224
} from "../icons/align-right";
import {
  default as default225
} from "../icons/align-justify";
import {
  default as default226
} from "../icons/text";
import {
  default as default227
} from "../icons/align-left";
import {
  default as default228
} from "../icons/letter-text";
import {
  default as default229
} from "../icons/wrap-text";
import {
  default as default230
} from "../icons/train";
import {
  default as default231
} from "../icons/palmtree";
import {
  default as default232
} from "../icons/alert-triangle";
import {
  default as default233
} from "../icons/tv-2";
import {
  default as default234
} from "../icons/school-2";
import {
  default as default235
} from "../icons/user-check-2";
import {
  default as default236
} from "../icons/user-cog-2";
import {
  default as default237
} from "../icons/user-minus-2";
import {
  default as default238
} from "../icons/user-plus-2";
import {
  default as default239
} from "../icons/user-x-2";
import {
  default as default240
} from "../icons/user-2";
import {
  default as default241
} from "../icons/users-2";
import {
  default as default242
} from "../icons/fork-knife-crossed";
import {
  default as default243
} from "../icons/fork-knife";
import {
  default as default244
} from "../icons/wallet-2";
import {
  default as default245
} from "../icons/wand-2";
export {
  default161 as ActivitySquare,
  default3 as AlarmCheck,
  default4 as AlarmMinus,
  default5 as AlarmPlus,
  default33 as AlertCircle,
  default136 as AlertOctagon,
  default232 as AlertTriangle,
  default223 as AlignCenter,
  default225 as AlignJustify,
  default227 as AlignLeft,
  default224 as AlignRight,
  default20 as AreaChart,
  default6 as ArrowDownAz,
  default34 as ArrowDownCircle,
  default35 as ArrowDownLeftFromCircle,
  default167 as ArrowDownLeftFromSquare,
  default162 as ArrowDownLeftSquare,
  default37 as ArrowDownRightFromCircle,
  default166 as ArrowDownRightFromSquare,
  default163 as ArrowDownRightSquare,
  default164 as ArrowDownSquare,
  default8 as ArrowDownZa,
  default36 as ArrowLeftCircle,
  default165 as ArrowLeftSquare,
  default40 as ArrowRightCircle,
  default170 as ArrowRightSquare,
  default9 as ArrowUpAz,
  default41 as ArrowUpCircle,
  default38 as ArrowUpLeftFromCircle,
  default168 as ArrowUpLeftFromSquare,
  default171 as ArrowUpLeftSquare,
  default39 as ArrowUpRightFromCircle,
  default169 as ArrowUpRightFromSquare,
  default172 as ArrowUpRightSquare,
  default173 as ArrowUpSquare,
  default11 as ArrowUpZa,
  default174 as AsteriskSquare,
  default12 as Axis3D,
  default14 as BadgeHelp,
  default28 as BarChart,
  default29 as BarChart2,
  default26 as BarChart3,
  default24 as BarChart4,
  default25 as BarChartBig,
  default22 as BarChartHorizontal,
  default21 as BarChartHorizontalBig,
  default15 as BetweenHorizonalEnd,
  default16 as BetweenHorizonalStart,
  default17 as BookTemplate,
  default191 as BoxSelect,
  default23 as CandlestickChart,
  default42 as CheckCircle,
  default43 as CheckCircle2,
  default180 as CheckSquare,
  default182 as CheckSquare2,
  default45 as ChevronDownCircle,
  default181 as ChevronDownSquare,
  default44 as ChevronLeftCircle,
  default183 as ChevronLeftSquare,
  default46 as ChevronRightCircle,
  default184 as ChevronRightSquare,
  default47 as ChevronUpCircle,
  default185 as ChevronUpSquare,
  default59 as CircleHelp,
  default60 as CircleSlashed,
  default66 as ClipboardEdit,
  default65 as ClipboardSignature,
  default69 as Code2,
  default186 as CodeSquare,
  default70 as Columns,
  default71 as ColumnsSettings,
  default74 as Contact2,
  default18 as CurlyBraces,
  default48 as DivideCircle,
  default192 as DivideSquare,
  default193 as DotSquare,
  default67 as DownloadCloud,
  default205 as Edit,
  default149 as Edit2,
  default148 as Edit3,
  default194 as EqualSquare,
  default91 as FileAudio,
  default92 as FileAudio2,
  default79 as FileAxis3D,
  default80 as FileBadge2,
  default83 as FileBarChart,
  default84 as FileBarChart2,
  default87 as FileCheck2,
  default88 as FileCode2,
  default89 as FileCog2,
  default97 as FileEdit,
  default82 as FileJson,
  default81 as FileJson2,
  default93 as FileKey2,
  default85 as FileLineChart,
  default94 as FileLock2,
  default95 as FileMinus2,
  default86 as FilePieChart,
  default99 as FilePlus2,
  default101 as FileQuestion,
  default100 as FileSearch2,
  default96 as FileSignature,
  default103 as FileType2,
  default98 as FileVideo,
  default104 as FileVideo2,
  default102 as FileVolume2,
  default90 as FileWarning,
  default105 as FileX2,
  default109 as Filter,
  default108 as FilterX,
  default2 as Fingerprint,
  default176 as FlipHorizontal,
  default177 as FlipVertical,
  default106 as FolderCog2,
  default107 as FolderEdit,
  default243 as ForkKnife,
  default242 as ForkKnifeCrossed,
  default151 as FormInput,
  default195 as FunctionSquare,
  default30 as GanttChart,
  default178 as GanttChartSquare,
  default49 as GaugeCircle,
  default110 as GitCommit,
  default76 as Globe2,
  default118 as Grab,
  default115 as Grid,
  default113 as Grid2X2,
  default111 as Grid2X2Check,
  default112 as Grid2X2Plus,
  default114 as Grid2X2X,
  default116 as Grid3X3,
  default58 as HelpCircle,
  default117 as HelpingHand,
  default119 as Home,
  default121 as IceCream,
  default120 as IceCream2,
  default126 as Indent,
  default125 as IndentDecrease,
  default127 as IndentIncrease,
  default201 as Inspect,
  default196 as KanbanSquare,
  default187 as KanbanSquareDashed,
  default122 as Laptop2,
  default123 as Layers3,
  default147 as Layout,
  default228 as LetterText,
  default197 as LibrarySquare,
  default27 as LineChart,
  default128 as Loader2,
  default132 as LocationEdit,
  default198 as MSquare,
  default131 as MailQuestion,
  default199 as MenuSquare,
  default133 as MessageCircleQuestion,
  default134 as Mic2,
  default50 as MinusCircle,
  default200 as MinusSquare,
  default78 as MoreHorizontal,
  default77 as MoreVertical,
  default188 as MousePointerSquareDashed,
  default135 as Move3D,
  default124 as Outdent,
  default139 as Paintbrush2,
  default231 as Palmtree,
  default140 as PanelBottomInactive,
  default141 as PanelLeftInactive,
  default145 as PanelRightInactive,
  default146 as PanelTopInactive,
  default73 as PanelsLeftRight,
  default154 as PanelsTopBottom,
  default53 as ParkingCircle,
  default51 as ParkingCircleOff,
  default203 as ParkingSquare,
  default202 as ParkingSquareOff,
  default52 as PauseCircle,
  default137 as PauseOctagon,
  default204 as PenBox,
  default206 as PenSquare,
  default54 as PercentCircle,
  default75 as PercentDiamond,
  default207 as PercentSquare,
  default208 as PiSquare,
  default31 as PieChart,
  default209 as PilcrowSquare,
  default55 as PlayCircle,
  default210 as PlaySquare,
  default150 as PlugZap2,
  default56 as PlusCircle,
  default211 as PlusSquare,
  default57 as PowerCircle,
  default212 as PowerSquare,
  default152 as Rotate3D,
  default153 as Rows,
  default155 as Scale3D,
  default32 as ScatterChart,
  default234 as School2,
  default213 as ScissorsSquare,
  default175 as ScissorsSquareDashedBottom,
  default156 as SendHorizonal,
  default158 as ShieldClose,
  default157 as ShieldQuestion,
  default144 as Sidebar,
  default142 as SidebarClose,
  default143 as SidebarOpen,
  default214 as SigmaSquare,
  default215 as SlashSquare,
  default159 as Sliders,
  default10 as SortAsc,
  default7 as SortDesc,
  default216 as SplitSquareHorizontal,
  default217 as SplitSquareVertical,
  default179 as SquareGanttChart,
  default160 as Stars,
  default61 as StopCircle,
  default19 as Subtitles,
  default72 as TableConfig,
  default218 as TerminalSquare,
  default222 as TestTube2,
  default226 as Text,
  default190 as TextSelect,
  default189 as TextSelection,
  default230 as Train,
  default233 as Tv2,
  default130 as Unlock,
  default129 as UnlockKeyhole,
  default68 as UploadCloud,
  default240 as User2,
  default235 as UserCheck2,
  default63 as UserCircle,
  default62 as UserCircle2,
  default236 as UserCog2,
  default237 as UserMinus2,
  default238 as UserPlus2,
  default220 as UserSquare,
  default219 as UserSquare2,
  default239 as UserX2,
  default241 as Users2,
  default13 as Verified,
  default244 as Wallet2,
  default245 as Wand2,
  default229 as WrapText,
  default64 as XCircle,
  default138 as XOctagon,
  default221 as XSquare
};
