// lucide-solid/dist/source/aliases/prefixed.jsx
/**
* @license lucide-solid v1.11.0 - ISC
*
* This source code is licensed under the ISC license.
* See the LICENSE file in the root directory of this source tree.
*/

// src/aliases/prefixed.ts
import { default as default2 } from "../icons/a-arrow-down";
import { default as default3 } from "../icons/a-arrow-up";
import { default as default4 } from "../icons/a-large-small";
import { default as default5 } from "../icons/accessibility";
import { default as default6 } from "../icons/activity";
import { default as default7 } from "../icons/air-vent";
import { default as default8 } from "../icons/airplay";
import { default as default9 } from "../icons/alarm-clock-off";
import { default as default10 } from "../icons/alarm-clock";
import { default as default11 } from "../icons/alarm-smoke";
import { default as default12 } from "../icons/album";
import { default as default13 } from "../icons/align-center-horizontal";
import { default as default14 } from "../icons/align-center-vertical";
import { default as default15 } from "../icons/align-end-horizontal";
import { default as default16 } from "../icons/align-end-vertical";
import { default as default17 } from "../icons/align-horizontal-distribute-center";
import { default as default18 } from "../icons/align-horizontal-distribute-end";
import { default as default19 } from "../icons/align-horizontal-distribute-start";
import { default as default20 } from "../icons/align-horizontal-justify-center";
import { default as default21 } from "../icons/align-horizontal-justify-end";
import { default as default22 } from "../icons/align-horizontal-justify-start";
import { default as default23 } from "../icons/align-horizontal-space-around";
import { default as default24 } from "../icons/align-horizontal-space-between";
import { default as default25 } from "../icons/align-start-horizontal";
import { default as default26 } from "../icons/align-start-vertical";
import { default as default27 } from "../icons/align-vertical-distribute-center";
import { default as default28 } from "../icons/align-vertical-distribute-end";
import { default as default29 } from "../icons/align-vertical-distribute-start";
import { default as default30 } from "../icons/align-vertical-justify-center";
import { default as default31 } from "../icons/align-vertical-justify-end";
import { default as default32 } from "../icons/align-vertical-justify-start";
import { default as default33 } from "../icons/align-vertical-space-around";
import { default as default34 } from "../icons/align-vertical-space-between";
import { default as default35 } from "../icons/ambulance";
import { default as default36 } from "../icons/ampersand";
import { default as default37 } from "../icons/ampersands";
import { default as default38 } from "../icons/amphora";
import { default as default39 } from "../icons/anchor";
import { default as default40 } from "../icons/angry";
import { default as default41 } from "../icons/annoyed";
import { default as default42 } from "../icons/antenna";
import { default as default43 } from "../icons/anvil";
import { default as default44 } from "../icons/aperture";
import { default as default45 } from "../icons/app-window-mac";
import { default as default46 } from "../icons/app-window";
import { default as default47 } from "../icons/apple";
import { default as default48 } from "../icons/archive-restore";
import { default as default49 } from "../icons/archive-x";
import { default as default50 } from "../icons/archive";
import { default as default51 } from "../icons/armchair";
import { default as default52 } from "../icons/arrow-big-down-dash";
import { default as default53 } from "../icons/arrow-big-down";
import { default as default54 } from "../icons/arrow-big-left-dash";
import { default as default55 } from "../icons/arrow-big-right-dash";
import { default as default56 } from "../icons/arrow-big-left";
import { default as default57 } from "../icons/arrow-big-right";
import { default as default58 } from "../icons/arrow-big-up-dash";
import { default as default59 } from "../icons/arrow-big-up";
import { default as default60 } from "../icons/arrow-down-from-line";
import { default as default61 } from "../icons/arrow-down-left";
import { default as default62 } from "../icons/arrow-down-right";
import { default as default63 } from "../icons/arrow-down-narrow-wide";
import { default as default64 } from "../icons/arrow-down-to-dot";
import { default as default65 } from "../icons/arrow-down-to-line";
import { default as default66 } from "../icons/arrow-down-up";
import { default as default67 } from "../icons/arrow-down";
import { default as default68 } from "../icons/arrow-left-from-line";
import { default as default69 } from "../icons/arrow-left-right";
import { default as default70 } from "../icons/arrow-left-to-line";
import { default as default71 } from "../icons/arrow-left";
import { default as default72 } from "../icons/arrow-right-from-line";
import { default as default73 } from "../icons/arrow-right-left";
import { default as default74 } from "../icons/arrow-right-to-line";
import { default as default75 } from "../icons/arrow-right";
import { default as default76 } from "../icons/arrow-up-down";
import { default as default77 } from "../icons/arrow-up-from-dot";
import { default as default78 } from "../icons/arrow-up-from-line";
import { default as default79 } from "../icons/arrow-up-left";
import { default as default80 } from "../icons/arrow-up-right";
import { default as default81 } from "../icons/arrow-up-to-line";
import { default as default82 } from "../icons/arrow-up-wide-narrow";
import { default as default83 } from "../icons/arrow-up";
import { default as default84 } from "../icons/arrows-up-from-line";
import { default as default85 } from "../icons/asterisk";
import { default as default86 } from "../icons/atom";
import { default as default87 } from "../icons/at-sign";
import { default as default88 } from "../icons/audio-lines";
import { default as default89 } from "../icons/audio-waveform";
import { default as default90 } from "../icons/axe";
import { default as default91 } from "../icons/award";
import { default as default92 } from "../icons/baby";
import { default as default93 } from "../icons/backpack";
import { default as default94 } from "../icons/badge-alert";
import { default as default95 } from "../icons/badge-cent";
import { default as default96 } from "../icons/badge-dollar-sign";
import { default as default97 } from "../icons/badge-euro";
import { default as default98 } from "../icons/badge-indian-rupee";
import { default as default99 } from "../icons/badge-info";
import { default as default100 } from "../icons/badge-minus";
import { default as default101 } from "../icons/badge-japanese-yen";
import { default as default102 } from "../icons/badge-percent";
import { default as default103 } from "../icons/badge-plus";
import { default as default104 } from "../icons/badge-pound-sterling";
import { default as default105 } from "../icons/badge-russian-ruble";
import { default as default106 } from "../icons/badge-swiss-franc";
import { default as default107 } from "../icons/badge-turkish-lira";
import { default as default108 } from "../icons/badge-x";
import { default as default109 } from "../icons/badge";
import { default as default110 } from "../icons/baggage-claim";
import { default as default111 } from "../icons/balloon";
import { default as default112 } from "../icons/ban";
import { default as default113 } from "../icons/banana";
import { default as default114 } from "../icons/bandage";
import { default as default115 } from "../icons/banknote-arrow-down";
import { default as default116 } from "../icons/banknote-arrow-up";
import { default as default117 } from "../icons/banknote-x";
import { default as default118 } from "../icons/banknote";
import { default as default119 } from "../icons/barcode";
import { default as default120 } from "../icons/barrel";
import { default as default121 } from "../icons/baseline";
import { default as default122 } from "../icons/bath";
import { default as default123 } from "../icons/battery-charging";
import { default as default124 } from "../icons/battery-full";
import { default as default125 } from "../icons/battery-low";
import { default as default126 } from "../icons/battery-medium";
import { default as default127 } from "../icons/battery-plus";
import { default as default128 } from "../icons/battery-warning";
import { default as default129 } from "../icons/battery";
import { default as default130 } from "../icons/beaker";
import { default as default131 } from "../icons/bean-off";
import { default as default132 } from "../icons/bean";
import { default as default133 } from "../icons/bed-double";
import { default as default134 } from "../icons/bed-single";
import { default as default135 } from "../icons/bed";
import { default as default136 } from "../icons/beef-off";
import { default as default137 } from "../icons/beef";
import { default as default138 } from "../icons/beer-off";
import { default as default139 } from "../icons/beer";
import { default as default140 } from "../icons/bell-check";
import { default as default141 } from "../icons/bell-dot";
import { default as default142 } from "../icons/bell-electric";
import { default as default143 } from "../icons/bell-minus";
import { default as default144 } from "../icons/bell-off";
import { default as default145 } from "../icons/bell-plus";
import { default as default146 } from "../icons/bell-ring";
import { default as default147 } from "../icons/bell";
import { default as default148 } from "../icons/between-vertical-end";
import { default as default149 } from "../icons/between-vertical-start";
import { default as default150 } from "../icons/biceps-flexed";
import { default as default151 } from "../icons/bike";
import { default as default152 } from "../icons/binary";
import { default as default153 } from "../icons/biohazard";
import { default as default154 } from "../icons/binoculars";
import { default as default155 } from "../icons/bird";
import { default as default156 } from "../icons/birdhouse";
import { default as default157 } from "../icons/bitcoin";
import { default as default158 } from "../icons/blend";
import { default as default159 } from "../icons/blinds";
import { default as default160 } from "../icons/blocks";
import { default as default161 } from "../icons/bluetooth-connected";
import { default as default162 } from "../icons/bluetooth-off";
import { default as default163 } from "../icons/bluetooth-searching";
import { default as default164 } from "../icons/bluetooth";
import { default as default165 } from "../icons/bold";
import { default as default166 } from "../icons/bolt";
import { default as default167 } from "../icons/bomb";
import { default as default168 } from "../icons/bone";
import { default as default169 } from "../icons/book-a";
import { default as default170 } from "../icons/book-alert";
import { default as default171 } from "../icons/book-audio";
import { default as default172 } from "../icons/book-check";
import { default as default173 } from "../icons/book-copy";
import { default as default174 } from "../icons/book-down";
import { default as default175 } from "../icons/book-headphones";
import { default as default176 } from "../icons/book-heart";
import { default as default177 } from "../icons/book-image";
import { default as default178 } from "../icons/book-key";
import { default as default179 } from "../icons/book-lock";
import { default as default180 } from "../icons/book-marked";
import { default as default181 } from "../icons/book-minus";
import { default as default182 } from "../icons/book-open-check";
import { default as default183 } from "../icons/book-open-text";
import { default as default184 } from "../icons/book-open";
import { default as default185 } from "../icons/book-plus";
import { default as default186 } from "../icons/book-search";
import { default as default187 } from "../icons/book-text";
import { default as default188 } from "../icons/book-type";
import { default as default189 } from "../icons/book-up-2";
import { default as default190 } from "../icons/book-up";
import { default as default191 } from "../icons/book-user";
import { default as default192 } from "../icons/book-x";
import { default as default193 } from "../icons/book";
import { default as default194 } from "../icons/bookmark-check";
import { default as default195 } from "../icons/bookmark-minus";
import { default as default196 } from "../icons/bookmark-off";
import { default as default197 } from "../icons/bookmark-plus";
import { default as default198 } from "../icons/bookmark-x";
import { default as default199 } from "../icons/bookmark";
import { default as default200 } from "../icons/boom-box";
import { default as default201 } from "../icons/bot-message-square";
import { default as default202 } from "../icons/bot-off";
import { default as default203 } from "../icons/bot";
import { default as default204 } from "../icons/bottle-wine";
import { default as default205 } from "../icons/bow-arrow";
import { default as default206 } from "../icons/box";
import { default as default207 } from "../icons/boxes";
import { default as default208 } from "../icons/brackets";
import { default as default209 } from "../icons/brain-circuit";
import { default as default210 } from "../icons/brain-cog";
import { default as default211 } from "../icons/brain";
import { default as default212 } from "../icons/brick-wall-fire";
import { default as default213 } from "../icons/brick-wall-shield";
import { default as default214 } from "../icons/brick-wall";
import { default as default215 } from "../icons/briefcase-business";
import { default as default216 } from "../icons/briefcase-conveyor-belt";
import { default as default217 } from "../icons/briefcase-medical";
import { default as default218 } from "../icons/briefcase";
import { default as default219 } from "../icons/bring-to-front";
import { default as default220 } from "../icons/brush-cleaning";
import { default as default221 } from "../icons/brush";
import { default as default222 } from "../icons/bubbles";
import { default as default223 } from "../icons/bug-off";
import { default as default224 } from "../icons/bug-play";
import { default as default225 } from "../icons/bug";
import { default as default226 } from "../icons/building-2";
import { default as default227 } from "../icons/building";
import { default as default228 } from "../icons/bus-front";
import { default as default229 } from "../icons/bus";
import { default as default230 } from "../icons/cable-car";
import { default as default231 } from "../icons/cable";
import { default as default232 } from "../icons/cake-slice";
import { default as default233 } from "../icons/cake";
import { default as default234 } from "../icons/calculator";
import { default as default235 } from "../icons/calendar-1";
import { default as default236 } from "../icons/calendar-arrow-down";
import { default as default237 } from "../icons/calendar-arrow-up";
import { default as default238 } from "../icons/calendar-check-2";
import { default as default239 } from "../icons/calendar-check";
import { default as default240 } from "../icons/calendar-clock";
import { default as default241 } from "../icons/calendar-cog";
import { default as default242 } from "../icons/calendar-days";
import { default as default243 } from "../icons/calendar-heart";
import { default as default244 } from "../icons/calendar-fold";
import { default as default245 } from "../icons/calendar-minus-2";
import { default as default246 } from "../icons/calendar-minus";
import { default as default247 } from "../icons/calendar-off";
import { default as default248 } from "../icons/calendar-plus-2";
import { default as default249 } from "../icons/calendar-plus";
import { default as default250 } from "../icons/calendar-range";
import { default as default251 } from "../icons/calendar-sync";
import { default as default252 } from "../icons/calendar-search";
import { default as default253 } from "../icons/calendar-x-2";
import { default as default254 } from "../icons/calendar-x";
import { default as default255 } from "../icons/calendar";
import { default as default256 } from "../icons/calendars";
import { default as default257 } from "../icons/camera-off";
import { default as default258 } from "../icons/camera";
import { default as default259 } from "../icons/candy-cane";
import { default as default260 } from "../icons/candy-off";
import { default as default261 } from "../icons/candy";
import { default as default262 } from "../icons/cannabis-off";
import { default as default263 } from "../icons/cannabis";
import { default as default264 } from "../icons/captions-off";
import { default as default265 } from "../icons/car-front";
import { default as default266 } from "../icons/car-taxi-front";
import { default as default267 } from "../icons/caravan";
import { default as default268 } from "../icons/car";
import { default as default269 } from "../icons/card-sim";
import { default as default270 } from "../icons/carrot";
import { default as default271 } from "../icons/case-lower";
import { default as default272 } from "../icons/case-sensitive";
import { default as default273 } from "../icons/case-upper";
import { default as default274 } from "../icons/cassette-tape";
import { default as default275 } from "../icons/cast";
import { default as default276 } from "../icons/castle";
import { default as default277 } from "../icons/cat";
import { default as default278 } from "../icons/cctv-off";
import { default as default279 } from "../icons/cctv";
import { default as default280 } from "../icons/chart-bar-decreasing";
import { default as default281 } from "../icons/chart-bar-increasing";
import { default as default282 } from "../icons/chart-bar-stacked";
import { default as default283 } from "../icons/chart-column-decreasing";
import { default as default284 } from "../icons/chart-column-stacked";
import { default as default285 } from "../icons/chart-gantt";
import { default as default286 } from "../icons/chart-network";
import { default as default287 } from "../icons/chart-no-axes-column-decreasing";
import { default as default288 } from "../icons/chart-no-axes-combined";
import { default as default289 } from "../icons/chart-spline";
import { default as default290 } from "../icons/check-check";
import { default as default291 } from "../icons/check-line";
import { default as default292 } from "../icons/check";
import { default as default293 } from "../icons/chef-hat";
import { default as default294 } from "../icons/cherry";
import { default as default295 } from "../icons/chess-bishop";
import { default as default296 } from "../icons/chess-king";
import { default as default297 } from "../icons/chess-knight";
import { default as default298 } from "../icons/chess-pawn";
import { default as default299 } from "../icons/chess-queen";
import { default as default300 } from "../icons/chess-rook";
import { default as default301 } from "../icons/chevron-down";
import { default as default302 } from "../icons/chevron-first";
import { default as default303 } from "../icons/chevron-last";
import { default as default304 } from "../icons/chevron-left";
import { default as default305 } from "../icons/chevron-right";
import { default as default306 } from "../icons/chevron-up";
import { default as default307 } from "../icons/chevrons-down-up";
import { default as default308 } from "../icons/chevrons-down";
import { default as default309 } from "../icons/chevrons-left-right-ellipsis";
import { default as default310 } from "../icons/chevrons-left-right";
import { default as default311 } from "../icons/chevrons-left";
import { default as default312 } from "../icons/chevrons-right-left";
import { default as default313 } from "../icons/chevrons-right";
import { default as default314 } from "../icons/chevrons-up-down";
import { default as default315 } from "../icons/chevrons-up";
import { default as default316 } from "../icons/church";
import { default as default317 } from "../icons/cigarette-off";
import { default as default318 } from "../icons/cigarette";
import { default as default319 } from "../icons/circle-dashed";
import { default as default320 } from "../icons/circle-dollar-sign";
import { default as default321 } from "../icons/circle-dot-dashed";
import { default as default322 } from "../icons/circle-ellipsis";
import { default as default323 } from "../icons/circle-dot";
import { default as default324 } from "../icons/circle-fading-arrow-up";
import { default as default325 } from "../icons/circle-equal";
import { default as default326 } from "../icons/circle-fading-plus";
import { default as default327 } from "../icons/circle-off";
import { default as default328 } from "../icons/circle-pile";
import { default as default329 } from "../icons/circle-pound-sterling";
import { default as default330 } from "../icons/circle-slash";
import { default as default331 } from "../icons/circle-small";
import { default as default332 } from "../icons/circle-star";
import { default as default333 } from "../icons/circle";
import { default as default334 } from "../icons/citrus";
import { default as default335 } from "../icons/circuit-board";
import { default as default336 } from "../icons/clapperboard";
import { default as default337 } from "../icons/clipboard-check";
import { default as default338 } from "../icons/clipboard-clock";
import { default as default339 } from "../icons/clipboard-copy";
import { default as default340 } from "../icons/clipboard-list";
import { default as default341 } from "../icons/clipboard-minus";
import { default as default342 } from "../icons/clipboard-paste";
import { default as default343 } from "../icons/clipboard-type";
import { default as default344 } from "../icons/clipboard-plus";
import { default as default345 } from "../icons/clipboard-x";
import { default as default346 } from "../icons/clipboard";
import { default as default347 } from "../icons/clock-1";
import { default as default348 } from "../icons/clock-10";
import { default as default349 } from "../icons/clock-11";
import { default as default350 } from "../icons/clock-12";
import { default as default351 } from "../icons/clock-2";
import { default as default352 } from "../icons/clock-3";
import { default as default353 } from "../icons/clock-4";
import { default as default354 } from "../icons/clock-5";
import { default as default355 } from "../icons/clock-6";
import { default as default356 } from "../icons/clock-7";
import { default as default357 } from "../icons/clock-8";
import { default as default358 } from "../icons/clock-9";
import { default as default359 } from "../icons/clock-alert";
import { default as default360 } from "../icons/clock-arrow-down";
import { default as default361 } from "../icons/clock-arrow-up";
import { default as default362 } from "../icons/clock-check";
import { default as default363 } from "../icons/clock-fading";
import { default as default364 } from "../icons/clock-plus";
import { default as default365 } from "../icons/clock";
import { default as default366 } from "../icons/closed-caption";
import { default as default367 } from "../icons/cloud-alert";
import { default as default368 } from "../icons/cloud-backup";
import { default as default369 } from "../icons/cloud-check";
import { default as default370 } from "../icons/cloud-cog";
import { default as default371 } from "../icons/cloud-drizzle";
import { default as default372 } from "../icons/cloud-fog";
import { default as default373 } from "../icons/cloud-hail";
import { default as default374 } from "../icons/cloud-lightning";
import { default as default375 } from "../icons/cloud-moon-rain";
import { default as default376 } from "../icons/cloud-moon";
import { default as default377 } from "../icons/cloud-off";
import { default as default378 } from "../icons/cloud-rain-wind";
import { default as default379 } from "../icons/cloud-rain";
import { default as default380 } from "../icons/cloud-snow";
import { default as default381 } from "../icons/cloud-sun-rain";
import { default as default382 } from "../icons/cloud-sun";
import { default as default383 } from "../icons/cloud-sync";
import { default as default384 } from "../icons/cloud";
import { default as default385 } from "../icons/cloudy";
import { default as default386 } from "../icons/clover";
import { default as default387 } from "../icons/club";
import { default as default388 } from "../icons/coffee";
import { default as default389 } from "../icons/cog";
import { default as default390 } from "../icons/coins";
import { default as default391 } from "../icons/combine";
import { default as default392 } from "../icons/code";
import { default as default393 } from "../icons/command";
import { default as default394 } from "../icons/compass";
import { default as default395 } from "../icons/columns-4";
import { default as default396 } from "../icons/component";
import { default as default397 } from "../icons/concierge-bell";
import { default as default398 } from "../icons/cone";
import { default as default399 } from "../icons/computer";
import { default as default400 } from "../icons/construction";
import { default as default401 } from "../icons/contact";
import { default as default402 } from "../icons/container";
import { default as default403 } from "../icons/contrast";
import { default as default404 } from "../icons/cookie";
import { default as default405 } from "../icons/cooking-pot";
import { default as default406 } from "../icons/copy-check";
import { default as default407 } from "../icons/copy-minus";
import { default as default408 } from "../icons/copy-plus";
import { default as default409 } from "../icons/copy-slash";
import { default as default410 } from "../icons/copy-x";
import { default as default411 } from "../icons/copy";
import { default as default412 } from "../icons/copyleft";
import { default as default413 } from "../icons/copyright";
import { default as default414 } from "../icons/corner-down-left";
import { default as default415 } from "../icons/corner-down-right";
import { default as default416 } from "../icons/corner-left-down";
import { default as default417 } from "../icons/corner-left-up";
import { default as default418 } from "../icons/corner-right-down";
import { default as default419 } from "../icons/corner-right-up";
import { default as default420 } from "../icons/corner-up-right";
import { default as default421 } from "../icons/corner-up-left";
import { default as default422 } from "../icons/cpu";
import { default as default423 } from "../icons/creative-commons";
import { default as default424 } from "../icons/credit-card";
import { default as default425 } from "../icons/croissant";
import { default as default426 } from "../icons/crop";
import { default as default427 } from "../icons/cross";
import { default as default428 } from "../icons/crosshair";
import { default as default429 } from "../icons/crown";
import { default as default430 } from "../icons/cuboid";
import { default as default431 } from "../icons/cup-soda";
import { default as default432 } from "../icons/currency";
import { default as default433 } from "../icons/cylinder";
import { default as default434 } from "../icons/dam";
import { default as default435 } from "../icons/database-backup";
import { default as default436 } from "../icons/database-search";
import { default as default437 } from "../icons/database-zap";
import { default as default438 } from "../icons/database";
import { default as default439 } from "../icons/decimals-arrow-left";
import { default as default440 } from "../icons/decimals-arrow-right";
import { default as default441 } from "../icons/delete";
import { default as default442 } from "../icons/dessert";
import { default as default443 } from "../icons/diameter";
import { default as default444 } from "../icons/diamond-minus";
import { default as default445 } from "../icons/diamond-plus";
import { default as default446 } from "../icons/dice-2";
import { default as default447 } from "../icons/diamond";
import { default as default448 } from "../icons/dice-1";
import { default as default449 } from "../icons/dice-3";
import { default as default450 } from "../icons/dice-4";
import { default as default451 } from "../icons/dice-5";
import { default as default452 } from "../icons/dice-6";
import { default as default453 } from "../icons/dices";
import { default as default454 } from "../icons/diff";
import { default as default455 } from "../icons/disc-2";
import { default as default456 } from "../icons/disc-3";
import { default as default457 } from "../icons/disc-album";
import { default as default458 } from "../icons/disc";
import { default as default459 } from "../icons/divide";
import { default as default460 } from "../icons/dna-off";
import { default as default461 } from "../icons/dna";
import { default as default462 } from "../icons/dock";
import { default as default463 } from "../icons/dog";
import { default as default464 } from "../icons/dollar-sign";
import { default as default465 } from "../icons/donut";
import { default as default466 } from "../icons/door-closed-locked";
import { default as default467 } from "../icons/door-closed";
import { default as default468 } from "../icons/door-open";
import { default as default469 } from "../icons/dot";
import { default as default470 } from "../icons/download";
import { default as default471 } from "../icons/drafting-compass";
import { default as default472 } from "../icons/drama";
import { default as default473 } from "../icons/drill";
import { default as default474 } from "../icons/drone";
import { default as default475 } from "../icons/droplet-off";
import { default as default476 } from "../icons/droplet";
import { default as default477 } from "../icons/droplets";
import { default as default478 } from "../icons/drum";
import { default as default479 } from "../icons/drumstick";
import { default as default480 } from "../icons/dumbbell";
import { default as default481 } from "../icons/ear-off";
import { default as default482 } from "../icons/ear";
import { default as default483 } from "../icons/earth-lock";
import { default as default484 } from "../icons/eclipse";
import { default as default485 } from "../icons/egg-fried";
import { default as default486 } from "../icons/egg-off";
import { default as default487 } from "../icons/egg";
import { default as default488 } from "../icons/ellipse";
import { default as default489 } from "../icons/equal-approximately";
import { default as default490 } from "../icons/equal-not";
import { default as default491 } from "../icons/equal";
import { default as default492 } from "../icons/eraser";
import { default as default493 } from "../icons/ethernet-port";
import { default as default494 } from "../icons/euro";
import { default as default495 } from "../icons/ev-charger";
import { default as default496 } from "../icons/expand";
import { default as default497 } from "../icons/external-link";
import { default as default498 } from "../icons/eye-closed";
import { default as default499 } from "../icons/eye-off";
import { default as default500 } from "../icons/eye";
import { default as default501 } from "../icons/factory";
import { default as default502 } from "../icons/fan";
import { default as default503 } from "../icons/fast-forward";
import { default as default504 } from "../icons/feather";
import { default as default505 } from "../icons/fence";
import { default as default506 } from "../icons/ferris-wheel";
import { default as default507 } from "../icons/file-archive";
import { default as default508 } from "../icons/file-box";
import { default as default509 } from "../icons/file-clock";
import { default as default510 } from "../icons/file-check";
import { default as default511 } from "../icons/file-code";
import { default as default512 } from "../icons/file-diff";
import { default as default513 } from "../icons/file-digit";
import { default as default514 } from "../icons/file-down";
import { default as default515 } from "../icons/file-heart";
import { default as default516 } from "../icons/file-image";
import { default as default517 } from "../icons/file-input";
import { default as default518 } from "../icons/file-minus";
import { default as default519 } from "../icons/file-music";
import { default as default520 } from "../icons/file-output";
import { default as default521 } from "../icons/file-plus";
import { default as default522 } from "../icons/file-scan";
import { default as default523 } from "../icons/file-search";
import { default as default524 } from "../icons/file-sliders";
import { default as default525 } from "../icons/file-spreadsheet";
import { default as default526 } from "../icons/file-stack";
import { default as default527 } from "../icons/file-symlink";
import { default as default528 } from "../icons/file-terminal";
import { default as default529 } from "../icons/file-text";
import { default as default530 } from "../icons/file-type";
import { default as default531 } from "../icons/file-up";
import { default as default532 } from "../icons/file-volume";
import { default as default533 } from "../icons/file-user";
import { default as default534 } from "../icons/file-x";
import { default as default535 } from "../icons/file";
import { default as default536 } from "../icons/files";
import { default as default537 } from "../icons/film";
import { default as default538 } from "../icons/fish-symbol";
import { default as default539 } from "../icons/fire-extinguisher";
import { default as default540 } from "../icons/fish-off";
import { default as default541 } from "../icons/fish";
import { default as default542 } from "../icons/fishing-hook";
import { default as default543 } from "../icons/fishing-rod";
import { default as default544 } from "../icons/flag-off";
import { default as default545 } from "../icons/flag-triangle-left";
import { default as default546 } from "../icons/flag-triangle-right";
import { default as default547 } from "../icons/flag";
import { default as default548 } from "../icons/flame-kindling";
import { default as default549 } from "../icons/flame";
import { default as default550 } from "../icons/flashlight-off";
import { default as default551 } from "../icons/flashlight";
import { default as default552 } from "../icons/flask-conical";
import { default as default553 } from "../icons/flask-conical-off";
import { default as default554 } from "../icons/flask-round";
import { default as default555 } from "../icons/flip-horizontal-2";
import { default as default556 } from "../icons/flip-vertical-2";
import { default as default557 } from "../icons/flower-2";
import { default as default558 } from "../icons/flower";
import { default as default559 } from "../icons/focus";
import { default as default560 } from "../icons/fold-horizontal";
import { default as default561 } from "../icons/fold-vertical";
import { default as default562 } from "../icons/folder-archive";
import { default as default563 } from "../icons/folder-check";
import { default as default564 } from "../icons/folder-clock";
import { default as default565 } from "../icons/folder-closed";
import { default as default566 } from "../icons/folder-code";
import { default as default567 } from "../icons/folder-dot";
import { default as default568 } from "../icons/folder-down";
import { default as default569 } from "../icons/folder-git-2";
import { default as default570 } from "../icons/folder-git";
import { default as default571 } from "../icons/folder-heart";
import { default as default572 } from "../icons/folder-input";
import { default as default573 } from "../icons/folder-kanban";
import { default as default574 } from "../icons/folder-key";
import { default as default575 } from "../icons/folder-minus";
import { default as default576 } from "../icons/folder-lock";
import { default as default577 } from "../icons/folder-open-dot";
import { default as default578 } from "../icons/folder-open";
import { default as default579 } from "../icons/folder-output";
import { default as default580 } from "../icons/folder-plus";
import { default as default581 } from "../icons/folder-root";
import { default as default582 } from "../icons/folder-search-2";
import { default as default583 } from "../icons/folder-search";
import { default as default584 } from "../icons/folder-symlink";
import { default as default585 } from "../icons/folder-sync";
import { default as default586 } from "../icons/folder-tree";
import { default as default587 } from "../icons/folder-up";
import { default as default588 } from "../icons/folder-x";
import { default as default589 } from "../icons/folder";
import { default as default590 } from "../icons/folders";
import { default as default591 } from "../icons/footprints";
import { default as default592 } from "../icons/forklift";
import { default as default593 } from "../icons/form";
import { default as default594 } from "../icons/forward";
import { default as default595 } from "../icons/frame";
import { default as default596 } from "../icons/frown";
import { default as default597 } from "../icons/fuel";
import { default as default598 } from "../icons/fullscreen";
import { default as default599 } from "../icons/funnel-plus";
import { default as default600 } from "../icons/gallery-horizontal-end";
import { default as default601 } from "../icons/gallery-horizontal";
import { default as default602 } from "../icons/gallery-thumbnails";
import { default as default603 } from "../icons/gallery-vertical";
import { default as default604 } from "../icons/gallery-vertical-end";
import { default as default605 } from "../icons/gamepad-2";
import { default as default606 } from "../icons/gamepad-directional";
import { default as default607 } from "../icons/gamepad";
import { default as default608 } from "../icons/gauge";
import { default as default609 } from "../icons/gavel";
import { default as default610 } from "../icons/gem";
import { default as default611 } from "../icons/georgian-lari";
import { default as default612 } from "../icons/ghost";
import { default as default613 } from "../icons/gift";
import { default as default614 } from "../icons/git-branch-minus";
import { default as default615 } from "../icons/git-branch-plus";
import { default as default616 } from "../icons/git-branch";
import { default as default617 } from "../icons/git-commit-vertical";
import { default as default618 } from "../icons/git-compare-arrows";
import { default as default619 } from "../icons/git-compare";
import { default as default620 } from "../icons/git-fork";
import { default as default621 } from "../icons/git-graph";
import { default as default622 } from "../icons/git-merge-conflict";
import { default as default623 } from "../icons/git-merge";
import { default as default624 } from "../icons/git-pull-request-arrow";
import { default as default625 } from "../icons/git-pull-request-create-arrow";
import { default as default626 } from "../icons/git-pull-request-closed";
import { default as default627 } from "../icons/git-pull-request-create";
import { default as default628 } from "../icons/git-pull-request-draft";
import { default as default629 } from "../icons/git-pull-request";
import { default as default630 } from "../icons/glass-water";
import { default as default631 } from "../icons/glasses";
import { default as default632 } from "../icons/globe-lock";
import { default as default633 } from "../icons/globe-off";
import { default as default634 } from "../icons/globe-x";
import { default as default635 } from "../icons/globe";
import { default as default636 } from "../icons/goal";
import { default as default637 } from "../icons/gpu";
import { default as default638 } from "../icons/graduation-cap";
import { default as default639 } from "../icons/grape";
import { default as default640 } from "../icons/grid-3x2";
import { default as default641 } from "../icons/grip-horizontal";
import { default as default642 } from "../icons/grip-vertical";
import { default as default643 } from "../icons/grip";
import { default as default644 } from "../icons/group";
import { default as default645 } from "../icons/guitar";
import { default as default646 } from "../icons/ham";
import { default as default647 } from "../icons/hammer";
import { default as default648 } from "../icons/hand-coins";
import { default as default649 } from "../icons/hamburger";
import { default as default650 } from "../icons/hand-fist";
import { default as default651 } from "../icons/hand-heart";
import { default as default652 } from "../icons/hand-metal";
import { default as default653 } from "../icons/hand-platter";
import { default as default654 } from "../icons/hand";
import { default as default655 } from "../icons/handbag";
import { default as default656 } from "../icons/handshake";
import { default as default657 } from "../icons/hard-drive-download";
import { default as default658 } from "../icons/hard-drive-upload";
import { default as default659 } from "../icons/hard-drive";
import { default as default660 } from "../icons/hard-hat";
import { default as default661 } from "../icons/hash";
import { default as default662 } from "../icons/hat-glasses";
import { default as default663 } from "../icons/hd";
import { default as default664 } from "../icons/haze";
import { default as default665 } from "../icons/hdmi-port";
import { default as default666 } from "../icons/heading-1";
import { default as default667 } from "../icons/heading-2";
import { default as default668 } from "../icons/heading-4";
import { default as default669 } from "../icons/heading-3";
import { default as default670 } from "../icons/heading-5";
import { default as default671 } from "../icons/heading-6";
import { default as default672 } from "../icons/heading";
import { default as default673 } from "../icons/headphone-off";
import { default as default674 } from "../icons/headphones";
import { default as default675 } from "../icons/headset";
import { default as default676 } from "../icons/heart-crack";
import { default as default677 } from "../icons/heart-handshake";
import { default as default678 } from "../icons/heart-minus";
import { default as default679 } from "../icons/heart-off";
import { default as default680 } from "../icons/heart-plus";
import { default as default681 } from "../icons/heart-pulse";
import { default as default682 } from "../icons/heart-x";
import { default as default683 } from "../icons/heart";
import { default as default684 } from "../icons/heater";
import { default as default685 } from "../icons/helicopter";
import { default as default686 } from "../icons/hexagon";
import { default as default687 } from "../icons/highlighter";
import { default as default688 } from "../icons/history";
import { default as default689 } from "../icons/hop-off";
import { default as default690 } from "../icons/hop";
import { default as default691 } from "../icons/hospital";
import { default as default692 } from "../icons/hotel";
import { default as default693 } from "../icons/hourglass";
import { default as default694 } from "../icons/house-heart";
import { default as default695 } from "../icons/house-plus";
import { default as default696 } from "../icons/house-wifi";
import { default as default697 } from "../icons/house-plug";
import { default as default698 } from "../icons/id-card-lanyard";
import { default as default699 } from "../icons/id-card";
import { default as default700 } from "../icons/image-down";
import { default as default701 } from "../icons/image-minus";
import { default as default702 } from "../icons/image-off";
import { default as default703 } from "../icons/image-play";
import { default as default704 } from "../icons/image-plus";
import { default as default705 } from "../icons/image-up";
import { default as default706 } from "../icons/image-upscale";
import { default as default707 } from "../icons/image";
import { default as default708 } from "../icons/images";
import { default as default709 } from "../icons/import";
import { default as default710 } from "../icons/inbox";
import { default as default711 } from "../icons/indian-rupee";
import { default as default712 } from "../icons/infinity";
import { default as default713 } from "../icons/info";
import { default as default714 } from "../icons/italic";
import { default as default715 } from "../icons/inspection-panel";
import { default as default716 } from "../icons/iteration-ccw";
import { default as default717 } from "../icons/iteration-cw";
import { default as default718 } from "../icons/japanese-yen";
import { default as default719 } from "../icons/joystick";
import { default as default720 } from "../icons/kanban";
import { default as default721 } from "../icons/kayak";
import { default as default722 } from "../icons/key-round";
import { default as default723 } from "../icons/key-square";
import { default as default724 } from "../icons/key";
import { default as default725 } from "../icons/keyboard-music";
import { default as default726 } from "../icons/keyboard-off";
import { default as default727 } from "../icons/keyboard";
import { default as default728 } from "../icons/lamp-ceiling";
import { default as default729 } from "../icons/lamp-floor";
import { default as default730 } from "../icons/lamp-wall-down";
import { default as default731 } from "../icons/lamp-desk";
import { default as default732 } from "../icons/lamp-wall-up";
import { default as default733 } from "../icons/lamp";
import { default as default734 } from "../icons/land-plot";
import { default as default735 } from "../icons/landmark";
import { default as default736 } from "../icons/languages";
import { default as default737 } from "../icons/laptop-minimal-check";
import { default as default738 } from "../icons/laptop";
import { default as default739 } from "../icons/lasso-select";
import { default as default740 } from "../icons/lasso";
import { default as default741 } from "../icons/laugh";
import { default as default742 } from "../icons/layers-2";
import { default as default743 } from "../icons/layers-minus";
import { default as default744 } from "../icons/layers-plus";
import { default as default745 } from "../icons/layout-dashboard";
import { default as default746 } from "../icons/layout-grid";
import { default as default747 } from "../icons/layout-list";
import { default as default748 } from "../icons/layout-panel-left";
import { default as default749 } from "../icons/layout-panel-top";
import { default as default750 } from "../icons/layout-template";
import { default as default751 } from "../icons/leaf";
import { default as default752 } from "../icons/lectern";
import { default as default753 } from "../icons/leafy-green";
import { default as default754 } from "../icons/lens-concave";
import { default as default755 } from "../icons/library-big";
import { default as default756 } from "../icons/lens-convex";
import { default as default757 } from "../icons/library";
import { default as default758 } from "../icons/life-buoy";
import { default as default759 } from "../icons/ligature";
import { default as default760 } from "../icons/lightbulb-off";
import { default as default761 } from "../icons/lightbulb";
import { default as default762 } from "../icons/line-dot-right-horizontal";
import { default as default763 } from "../icons/line-squiggle";
import { default as default764 } from "../icons/line-style";
import { default as default765 } from "../icons/link-2-off";
import { default as default766 } from "../icons/link-2";
import { default as default767 } from "../icons/link";
import { default as default768 } from "../icons/list-check";
import { default as default769 } from "../icons/list-checks";
import { default as default770 } from "../icons/list-chevrons-down-up";
import { default as default771 } from "../icons/list-chevrons-up-down";
import { default as default772 } from "../icons/list-collapse";
import { default as default773 } from "../icons/list-end";
import { default as default774 } from "../icons/list-filter-plus";
import { default as default775 } from "../icons/list-filter";
import { default as default776 } from "../icons/list-minus";
import { default as default777 } from "../icons/list-music";
import { default as default778 } from "../icons/list-ordered";
import { default as default779 } from "../icons/list-plus";
import { default as default780 } from "../icons/list-restart";
import { default as default781 } from "../icons/list-start";
import { default as default782 } from "../icons/list-todo";
import { default as default783 } from "../icons/list-tree";
import { default as default784 } from "../icons/list-video";
import { default as default785 } from "../icons/list-x";
import { default as default786 } from "../icons/list";
import { default as default787 } from "../icons/loader-pinwheel";
import { default as default788 } from "../icons/loader";
import { default as default789 } from "../icons/locate-fixed";
import { default as default790 } from "../icons/locate-off";
import { default as default791 } from "../icons/locate";
import { default as default792 } from "../icons/lock-keyhole";
import { default as default793 } from "../icons/lock";
import { default as default794 } from "../icons/log-in";
import { default as default795 } from "../icons/log-out";
import { default as default796 } from "../icons/logs";
import { default as default797 } from "../icons/lollipop";
import { default as default798 } from "../icons/luggage";
import { default as default799 } from "../icons/magnet";
import { default as default800 } from "../icons/mail-check";
import { default as default801 } from "../icons/mail-minus";
import { default as default802 } from "../icons/mail-plus";
import { default as default803 } from "../icons/mail-search";
import { default as default804 } from "../icons/mail-warning";
import { default as default805 } from "../icons/mail-open";
import { default as default806 } from "../icons/mail-x";
import { default as default807 } from "../icons/mail";
import { default as default808 } from "../icons/mailbox";
import { default as default809 } from "../icons/mails";
import { default as default810 } from "../icons/map-pin-check-inside";
import { default as default811 } from "../icons/map-minus";
import { default as default812 } from "../icons/map-pin-check";
import { default as default813 } from "../icons/map-pin-house";
import { default as default814 } from "../icons/map-pin-minus-inside";
import { default as default815 } from "../icons/map-pin-minus";
import { default as default816 } from "../icons/map-pin-off";
import { default as default817 } from "../icons/map-pin-plus-inside";
import { default as default818 } from "../icons/map-pin-plus";
import { default as default819 } from "../icons/map-pin-search";
import { default as default820 } from "../icons/map-pin-x-inside";
import { default as default821 } from "../icons/map-pin-x";
import { default as default822 } from "../icons/map-pin";
import { default as default823 } from "../icons/map-pinned";
import { default as default824 } from "../icons/map-plus";
import { default as default825 } from "../icons/map";
import { default as default826 } from "../icons/mars-stroke";
import { default as default827 } from "../icons/martini";
import { default as default828 } from "../icons/mars";
import { default as default829 } from "../icons/maximize-2";
import { default as default830 } from "../icons/megaphone-off";
import { default as default831 } from "../icons/medal";
import { default as default832 } from "../icons/maximize";
import { default as default833 } from "../icons/megaphone";
import { default as default834 } from "../icons/meh";
import { default as default835 } from "../icons/memory-stick";
import { default as default836 } from "../icons/menu";
import { default as default837 } from "../icons/merge";
import { default as default838 } from "../icons/message-circle-check";
import { default as default839 } from "../icons/message-circle-code";
import { default as default840 } from "../icons/message-circle-dashed";
import { default as default841 } from "../icons/message-circle-heart";
import { default as default842 } from "../icons/message-circle-more";
import { default as default843 } from "../icons/message-circle-off";
import { default as default844 } from "../icons/message-circle-plus";
import { default as default845 } from "../icons/message-circle-reply";
import { default as default846 } from "../icons/message-circle-warning";
import { default as default847 } from "../icons/message-circle-x";
import { default as default848 } from "../icons/message-square-check";
import { default as default849 } from "../icons/message-circle";
import { default as default850 } from "../icons/message-square-code";
import { default as default851 } from "../icons/message-square-dashed";
import { default as default852 } from "../icons/message-square-diff";
import { default as default853 } from "../icons/message-square-dot";
import { default as default854 } from "../icons/message-square-heart";
import { default as default855 } from "../icons/message-square-lock";
import { default as default856 } from "../icons/message-square-more";
import { default as default857 } from "../icons/message-square-off";
import { default as default858 } from "../icons/message-square-plus";
import { default as default859 } from "../icons/message-square-quote";
import { default as default860 } from "../icons/message-square-reply";
import { default as default861 } from "../icons/message-square-share";
import { default as default862 } from "../icons/message-square-text";
import { default as default863 } from "../icons/message-square-warning";
import { default as default864 } from "../icons/message-square-x";
import { default as default865 } from "../icons/message-square";
import { default as default866 } from "../icons/messages-square";
import { default as default867 } from "../icons/metronome";
import { default as default868 } from "../icons/mic-off";
import { default as default869 } from "../icons/mic";
import { default as default870 } from "../icons/microchip";
import { default as default871 } from "../icons/microscope";
import { default as default872 } from "../icons/microwave";
import { default as default873 } from "../icons/milestone";
import { default as default874 } from "../icons/milk-off";
import { default as default875 } from "../icons/milk";
import { default as default876 } from "../icons/minimize-2";
import { default as default877 } from "../icons/minimize";
import { default as default878 } from "../icons/minus";
import { default as default879 } from "../icons/mirror-rectangular";
import { default as default880 } from "../icons/mirror-round";
import { default as default881 } from "../icons/monitor-check";
import { default as default882 } from "../icons/monitor-cloud";
import { default as default883 } from "../icons/monitor-cog";
import { default as default884 } from "../icons/monitor-dot";
import { default as default885 } from "../icons/monitor-down";
import { default as default886 } from "../icons/monitor-off";
import { default as default887 } from "../icons/monitor-pause";
import { default as default888 } from "../icons/monitor-play";
import { default as default889 } from "../icons/monitor-smartphone";
import { default as default890 } from "../icons/monitor-speaker";
import { default as default891 } from "../icons/monitor-stop";
import { default as default892 } from "../icons/monitor-up";
import { default as default893 } from "../icons/monitor-x";
import { default as default894 } from "../icons/monitor";
import { default as default895 } from "../icons/moon-star";
import { default as default896 } from "../icons/moon";
import { default as default897 } from "../icons/motorbike";
import { default as default898 } from "../icons/mountain-snow";
import { default as default899 } from "../icons/mountain";
import { default as default900 } from "../icons/mouse-left";
import { default as default901 } from "../icons/mouse-off";
import { default as default902 } from "../icons/mouse-pointer-2-off";
import { default as default903 } from "../icons/mouse-pointer-2";
import { default as default904 } from "../icons/mouse-pointer-ban";
import { default as default905 } from "../icons/mouse-pointer-click";
import { default as default906 } from "../icons/mouse-pointer";
import { default as default907 } from "../icons/mouse-right";
import { default as default908 } from "../icons/mouse";
import { default as default909 } from "../icons/move-diagonal-2";
import { default as default910 } from "../icons/move-diagonal";
import { default as default911 } from "../icons/move-down-left";
import { default as default912 } from "../icons/move-down-right";
import { default as default913 } from "../icons/move-down";
import { default as default914 } from "../icons/move-horizontal";
import { default as default915 } from "../icons/move-left";
import { default as default916 } from "../icons/move-right";
import { default as default917 } from "../icons/move-up-left";
import { default as default918 } from "../icons/move-up-right";
import { default as default919 } from "../icons/move-up";
import { default as default920 } from "../icons/move-vertical";
import { default as default921 } from "../icons/move";
import { default as default922 } from "../icons/music-2";
import { default as default923 } from "../icons/music-3";
import { default as default924 } from "../icons/music-4";
import { default as default925 } from "../icons/navigation-2";
import { default as default926 } from "../icons/navigation-2-off";
import { default as default927 } from "../icons/music";
import { default as default928 } from "../icons/navigation-off";
import { default as default929 } from "../icons/navigation";
import { default as default930 } from "../icons/newspaper";
import { default as default931 } from "../icons/network";
import { default as default932 } from "../icons/nfc";
import { default as default933 } from "../icons/non-binary";
import { default as default934 } from "../icons/notebook-pen";
import { default as default935 } from "../icons/notebook-tabs";
import { default as default936 } from "../icons/notebook-text";
import { default as default937 } from "../icons/notebook";
import { default as default938 } from "../icons/notepad-text-dashed";
import { default as default939 } from "../icons/notepad-text";
import { default as default940 } from "../icons/nut-off";
import { default as default941 } from "../icons/octagon-minus";
import { default as default942 } from "../icons/nut";
import { default as default943 } from "../icons/option";
import { default as default944 } from "../icons/omega";
import { default as default945 } from "../icons/orbit";
import { default as default946 } from "../icons/origami";
import { default as default947 } from "../icons/package-2";
import { default as default948 } from "../icons/package-check";
import { default as default949 } from "../icons/package-minus";
import { default as default950 } from "../icons/package-open";
import { default as default951 } from "../icons/package-plus";
import { default as default952 } from "../icons/package-search";
import { default as default953 } from "../icons/package-x";
import { default as default954 } from "../icons/paint-bucket";
import { default as default955 } from "../icons/package";
import { default as default956 } from "../icons/paint-roller";
import { default as default957 } from "../icons/paintbrush";
import { default as default958 } from "../icons/palette";
import { default as default959 } from "../icons/panda";
import { default as default960 } from "../icons/panel-bottom-open";
import { default as default961 } from "../icons/panel-bottom-close";
import { default as default962 } from "../icons/panel-bottom";
import { default as default963 } from "../icons/panel-left-right-dashed";
import { default as default964 } from "../icons/panel-right-close";
import { default as default965 } from "../icons/panel-right-open";
import { default as default966 } from "../icons/panel-right";
import { default as default967 } from "../icons/panel-top-bottom-dashed";
import { default as default968 } from "../icons/panel-top-close";
import { default as default969 } from "../icons/panel-top-open";
import { default as default970 } from "../icons/panel-top";
import { default as default971 } from "../icons/panels-left-bottom";
import { default as default972 } from "../icons/panels-right-bottom";
import { default as default973 } from "../icons/paperclip";
import { default as default974 } from "../icons/parentheses";
import { default as default975 } from "../icons/parking-meter";
import { default as default976 } from "../icons/party-popper";
import { default as default977 } from "../icons/paw-print";
import { default as default978 } from "../icons/pause";
import { default as default979 } from "../icons/pc-case";
import { default as default980 } from "../icons/pen-off";
import { default as default981 } from "../icons/pencil-line";
import { default as default982 } from "../icons/pen-tool";
import { default as default983 } from "../icons/pencil-off";
import { default as default984 } from "../icons/pencil-ruler";
import { default as default985 } from "../icons/pencil";
import { default as default986 } from "../icons/pentagon";
import { default as default987 } from "../icons/percent";
import { default as default988 } from "../icons/person-standing";
import { default as default989 } from "../icons/philippine-peso";
import { default as default990 } from "../icons/phone-call";
import { default as default991 } from "../icons/phone-forwarded";
import { default as default992 } from "../icons/phone-incoming";
import { default as default993 } from "../icons/phone-missed";
import { default as default994 } from "../icons/phone-off";
import { default as default995 } from "../icons/phone-outgoing";
import { default as default996 } from "../icons/phone";
import { default as default997 } from "../icons/pi";
import { default as default998 } from "../icons/piano";
import { default as default999 } from "../icons/pickaxe";
import { default as default1000 } from "../icons/picture-in-picture-2";
import { default as default1001 } from "../icons/picture-in-picture";
import { default as default1002 } from "../icons/piggy-bank";
import { default as default1003 } from "../icons/pilcrow-left";
import { default as default1004 } from "../icons/pilcrow-right";
import { default as default1005 } from "../icons/pilcrow";
import { default as default1006 } from "../icons/pill-bottle";
import { default as default1007 } from "../icons/pill";
import { default as default1008 } from "../icons/pin";
import { default as default1009 } from "../icons/pipette";
import { default as default1010 } from "../icons/pizza";
import { default as default1011 } from "../icons/pin-off";
import { default as default1012 } from "../icons/plane-landing";
import { default as default1013 } from "../icons/plane-takeoff";
import { default as default1014 } from "../icons/plane";
import { default as default1015 } from "../icons/plug-2";
import { default as default1016 } from "../icons/plug";
import { default as default1017 } from "../icons/play";
import { default as default1018 } from "../icons/plus";
import { default as default1019 } from "../icons/pocket-knife";
import { default as default1020 } from "../icons/podcast";
import { default as default1021 } from "../icons/pointer-off";
import { default as default1022 } from "../icons/pointer";
import { default as default1023 } from "../icons/popcorn";
import { default as default1024 } from "../icons/popsicle";
import { default as default1025 } from "../icons/pound-sterling";
import { default as default1026 } from "../icons/power-off";
import { default as default1027 } from "../icons/power";
import { default as default1028 } from "../icons/printer-check";
import { default as default1029 } from "../icons/presentation";
import { default as default1030 } from "../icons/printer-x";
import { default as default1031 } from "../icons/projector";
import { default as default1032 } from "../icons/printer";
import { default as default1033 } from "../icons/proportions";
import { default as default1034 } from "../icons/puzzle";
import { default as default1035 } from "../icons/pyramid";
import { default as default1036 } from "../icons/qr-code";
import { default as default1037 } from "../icons/quote";
import { default as default1038 } from "../icons/rabbit";
import { default as default1039 } from "../icons/radar";
import { default as default1040 } from "../icons/radiation";
import { default as default1041 } from "../icons/radical";
import { default as default1042 } from "../icons/radio-off";
import { default as default1043 } from "../icons/radio-receiver";
import { default as default1044 } from "../icons/radio-tower";
import { default as default1045 } from "../icons/radio";
import { default as default1046 } from "../icons/radius";
import { default as default1047 } from "../icons/rainbow";
import { default as default1048 } from "../icons/rat";
import { default as default1049 } from "../icons/ratio";
import { default as default1050 } from "../icons/receipt-cent";
import { default as default1051 } from "../icons/receipt-euro";
import { default as default1052 } from "../icons/receipt-indian-rupee";
import { default as default1053 } from "../icons/receipt-japanese-yen";
import { default as default1054 } from "../icons/receipt-pound-sterling";
import { default as default1055 } from "../icons/receipt-russian-ruble";
import { default as default1056 } from "../icons/receipt-swiss-franc";
import { default as default1057 } from "../icons/receipt-text";
import { default as default1058 } from "../icons/receipt-turkish-lira";
import { default as default1059 } from "../icons/receipt";
import { default as default1060 } from "../icons/octagon";
import { default as default1061 } from "../icons/rectangle-circle";
import { default as default1062 } from "../icons/rectangle-goggles";
import { default as default1063 } from "../icons/rectangle-horizontal";
import { default as default1064 } from "../icons/rectangle-vertical";
import { default as default1065 } from "../icons/recycle";
import { default as default1066 } from "../icons/redo-2";
import { default as default1067 } from "../icons/redo-dot";
import { default as default1068 } from "../icons/redo";
import { default as default1069 } from "../icons/refresh-ccw";
import { default as default1070 } from "../icons/refresh-ccw-dot";
import { default as default1071 } from "../icons/refresh-cw-off";
import { default as default1072 } from "../icons/refresh-cw";
import { default as default1073 } from "../icons/refrigerator";
import { default as default1074 } from "../icons/regex";
import { default as default1075 } from "../icons/remove-formatting";
import { default as default1076 } from "../icons/repeat-1";
import { default as default1077 } from "../icons/repeat-2";
import { default as default1078 } from "../icons/repeat";
import { default as default1079 } from "../icons/replace-all";
import { default as default1080 } from "../icons/replace";
import { default as default1081 } from "../icons/reply-all";
import { default as default1082 } from "../icons/reply";
import { default as default1083 } from "../icons/rewind";
import { default as default1084 } from "../icons/ribbon";
import { default as default1085 } from "../icons/road";
import { default as default1086 } from "../icons/rocket";
import { default as default1087 } from "../icons/rocking-chair";
import { default as default1088 } from "../icons/roller-coaster";
import { default as default1089 } from "../icons/rose";
import { default as default1090 } from "../icons/rotate-ccw-key";
import { default as default1091 } from "../icons/rotate-ccw-square";
import { default as default1092 } from "../icons/rotate-ccw";
import { default as default1093 } from "../icons/rotate-cw-square";
import { default as default1094 } from "../icons/rotate-cw";
import { default as default1095 } from "../icons/route-off";
import { default as default1096 } from "../icons/route";
import { default as default1097 } from "../icons/router";
import { default as default1098 } from "../icons/rows-4";
import { default as default1099 } from "../icons/rss";
import { default as default1100 } from "../icons/ruler-dimension-line";
import { default as default1101 } from "../icons/ruler";
import { default as default1102 } from "../icons/russian-ruble";
import { default as default1103 } from "../icons/sailboat";
import { default as default1104 } from "../icons/salad";
import { default as default1105 } from "../icons/sandwich";
import { default as default1106 } from "../icons/satellite-dish";
import { default as default1107 } from "../icons/satellite";
import { default as default1108 } from "../icons/saudi-riyal";
import { default as default1109 } from "../icons/save-all";
import { default as default1110 } from "../icons/save-off";
import { default as default1111 } from "../icons/save";
import { default as default1112 } from "../icons/scale";
import { default as default1113 } from "../icons/scaling";
import { default as default1114 } from "../icons/scan-barcode";
import { default as default1115 } from "../icons/scan-face";
import { default as default1116 } from "../icons/scan-eye";
import { default as default1117 } from "../icons/scan-heart";
import { default as default1118 } from "../icons/scan-line";
import { default as default1119 } from "../icons/scan-qr-code";
import { default as default1120 } from "../icons/scan-search";
import { default as default1121 } from "../icons/scan-text";
import { default as default1122 } from "../icons/scan";
import { default as default1123 } from "../icons/school";
import { default as default1124 } from "../icons/scissors-line-dashed";
import { default as default1125 } from "../icons/scissors";
import { default as default1126 } from "../icons/scooter";
import { default as default1127 } from "../icons/screen-share-off";
import { default as default1128 } from "../icons/screen-share";
import { default as default1129 } from "../icons/scroll-text";
import { default as default1130 } from "../icons/scroll";
import { default as default1131 } from "../icons/search-alert";
import { default as default1132 } from "../icons/search-check";
import { default as default1133 } from "../icons/search-code";
import { default as default1134 } from "../icons/search-slash";
import { default as default1135 } from "../icons/search-x";
import { default as default1136 } from "../icons/search";
import { default as default1137 } from "../icons/section";
import { default as default1138 } from "../icons/send-to-back";
import { default as default1139 } from "../icons/send";
import { default as default1140 } from "../icons/separator-horizontal";
import { default as default1141 } from "../icons/server-cog";
import { default as default1142 } from "../icons/separator-vertical";
import { default as default1143 } from "../icons/server-crash";
import { default as default1144 } from "../icons/server-off";
import { default as default1145 } from "../icons/server";
import { default as default1146 } from "../icons/settings-2";
import { default as default1147 } from "../icons/settings";
import { default as default1148 } from "../icons/share";
import { default as default1149 } from "../icons/share-2";
import { default as default1150 } from "../icons/shapes";
import { default as default1151 } from "../icons/shell";
import { default as default1152 } from "../icons/sheet";
import { default as default1153 } from "../icons/shelving-unit";
import { default as default1154 } from "../icons/shield-alert";
import { default as default1155 } from "../icons/shield-ban";
import { default as default1156 } from "../icons/shield-check";
import { default as default1157 } from "../icons/shield-cog-corner";
import { default as default1158 } from "../icons/shield-cog";
import { default as default1159 } from "../icons/shield-ellipsis";
import { default as default1160 } from "../icons/shield-half";
import { default as default1161 } from "../icons/shield-minus";
import { default as default1162 } from "../icons/shield-off";
import { default as default1163 } from "../icons/shield-plus";
import { default as default1164 } from "../icons/shield-user";
import { default as default1165 } from "../icons/shield";
import { default as default1166 } from "../icons/ship-wheel";
import { default as default1167 } from "../icons/ship";
import { default as default1168 } from "../icons/shirt";
import { default as default1169 } from "../icons/shopping-bag";
import { default as default1170 } from "../icons/shopping-basket";
import { default as default1171 } from "../icons/shopping-cart";
import { default as default1172 } from "../icons/shovel";
import { default as default1173 } from "../icons/shower-head";
import { default as default1174 } from "../icons/shredder";
import { default as default1175 } from "../icons/shrimp";
import { default as default1176 } from "../icons/shrink";
import { default as default1177 } from "../icons/shrub";
import { default as default1178 } from "../icons/shuffle";
import { default as default1179 } from "../icons/sigma";
import { default as default1180 } from "../icons/signal-high";
import { default as default1181 } from "../icons/signal-low";
import { default as default1182 } from "../icons/signal-medium";
import { default as default1183 } from "../icons/signal-zero";
import { default as default1184 } from "../icons/signal";
import { default as default1185 } from "../icons/signature";
import { default as default1186 } from "../icons/signpost-big";
import { default as default1187 } from "../icons/signpost";
import { default as default1188 } from "../icons/siren";
import { default as default1189 } from "../icons/skip-back";
import { default as default1190 } from "../icons/skip-forward";
import { default as default1191 } from "../icons/skull";
import { default as default1192 } from "../icons/slash";
import { default as default1193 } from "../icons/slice";
import { default as default1194 } from "../icons/sliders-horizontal";
import { default as default1195 } from "../icons/smartphone-charging";
import { default as default1196 } from "../icons/smartphone-nfc";
import { default as default1197 } from "../icons/smartphone";
import { default as default1198 } from "../icons/smile-plus";
import { default as default1199 } from "../icons/smile";
import { default as default1200 } from "../icons/snail";
import { default as default1201 } from "../icons/snowflake";
import { default as default1202 } from "../icons/soap-dispenser-droplet";
import { default as default1203 } from "../icons/sofa";
import { default as default1204 } from "../icons/solar-panel";
import { default as default1205 } from "../icons/soup";
import { default as default1206 } from "../icons/space";
import { default as default1207 } from "../icons/spade";
import { default as default1208 } from "../icons/sparkle";
import { default as default1209 } from "../icons/speaker";
import { default as default1210 } from "../icons/speech";
import { default as default1211 } from "../icons/spell-check-2";
import { default as default1212 } from "../icons/spell-check";
import { default as default1213 } from "../icons/spline-pointer";
import { default as default1214 } from "../icons/spline";
import { default as default1215 } from "../icons/split";
import { default as default1216 } from "../icons/spool";
import { default as default1217 } from "../icons/sport-shoe";
import { default as default1218 } from "../icons/spotlight";
import { default as default1219 } from "../icons/spray-can";
import { default as default1220 } from "../icons/sprout";
import { default as default1221 } from "../icons/square-arrow-right-enter";
import { default as default1222 } from "../icons/square-arrow-right-exit";
import { default as default1223 } from "../icons/square-dashed-bottom-code";
import { default as default1224 } from "../icons/square-dashed-bottom";
import { default as default1225 } from "../icons/square-dashed-top-solid";
import { default as default1226 } from "../icons/square-pause";
import { default as default1227 } from "../icons/square-radical";
import { default as default1228 } from "../icons/square-round-corner";
import { default as default1229 } from "../icons/square-square";
import { default as default1230 } from "../icons/square-stack";
import { default as default1231 } from "../icons/square-star";
import { default as default1232 } from "../icons/square-stop";
import { default as default1233 } from "../icons/square";
import { default as default1234 } from "../icons/squares-exclude";
import { default as default1235 } from "../icons/squares-intersect";
import { default as default1236 } from "../icons/squares-subtract";
import { default as default1237 } from "../icons/squares-unite";
import { default as default1238 } from "../icons/squircle-dashed";
import { default as default1239 } from "../icons/squircle";
import { default as default1240 } from "../icons/squirrel";
import { default as default1241 } from "../icons/stamp";
import { default as default1242 } from "../icons/star-half";
import { default as default1243 } from "../icons/star-off";
import { default as default1244 } from "../icons/star";
import { default as default1245 } from "../icons/step-back";
import { default as default1246 } from "../icons/step-forward";
import { default as default1247 } from "../icons/sticker";
import { default as default1248 } from "../icons/stethoscope";
import { default as default1249 } from "../icons/sticky-note";
import { default as default1250 } from "../icons/store";
import { default as default1251 } from "../icons/stretch-horizontal";
import { default as default1252 } from "../icons/stone";
import { default as default1253 } from "../icons/stretch-vertical";
import { default as default1254 } from "../icons/strikethrough";
import { default as default1255 } from "../icons/subscript";
import { default as default1256 } from "../icons/sun-dim";
import { default as default1257 } from "../icons/sun-medium";
import { default as default1258 } from "../icons/sun-moon";
import { default as default1259 } from "../icons/sun-snow";
import { default as default1260 } from "../icons/sun";
import { default as default1261 } from "../icons/sunrise";
import { default as default1262 } from "../icons/sunset";
import { default as default1263 } from "../icons/superscript";
import { default as default1264 } from "../icons/swatch-book";
import { default as default1265 } from "../icons/swiss-franc";
import { default as default1266 } from "../icons/switch-camera";
import { default as default1267 } from "../icons/sword";
import { default as default1268 } from "../icons/swords";
import { default as default1269 } from "../icons/syringe";
import { default as default1270 } from "../icons/table-cells-merge";
import { default as default1271 } from "../icons/table-2";
import { default as default1272 } from "../icons/table-cells-split";
import { default as default1273 } from "../icons/table-columns-split";
import { default as default1274 } from "../icons/table-of-contents";
import { default as default1275 } from "../icons/table-properties";
import { default as default1276 } from "../icons/table-rows-split";
import { default as default1277 } from "../icons/table";
import { default as default1278 } from "../icons/tablet-smartphone";
import { default as default1279 } from "../icons/tablet";
import { default as default1280 } from "../icons/tablets";
import { default as default1281 } from "../icons/tag";
import { default as default1282 } from "../icons/tally-1";
import { default as default1283 } from "../icons/tags";
import { default as default1284 } from "../icons/tally-2";
import { default as default1285 } from "../icons/tally-3";
import { default as default1286 } from "../icons/tally-4";
import { default as default1287 } from "../icons/tally-5";
import { default as default1288 } from "../icons/tangent";
import { default as default1289 } from "../icons/target";
import { default as default1290 } from "../icons/telescope";
import { default as default1291 } from "../icons/tent-tree";
import { default as default1292 } from "../icons/tent";
import { default as default1293 } from "../icons/terminal";
import { default as default1294 } from "../icons/test-tube";
import { default as default1295 } from "../icons/test-tubes";
import { default as default1296 } from "../icons/text-cursor-input";
import { default as default1297 } from "../icons/text-cursor";
import { default as default1298 } from "../icons/text-quote";
import { default as default1299 } from "../icons/text-search";
import { default as default1300 } from "../icons/theater";
import { default as default1301 } from "../icons/thermometer-snowflake";
import { default as default1302 } from "../icons/thermometer-sun";
import { default as default1303 } from "../icons/thermometer";
import { default as default1304 } from "../icons/thumbs-down";
import { default as default1305 } from "../icons/thumbs-up";
import { default as default1306 } from "../icons/ticket-check";
import { default as default1307 } from "../icons/ticket-minus";
import { default as default1308 } from "../icons/ticket-percent";
import { default as default1309 } from "../icons/ticket-plus";
import { default as default1310 } from "../icons/ticket-slash";
import { default as default1311 } from "../icons/ticket-x";
import { default as default1312 } from "../icons/ticket";
import { default as default1313 } from "../icons/tickets-plane";
import { default as default1314 } from "../icons/tickets";
import { default as default1315 } from "../icons/timeline";
import { default as default1316 } from "../icons/timer-off";
import { default as default1317 } from "../icons/timer";
import { default as default1318 } from "../icons/timer-reset";
import { default as default1319 } from "../icons/toggle-left";
import { default as default1320 } from "../icons/toggle-right";
import { default as default1321 } from "../icons/toilet";
import { default as default1322 } from "../icons/tool-case";
import { default as default1323 } from "../icons/toolbox";
import { default as default1324 } from "../icons/tornado";
import { default as default1325 } from "../icons/torus";
import { default as default1326 } from "../icons/touchpad-off";
import { default as default1327 } from "../icons/touchpad";
import { default as default1328 } from "../icons/towel-rack";
import { default as default1329 } from "../icons/tower-control";
import { default as default1330 } from "../icons/toy-brick";
import { default as default1331 } from "../icons/tractor";
import { default as default1332 } from "../icons/train-front-tunnel";
import { default as default1333 } from "../icons/traffic-cone";
import { default as default1334 } from "../icons/train-front";
import { default as default1335 } from "../icons/train-track";
import { default as default1336 } from "../icons/transgender";
import { default as default1337 } from "../icons/trash-2";
import { default as default1338 } from "../icons/trash";
import { default as default1339 } from "../icons/tree-deciduous";
import { default as default1340 } from "../icons/tree-pine";
import { default as default1341 } from "../icons/trees";
import { default as default1342 } from "../icons/trending-down";
import { default as default1343 } from "../icons/trending-up-down";
import { default as default1344 } from "../icons/trending-up";
import { default as default1345 } from "../icons/triangle-dashed";
import { default as default1346 } from "../icons/triangle-right";
import { default as default1347 } from "../icons/triangle";
import { default as default1348 } from "../icons/trophy";
import { default as default1349 } from "../icons/truck-electric";
import { default as default1350 } from "../icons/truck";
import { default as default1351 } from "../icons/turkish-lira";
import { default as default1352 } from "../icons/turntable";
import { default as default1353 } from "../icons/turtle";
import { default as default1354 } from "../icons/tv-minimal-play";
import { default as default1355 } from "../icons/tv";
import { default as default1356 } from "../icons/type-outline";
import { default as default1357 } from "../icons/type";
import { default as default1358 } from "../icons/umbrella-off";
import { default as default1359 } from "../icons/umbrella";
import { default as default1360 } from "../icons/underline";
import { default as default1361 } from "../icons/undo-2";
import { default as default1362 } from "../icons/undo-dot";
import { default as default1363 } from "../icons/undo";
import { default as default1364 } from "../icons/unfold-horizontal";
import { default as default1365 } from "../icons/unfold-vertical";
import { default as default1366 } from "../icons/ungroup";
import { default as default1367 } from "../icons/unlink-2";
import { default as default1368 } from "../icons/unlink";
import { default as default1369 } from "../icons/unplug";
import { default as default1370 } from "../icons/upload";
import { default as default1371 } from "../icons/usb";
import { default as default1372 } from "../icons/user-check";
import { default as default1373 } from "../icons/user-cog";
import { default as default1374 } from "../icons/user-key";
import { default as default1375 } from "../icons/user-lock";
import { default as default1376 } from "../icons/user-minus";
import { default as default1377 } from "../icons/user-pen";
import { default as default1378 } from "../icons/user-plus";
import { default as default1379 } from "../icons/user-round-key";
import { default as default1380 } from "../icons/user-round-pen";
import { default as default1381 } from "../icons/user-round-search";
import { default as default1382 } from "../icons/user-search";
import { default as default1383 } from "../icons/user-star";
import { default as default1384 } from "../icons/user-x";
import { default as default1385 } from "../icons/user";
import { default as default1386 } from "../icons/users";
import { default as default1387 } from "../icons/utility-pole";
import { default as default1388 } from "../icons/van";
import { default as default1389 } from "../icons/variable";
import { default as default1390 } from "../icons/vault";
import { default as default1391 } from "../icons/vector-square";
import { default as default1392 } from "../icons/vegan";
import { default as default1393 } from "../icons/venetian-mask";
import { default as default1394 } from "../icons/venus-and-mars";
import { default as default1395 } from "../icons/venus";
import { default as default1396 } from "../icons/vibrate-off";
import { default as default1397 } from "../icons/vibrate";
import { default as default1398 } from "../icons/video-off";
import { default as default1399 } from "../icons/video";
import { default as default1400 } from "../icons/videotape";
import { default as default1401 } from "../icons/view";
import { default as default1402 } from "../icons/volleyball";
import { default as default1403 } from "../icons/voicemail";
import { default as default1404 } from "../icons/volume-1";
import { default as default1405 } from "../icons/volume-2";
import { default as default1406 } from "../icons/volume-off";
import { default as default1407 } from "../icons/volume-x";
import { default as default1408 } from "../icons/volume";
import { default as default1409 } from "../icons/vote";
import { default as default1410 } from "../icons/wallet-cards";
import { default as default1411 } from "../icons/wallet";
import { default as default1412 } from "../icons/wallpaper";
import { default as default1413 } from "../icons/wand";
import { default as default1414 } from "../icons/warehouse";
import { default as default1415 } from "../icons/washing-machine";
import { default as default1416 } from "../icons/watch";
import { default as default1417 } from "../icons/waves-arrow-down";
import { default as default1418 } from "../icons/waves-arrow-up";
import { default as default1419 } from "../icons/waves-ladder";
import { default as default1420 } from "../icons/waves";
import { default as default1421 } from "../icons/waypoints";
import { default as default1422 } from "../icons/webcam";
import { default as default1423 } from "../icons/webhook-off";
import { default as default1424 } from "../icons/webhook";
import { default as default1425 } from "../icons/weight-tilde";
import { default as default1426 } from "../icons/weight";
import { default as default1427 } from "../icons/wheat";
import { default as default1428 } from "../icons/wheat-off";
import { default as default1429 } from "../icons/whole-word";
import { default as default1430 } from "../icons/wifi-cog";
import { default as default1431 } from "../icons/wifi-high";
import { default as default1432 } from "../icons/wifi-low";
import { default as default1433 } from "../icons/wifi-off";
import { default as default1434 } from "../icons/wifi-pen";
import { default as default1435 } from "../icons/wifi-sync";
import { default as default1436 } from "../icons/wifi-zero";
import { default as default1437 } from "../icons/wifi";
import { default as default1438 } from "../icons/wind-arrow-down";
import { default as default1439 } from "../icons/wind";
import { default as default1440 } from "../icons/wine-off";
import { default as default1441 } from "../icons/wine";
import { default as default1442 } from "../icons/workflow";
import { default as default1443 } from "../icons/wrench";
import { default as default1444 } from "../icons/worm";
import { default as default1445 } from "../icons/x";
import { default as default1446 } from "../icons/x-line-top";
import { default as default1447 } from "../icons/zap";
import { default as default1448 } from "../icons/zap-off";
import { default as default1449 } from "../icons/zodiac-aquarius";
import { default as default1450 } from "../icons/zodiac-aries";
import { default as default1451 } from "../icons/zodiac-cancer";
import { default as default1452 } from "../icons/zodiac-capricorn";
import { default as default1453 } from "../icons/zodiac-gemini";
import { default as default1454 } from "../icons/zodiac-leo";
import { default as default1455 } from "../icons/zodiac-libra";
import { default as default1456 } from "../icons/zodiac-ophiuchus";
import { default as default1457 } from "../icons/zodiac-pisces";
import { default as default1458 } from "../icons/zodiac-sagittarius";
import { default as default1459 } from "../icons/zodiac-scorpio";
import { default as default1460 } from "../icons/zodiac-taurus";
import { default as default1461 } from "../icons/zoom-in";
import { default as default1462 } from "../icons/zodiac-virgo";
import { default as default1463 } from "../icons/zoom-out";
import { default as default1464 } from "../icons/fingerprint-pattern";
import {
  default as default1465
} from "../icons/fingerprint-pattern";
import { default as default1466 } from "../icons/arrow-down-0-1";
import { default as default1467 } from "../icons/arrow-down-1-0";
import { default as default1468 } from "../icons/arrow-up-0-1";
import { default as default1469 } from "../icons/arrow-up-1-0";
import { default as default1470 } from "../icons/alarm-clock-check";
import {
  default as default1471
} from "../icons/alarm-check";
import { default as default1472 } from "../icons/alarm-clock-minus";
import {
  default as default1473
} from "../icons/alarm-minus";
import { default as default1474 } from "../icons/alarm-clock-plus";
import {
  default as default1475
} from "../icons/alarm-plus";
import { default as default1476 } from "../icons/arrow-down-a-z";
import {
  default as default1477
} from "../icons/arrow-down-az";
import { default as default1478 } from "../icons/arrow-down-wide-narrow";
import {
  default as default1479
} from "../icons/sort-desc";
import { default as default1480 } from "../icons/arrow-down-z-a";
import {
  default as default1481
} from "../icons/arrow-down-za";
import { default as default1482 } from "../icons/arrow-up-a-z";
import {
  default as default1483
} from "../icons/arrow-up-az";
import { default as default1484 } from "../icons/arrow-up-narrow-wide";
import {
  default as default1485
} from "../icons/sort-asc";
import { default as default1486 } from "../icons/arrow-up-z-a";
import {
  default as default1487
} from "../icons/arrow-up-za";
import { default as default1488 } from "../icons/axis-3d";
import {
  default as default1489
} from "../icons/axis-3-d";
import { default as default1490 } from "../icons/badge-check";
import {
  default as default1491
} from "../icons/verified";
import { default as default1492 } from "../icons/badge-question-mark";
import {
  default as default1493
} from "../icons/badge-help";
import { default as default1494 } from "../icons/between-horizontal-end";
import {
  default as default1495
} from "../icons/between-horizonal-end";
import { default as default1496 } from "../icons/between-horizontal-start";
import {
  default as default1497
} from "../icons/between-horizonal-start";
import { default as default1498 } from "../icons/book-dashed";
import {
  default as default1499
} from "../icons/book-template";
import { default as default1500 } from "../icons/braces";
import {
  default as default1501
} from "../icons/curly-braces";
import { default as default1502 } from "../icons/captions";
import {
  default as default1503
} from "../icons/subtitles";
import { default as default1504 } from "../icons/chart-area";
import {
  default as default1505
} from "../icons/area-chart";
import { default as default1506 } from "../icons/chart-bar-big";
import {
  default as default1507
} from "../icons/bar-chart-horizontal-big";
import { default as default1508 } from "../icons/chart-bar";
import {
  default as default1509
} from "../icons/bar-chart-horizontal";
import { default as default1510 } from "../icons/chart-candlestick";
import {
  default as default1511
} from "../icons/candlestick-chart";
import { default as default1512 } from "../icons/chart-column-increasing";
import {
  default as default1513
} from "../icons/bar-chart-4";
import { default as default1514 } from "../icons/chart-column-big";
import {
  default as default1515
} from "../icons/bar-chart-big";
import { default as default1516 } from "../icons/chart-column";
import {
  default as default1517
} from "../icons/bar-chart-3";
import { default as default1518 } from "../icons/chart-no-axes-column-increasing";
import {
  default as default1519
} from "../icons/bar-chart";
import { default as default1520 } from "../icons/chart-line";
import {
  default as default1521
} from "../icons/line-chart";
import { default as default1522 } from "../icons/chart-no-axes-gantt";
import {
  default as default1523
} from "../icons/gantt-chart";
import { default as default1524 } from "../icons/chart-no-axes-column";
import {
  default as default1525
} from "../icons/bar-chart-2";
import { default as default1526 } from "../icons/chart-pie";
import {
  default as default1527
} from "../icons/pie-chart";
import { default as default1528 } from "../icons/chart-scatter";
import {
  default as default1529
} from "../icons/scatter-chart";
import { default as default1530 } from "../icons/circle-alert";
import {
  default as default1531
} from "../icons/alert-circle";
import { default as default1532 } from "../icons/circle-arrow-down";
import {
  default as default1533
} from "../icons/arrow-down-circle";
import { default as default1534 } from "../icons/circle-arrow-out-down-left";
import {
  default as default1535
} from "../icons/arrow-down-left-from-circle";
import { default as default1536 } from "../icons/circle-arrow-left";
import {
  default as default1537
} from "../icons/arrow-left-circle";
import { default as default1538 } from "../icons/circle-arrow-out-down-right";
import {
  default as default1539
} from "../icons/arrow-down-right-from-circle";
import { default as default1540 } from "../icons/circle-arrow-out-up-right";
import {
  default as default1541
} from "../icons/arrow-up-right-from-circle";
import { default as default1542 } from "../icons/circle-arrow-out-up-left";
import {
  default as default1543
} from "../icons/arrow-up-left-from-circle";
import { default as default1544 } from "../icons/circle-arrow-right";
import {
  default as default1545
} from "../icons/arrow-right-circle";
import { default as default1546 } from "../icons/circle-arrow-up";
import {
  default as default1547
} from "../icons/arrow-up-circle";
import { default as default1548 } from "../icons/circle-check-big";
import {
  default as default1549
} from "../icons/check-circle";
import { default as default1550 } from "../icons/circle-check";
import {
  default as default1551
} from "../icons/check-circle-2";
import { default as default1552 } from "../icons/circle-chevron-left";
import {
  default as default1553
} from "../icons/chevron-left-circle";
import { default as default1554 } from "../icons/circle-chevron-down";
import {
  default as default1555
} from "../icons/chevron-down-circle";
import { default as default1556 } from "../icons/circle-chevron-right";
import {
  default as default1557
} from "../icons/chevron-right-circle";
import { default as default1558 } from "../icons/circle-chevron-up";
import {
  default as default1559
} from "../icons/chevron-up-circle";
import { default as default1560 } from "../icons/circle-divide";
import {
  default as default1561
} from "../icons/divide-circle";
import { default as default1562 } from "../icons/circle-gauge";
import {
  default as default1563
} from "../icons/gauge-circle";
import { default as default1564 } from "../icons/circle-minus";
import {
  default as default1565
} from "../icons/minus-circle";
import { default as default1566 } from "../icons/circle-parking-off";
import {
  default as default1567
} from "../icons/parking-circle-off";
import { default as default1568 } from "../icons/circle-pause";
import {
  default as default1569
} from "../icons/pause-circle";
import { default as default1570 } from "../icons/circle-parking";
import {
  default as default1571
} from "../icons/parking-circle";
import { default as default1572 } from "../icons/circle-percent";
import {
  default as default1573
} from "../icons/percent-circle";
import { default as default1574 } from "../icons/circle-plus";
import {
  default as default1575
} from "../icons/plus-circle";
import { default as default1576 } from "../icons/circle-power";
import {
  default as default1577
} from "../icons/power-circle";
import { default as default1578 } from "../icons/circle-play";
import {
  default as default1579
} from "../icons/play-circle";
import { default as default1580 } from "../icons/circle-question-mark";
import {
  default as default1581
} from "../icons/help-circle";
import {
  default as default1582
} from "../icons/circle-help";
import { default as default1583 } from "../icons/circle-slash-2";
import {
  default as default1584
} from "../icons/circle-slashed";
import { default as default1585 } from "../icons/circle-stop";
import {
  default as default1586
} from "../icons/stop-circle";
import { default as default1587 } from "../icons/circle-user-round";
import {
  default as default1588
} from "../icons/user-circle-2";
import { default as default1589 } from "../icons/circle-user";
import {
  default as default1590
} from "../icons/user-circle";
import { default as default1591 } from "../icons/circle-x";
import {
  default as default1592
} from "../icons/x-circle";
import { default as default1593 } from "../icons/clipboard-pen-line";
import {
  default as default1594
} from "../icons/clipboard-signature";
import { default as default1595 } from "../icons/clipboard-pen";
import {
  default as default1596
} from "../icons/clipboard-edit";
import { default as default1597 } from "../icons/cloud-download";
import {
  default as default1598
} from "../icons/download-cloud";
import { default as default1599 } from "../icons/cloud-upload";
import {
  default as default1600
} from "../icons/upload-cloud";
import { default as default1601 } from "../icons/code-xml";
import {
  default as default1602
} from "../icons/code-2";
import { default as default1603 } from "../icons/columns-2";
import {
  default as default1604
} from "../icons/columns";
import { default as default1605 } from "../icons/columns-3-cog";
import {
  default as default1606
} from "../icons/columns-settings";
import {
  default as default1607
} from "../icons/table-config";
import { default as default1608 } from "../icons/columns-3";
import {
  default as default1609
} from "../icons/panels-left-right";
import { default as default1610 } from "../icons/contact-round";
import {
  default as default1611
} from "../icons/contact-2";
import { default as default1612 } from "../icons/diamond-percent";
import {
  default as default1613
} from "../icons/percent-diamond";
import { default as default1614 } from "../icons/earth";
import {
  default as default1615
} from "../icons/globe-2";
import { default as default1616 } from "../icons/ellipsis-vertical";
import {
  default as default1617
} from "../icons/more-vertical";
import { default as default1618 } from "../icons/ellipsis";
import {
  default as default1619
} from "../icons/more-horizontal";
import { default as default1620 } from "../icons/file-axis-3d";
import {
  default as default1621
} from "../icons/file-axis-3-d";
import { default as default1622 } from "../icons/file-badge";
import {
  default as default1623
} from "../icons/file-badge-2";
import { default as default1624 } from "../icons/file-braces-corner";
import {
  default as default1625
} from "../icons/file-json-2";
import { default as default1626 } from "../icons/file-braces";
import {
  default as default1627
} from "../icons/file-json";
import { default as default1628 } from "../icons/file-chart-column-increasing";
import {
  default as default1629
} from "../icons/file-bar-chart";
import { default as default1630 } from "../icons/file-chart-column";
import {
  default as default1631
} from "../icons/file-bar-chart-2";
import { default as default1632 } from "../icons/file-chart-line";
import {
  default as default1633
} from "../icons/file-line-chart";
import { default as default1634 } from "../icons/file-chart-pie";
import {
  default as default1635
} from "../icons/file-pie-chart";
import { default as default1636 } from "../icons/file-check-corner";
import {
  default as default1637
} from "../icons/file-check-2";
import { default as default1638 } from "../icons/file-code-corner";
import {
  default as default1639
} from "../icons/file-code-2";
import { default as default1640 } from "../icons/file-cog";
import {
  default as default1641
} from "../icons/file-cog-2";
import { default as default1642 } from "../icons/file-exclamation-point";
import {
  default as default1643
} from "../icons/file-warning";
import { default as default1644 } from "../icons/file-headphone";
import {
  default as default1645
} from "../icons/file-audio";
import {
  default as default1646
} from "../icons/file-audio-2";
import { default as default1647 } from "../icons/file-lock";
import {
  default as default1648
} from "../icons/file-lock-2";
import { default as default1649 } from "../icons/file-minus-corner";
import {
  default as default1650
} from "../icons/file-minus-2";
import { default as default1651 } from "../icons/file-key";
import {
  default as default1652
} from "../icons/file-key-2";
import { default as default1653 } from "../icons/file-pen-line";
import {
  default as default1654
} from "../icons/file-signature";
import { default as default1655 } from "../icons/file-pen";
import {
  default as default1656
} from "../icons/file-edit";
import { default as default1657 } from "../icons/file-play";
import {
  default as default1658
} from "../icons/file-video";
import { default as default1659 } from "../icons/file-plus-corner";
import {
  default as default1660
} from "../icons/file-plus-2";
import { default as default1661 } from "../icons/file-search-corner";
import {
  default as default1662
} from "../icons/file-search-2";
import { default as default1663 } from "../icons/file-question-mark";
import {
  default as default1664
} from "../icons/file-question";
import { default as default1665 } from "../icons/file-signal";
import {
  default as default1666
} from "../icons/file-volume-2";
import { default as default1667 } from "../icons/file-type-corner";
import {
  default as default1668
} from "../icons/file-type-2";
import { default as default1669 } from "../icons/file-video-camera";
import {
  default as default1670
} from "../icons/file-video-2";
import { default as default1671 } from "../icons/file-x-corner";
import {
  default as default1672
} from "../icons/file-x-2";
import { default as default1673 } from "../icons/folder-cog";
import {
  default as default1674
} from "../icons/folder-cog-2";
import { default as default1675 } from "../icons/folder-pen";
import {
  default as default1676
} from "../icons/folder-edit";
import { default as default1677 } from "../icons/funnel-x";
import {
  default as default1678
} from "../icons/filter-x";
import { default as default1679 } from "../icons/funnel";
import {
  default as default1680
} from "../icons/filter";
import { default as default1681 } from "../icons/git-commit-horizontal";
import {
  default as default1682
} from "../icons/git-commit";
import { default as default1683 } from "../icons/grid-2x2-check";
import {
  default as default1684
} from "../icons/grid-2-x-2-check";
import { default as default1685 } from "../icons/grid-2x2-plus";
import {
  default as default1686
} from "../icons/grid-2-x-2-plus";
import { default as default1687 } from "../icons/grid-2x2";
import {
  default as default1688
} from "../icons/grid-2-x-2";
import { default as default1689 } from "../icons/grid-2x2-x";
import {
  default as default1690
} from "../icons/grid-2-x-2-x";
import { default as default1691 } from "../icons/grid-3x3";
import {
  default as default1692
} from "../icons/grid";
import {
  default as default1693
} from "../icons/grid-3-x-3";
import { default as default1694 } from "../icons/hand-helping";
import {
  default as default1695
} from "../icons/helping-hand";
import { default as default1696 } from "../icons/hand-grab";
import {
  default as default1697
} from "../icons/grab";
import { default as default1698 } from "../icons/house";
import {
  default as default1699
} from "../icons/home";
import { default as default1700 } from "../icons/ice-cream-bowl";
import {
  default as default1701
} from "../icons/ice-cream-2";
import { default as default1702 } from "../icons/ice-cream-cone";
import {
  default as default1703
} from "../icons/ice-cream";
import { default as default1704 } from "../icons/laptop-minimal";
import {
  default as default1705
} from "../icons/laptop-2";
import { default as default1706 } from "../icons/layers";
import {
  default as default1707
} from "../icons/layers-3";
import { default as default1708 } from "../icons/list-indent-decrease";
import {
  default as default1709
} from "../icons/outdent";
import {
  default as default1710
} from "../icons/indent-decrease";
import { default as default1711 } from "../icons/list-indent-increase";
import {
  default as default1712
} from "../icons/indent";
import {
  default as default1713
} from "../icons/indent-increase";
import { default as default1714 } from "../icons/loader-circle";
import {
  default as default1715
} from "../icons/loader-2";
import { default as default1716 } from "../icons/lock-open";
import {
  default as default1717
} from "../icons/unlock";
import { default as default1718 } from "../icons/lock-keyhole-open";
import {
  default as default1719
} from "../icons/unlock-keyhole";
import { default as default1720 } from "../icons/mail-question-mark";
import {
  default as default1721
} from "../icons/mail-question";
import { default as default1722 } from "../icons/map-pin-pen";
import {
  default as default1723
} from "../icons/location-edit";
import { default as default1724 } from "../icons/message-circle-question-mark";
import {
  default as default1725
} from "../icons/message-circle-question";
import { default as default1726 } from "../icons/mic-vocal";
import {
  default as default1727
} from "../icons/mic-2";
import { default as default1728 } from "../icons/move-3d";
import {
  default as default1729
} from "../icons/move-3-d";
import { default as default1730 } from "../icons/octagon-alert";
import {
  default as default1731
} from "../icons/alert-octagon";
import { default as default1732 } from "../icons/octagon-pause";
import {
  default as default1733
} from "../icons/pause-octagon";
import { default as default1734 } from "../icons/octagon-x";
import {
  default as default1735
} from "../icons/x-octagon";
import { default as default1736 } from "../icons/paintbrush-vertical";
import {
  default as default1737
} from "../icons/paintbrush-2";
import { default as default1738 } from "../icons/panel-bottom-dashed";
import {
  default as default1739
} from "../icons/panel-bottom-inactive";
import { default as default1740 } from "../icons/panel-left-dashed";
import {
  default as default1741
} from "../icons/panel-left-inactive";
import { default as default1742 } from "../icons/panel-left-close";
import {
  default as default1743
} from "../icons/sidebar-close";
import { default as default1744 } from "../icons/panel-left";
import {
  default as default1745
} from "../icons/sidebar";
import { default as default1746 } from "../icons/panel-left-open";
import {
  default as default1747
} from "../icons/sidebar-open";
import { default as default1748 } from "../icons/panel-right-dashed";
import {
  default as default1749
} from "../icons/panel-right-inactive";
import { default as default1750 } from "../icons/panel-top-dashed";
import {
  default as default1751
} from "../icons/panel-top-inactive";
import { default as default1752 } from "../icons/panels-top-left";
import {
  default as default1753
} from "../icons/layout";
import { default as default1754 } from "../icons/pen-line";
import {
  default as default1755
} from "../icons/edit-3";
import { default as default1756 } from "../icons/pen";
import {
  default as default1757
} from "../icons/edit-2";
import { default as default1758 } from "../icons/plug-zap";
import {
  default as default1759
} from "../icons/plug-zap-2";
import { default as default1760 } from "../icons/rectangle-ellipsis";
import {
  default as default1761
} from "../icons/form-input";
import { default as default1762 } from "../icons/rotate-3d";
import {
  default as default1763
} from "../icons/rotate-3-d";
import { default as default1764 } from "../icons/rows-2";
import {
  default as default1765
} from "../icons/rows";
import { default as default1766 } from "../icons/rows-3";
import {
  default as default1767
} from "../icons/panels-top-bottom";
import { default as default1768 } from "../icons/scale-3d";
import {
  default as default1769
} from "../icons/scale-3-d";
import { default as default1770 } from "../icons/send-horizontal";
import {
  default as default1771
} from "../icons/send-horizonal";
import { default as default1772 } from "../icons/shield-question-mark";
import {
  default as default1773
} from "../icons/shield-question";
import { default as default1774 } from "../icons/shield-x";
import {
  default as default1775
} from "../icons/shield-close";
import { default as default1776 } from "../icons/sliders-vertical";
import {
  default as default1777
} from "../icons/sliders";
import { default as default1778 } from "../icons/sparkles";
import {
  default as default1779
} from "../icons/stars";
import { default as default1780 } from "../icons/square-activity";
import {
  default as default1781
} from "../icons/activity-square";
import { default as default1782 } from "../icons/square-arrow-down-left";
import {
  default as default1783
} from "../icons/arrow-down-left-square";
import { default as default1784 } from "../icons/square-arrow-down-right";
import {
  default as default1785
} from "../icons/arrow-down-right-square";
import { default as default1786 } from "../icons/square-arrow-down";
import {
  default as default1787
} from "../icons/arrow-down-square";
import { default as default1788 } from "../icons/square-arrow-left";
import {
  default as default1789
} from "../icons/arrow-left-square";
import { default as default1790 } from "../icons/square-arrow-out-down-right";
import {
  default as default1791
} from "../icons/arrow-down-right-from-square";
import { default as default1792 } from "../icons/square-arrow-out-up-left";
import {
  default as default1793
} from "../icons/arrow-up-left-from-square";
import { default as default1794 } from "../icons/square-arrow-out-down-left";
import {
  default as default1795
} from "../icons/arrow-down-left-from-square";
import { default as default1796 } from "../icons/square-arrow-out-up-right";
import {
  default as default1797
} from "../icons/arrow-up-right-from-square";
import { default as default1798 } from "../icons/square-arrow-right";
import {
  default as default1799
} from "../icons/arrow-right-square";
import { default as default1800 } from "../icons/square-arrow-up-left";
import {
  default as default1801
} from "../icons/arrow-up-left-square";
import { default as default1802 } from "../icons/square-arrow-up-right";
import {
  default as default1803
} from "../icons/arrow-up-right-square";
import { default as default1804 } from "../icons/square-arrow-up";
import {
  default as default1805
} from "../icons/arrow-up-square";
import { default as default1806 } from "../icons/square-bottom-dashed-scissors";
import {
  default as default1807
} from "../icons/scissors-square-dashed-bottom";
import { default as default1808 } from "../icons/square-asterisk";
import {
  default as default1809
} from "../icons/asterisk-square";
import { default as default1810 } from "../icons/square-centerline-dashed-horizontal";
import {
  default as default1811
} from "../icons/flip-horizontal";
import { default as default1812 } from "../icons/square-centerline-dashed-vertical";
import {
  default as default1813
} from "../icons/flip-vertical";
import { default as default1814 } from "../icons/square-chart-gantt";
import {
  default as default1815
} from "../icons/gantt-chart-square";
import {
  default as default1816
} from "../icons/square-gantt-chart";
import { default as default1817 } from "../icons/square-chevron-down";
import {
  default as default1818
} from "../icons/chevron-down-square";
import { default as default1819 } from "../icons/square-check-big";
import {
  default as default1820
} from "../icons/check-square";
import { default as default1821 } from "../icons/square-check";
import {
  default as default1822
} from "../icons/check-square-2";
import { default as default1823 } from "../icons/square-chevron-left";
import {
  default as default1824
} from "../icons/chevron-left-square";
import { default as default1825 } from "../icons/square-chevron-right";
import {
  default as default1826
} from "../icons/chevron-right-square";
import { default as default1827 } from "../icons/square-chevron-up";
import {
  default as default1828
} from "../icons/chevron-up-square";
import { default as default1829 } from "../icons/square-code";
import {
  default as default1830
} from "../icons/code-square";
import { default as default1831 } from "../icons/square-dashed-kanban";
import {
  default as default1832
} from "../icons/kanban-square-dashed";
import { default as default1833 } from "../icons/square-dashed-mouse-pointer";
import {
  default as default1834
} from "../icons/mouse-pointer-square-dashed";
import { default as default1835 } from "../icons/square-dashed-text";
import {
  default as default1836
} from "../icons/text-selection";
import {
  default as default1837
} from "../icons/text-select";
import { default as default1838 } from "../icons/square-dashed";
import {
  default as default1839
} from "../icons/box-select";
import { default as default1840 } from "../icons/square-divide";
import {
  default as default1841
} from "../icons/divide-square";
import { default as default1842 } from "../icons/square-dot";
import {
  default as default1843
} from "../icons/dot-square";
import { default as default1844 } from "../icons/square-function";
import {
  default as default1845
} from "../icons/function-square";
import { default as default1846 } from "../icons/square-equal";
import {
  default as default1847
} from "../icons/equal-square";
import { default as default1848 } from "../icons/square-kanban";
import {
  default as default1849
} from "../icons/kanban-square";
import { default as default1850 } from "../icons/square-library";
import {
  default as default1851
} from "../icons/library-square";
import { default as default1852 } from "../icons/square-m";
import {
  default as default1853
} from "../icons/m-square";
import { default as default1854 } from "../icons/square-menu";
import {
  default as default1855
} from "../icons/menu-square";
import { default as default1856 } from "../icons/square-minus";
import {
  default as default1857
} from "../icons/minus-square";
import { default as default1858 } from "../icons/square-mouse-pointer";
import {
  default as default1859
} from "../icons/inspect";
import { default as default1860 } from "../icons/square-parking";
import {
  default as default1861
} from "../icons/parking-square";
import { default as default1862 } from "../icons/square-percent";
import {
  default as default1863
} from "../icons/percent-square";
import { default as default1864 } from "../icons/square-pen";
import {
  default as default1865
} from "../icons/pen-box";
import {
  default as default1866
} from "../icons/edit";
import {
  default as default1867
} from "../icons/pen-square";
import { default as default1868 } from "../icons/square-parking-off";
import {
  default as default1869
} from "../icons/parking-square-off";
import { default as default1870 } from "../icons/square-pi";
import {
  default as default1871
} from "../icons/pi-square";
import { default as default1872 } from "../icons/square-pilcrow";
import {
  default as default1873
} from "../icons/pilcrow-square";
import { default as default1874 } from "../icons/square-play";
import {
  default as default1875
} from "../icons/play-square";
import { default as default1876 } from "../icons/square-plus";
import {
  default as default1877
} from "../icons/plus-square";
import { default as default1878 } from "../icons/square-scissors";
import {
  default as default1879
} from "../icons/scissors-square";
import { default as default1880 } from "../icons/square-sigma";
import {
  default as default1881
} from "../icons/sigma-square";
import { default as default1882 } from "../icons/square-power";
import {
  default as default1883
} from "../icons/power-square";
import { default as default1884 } from "../icons/square-slash";
import {
  default as default1885
} from "../icons/slash-square";
import { default as default1886 } from "../icons/square-split-horizontal";
import {
  default as default1887
} from "../icons/split-square-horizontal";
import { default as default1888 } from "../icons/square-split-vertical";
import {
  default as default1889
} from "../icons/split-square-vertical";
import { default as default1890 } from "../icons/square-terminal";
import {
  default as default1891
} from "../icons/terminal-square";
import { default as default1892 } from "../icons/square-user-round";
import {
  default as default1893
} from "../icons/user-square-2";
import { default as default1894 } from "../icons/square-user";
import {
  default as default1895
} from "../icons/user-square";
import { default as default1896 } from "../icons/square-x";
import {
  default as default1897
} from "../icons/x-square";
import { default as default1898 } from "../icons/test-tube-diagonal";
import {
  default as default1899
} from "../icons/test-tube-2";
import { default as default1900 } from "../icons/text-align-center";
import {
  default as default1901
} from "../icons/align-center";
import { default as default1902 } from "../icons/text-align-end";
import {
  default as default1903
} from "../icons/align-right";
import { default as default1904 } from "../icons/text-align-justify";
import {
  default as default1905
} from "../icons/align-justify";
import { default as default1906 } from "../icons/text-align-start";
import {
  default as default1907
} from "../icons/text";
import {
  default as default1908
} from "../icons/align-left";
import { default as default1909 } from "../icons/text-initial";
import {
  default as default1910
} from "../icons/letter-text";
import { default as default1911 } from "../icons/text-wrap";
import {
  default as default1912
} from "../icons/wrap-text";
import { default as default1913 } from "../icons/tram-front";
import {
  default as default1914
} from "../icons/train";
import { default as default1915 } from "../icons/tree-palm";
import {
  default as default1916
} from "../icons/palmtree";
import { default as default1917 } from "../icons/triangle-alert";
import {
  default as default1918
} from "../icons/alert-triangle";
import { default as default1919 } from "../icons/tv-minimal";
import {
  default as default1920
} from "../icons/tv-2";
import { default as default1921 } from "../icons/university";
import {
  default as default1922
} from "../icons/school-2";
import { default as default1923 } from "../icons/user-round-check";
import {
  default as default1924
} from "../icons/user-check-2";
import { default as default1925 } from "../icons/user-round-cog";
import {
  default as default1926
} from "../icons/user-cog-2";
import { default as default1927 } from "../icons/user-round-minus";
import {
  default as default1928
} from "../icons/user-minus-2";
import { default as default1929 } from "../icons/user-round-plus";
import {
  default as default1930
} from "../icons/user-plus-2";
import { default as default1931 } from "../icons/user-round-x";
import {
  default as default1932
} from "../icons/user-x-2";
import { default as default1933 } from "../icons/user-round";
import {
  default as default1934
} from "../icons/user-2";
import { default as default1935 } from "../icons/users-round";
import {
  default as default1936
} from "../icons/users-2";
import { default as default1937 } from "../icons/utensils-crossed";
import {
  default as default1938
} from "../icons/fork-knife-crossed";
import { default as default1939 } from "../icons/utensils";
import {
  default as default1940
} from "../icons/fork-knife";
import { default as default1941 } from "../icons/wallet-minimal";
import {
  default as default1942
} from "../icons/wallet-2";
import { default as default1943 } from "../icons/wand-sparkles";
import {
  default as default1944
} from "../icons/wand-2";
export {
  default2 as LucideAArrowDown,
  default3 as LucideAArrowUp,
  default4 as LucideALargeSmall,
  default5 as LucideAccessibility,
  default6 as LucideActivity,
  default1781 as LucideActivitySquare,
  default7 as LucideAirVent,
  default8 as LucideAirplay,
  default1471 as LucideAlarmCheck,
  default10 as LucideAlarmClock,
  default1470 as LucideAlarmClockCheck,
  default1472 as LucideAlarmClockMinus,
  default9 as LucideAlarmClockOff,
  default1474 as LucideAlarmClockPlus,
  default1473 as LucideAlarmMinus,
  default1475 as LucideAlarmPlus,
  default11 as LucideAlarmSmoke,
  default12 as LucideAlbum,
  default1531 as LucideAlertCircle,
  default1731 as LucideAlertOctagon,
  default1918 as LucideAlertTriangle,
  default1901 as LucideAlignCenter,
  default13 as LucideAlignCenterHorizontal,
  default14 as LucideAlignCenterVertical,
  default15 as LucideAlignEndHorizontal,
  default16 as LucideAlignEndVertical,
  default17 as LucideAlignHorizontalDistributeCenter,
  default18 as LucideAlignHorizontalDistributeEnd,
  default19 as LucideAlignHorizontalDistributeStart,
  default20 as LucideAlignHorizontalJustifyCenter,
  default21 as LucideAlignHorizontalJustifyEnd,
  default22 as LucideAlignHorizontalJustifyStart,
  default23 as LucideAlignHorizontalSpaceAround,
  default24 as LucideAlignHorizontalSpaceBetween,
  default1905 as LucideAlignJustify,
  default1908 as LucideAlignLeft,
  default1903 as LucideAlignRight,
  default25 as LucideAlignStartHorizontal,
  default26 as LucideAlignStartVertical,
  default27 as LucideAlignVerticalDistributeCenter,
  default28 as LucideAlignVerticalDistributeEnd,
  default29 as LucideAlignVerticalDistributeStart,
  default30 as LucideAlignVerticalJustifyCenter,
  default31 as LucideAlignVerticalJustifyEnd,
  default32 as LucideAlignVerticalJustifyStart,
  default33 as LucideAlignVerticalSpaceAround,
  default34 as LucideAlignVerticalSpaceBetween,
  default35 as LucideAmbulance,
  default36 as LucideAmpersand,
  default37 as LucideAmpersands,
  default38 as LucideAmphora,
  default39 as LucideAnchor,
  default40 as LucideAngry,
  default41 as LucideAnnoyed,
  default42 as LucideAntenna,
  default43 as LucideAnvil,
  default44 as LucideAperture,
  default46 as LucideAppWindow,
  default45 as LucideAppWindowMac,
  default47 as LucideApple,
  default50 as LucideArchive,
  default48 as LucideArchiveRestore,
  default49 as LucideArchiveX,
  default1505 as LucideAreaChart,
  default51 as LucideArmchair,
  default53 as LucideArrowBigDown,
  default52 as LucideArrowBigDownDash,
  default56 as LucideArrowBigLeft,
  default54 as LucideArrowBigLeftDash,
  default57 as LucideArrowBigRight,
  default55 as LucideArrowBigRightDash,
  default59 as LucideArrowBigUp,
  default58 as LucideArrowBigUpDash,
  default67 as LucideArrowDown,
  default1466 as LucideArrowDown01,
  default1467 as LucideArrowDown10,
  default1476 as LucideArrowDownAZ,
  default1477 as LucideArrowDownAz,
  default1533 as LucideArrowDownCircle,
  default60 as LucideArrowDownFromLine,
  default61 as LucideArrowDownLeft,
  default1535 as LucideArrowDownLeftFromCircle,
  default1795 as LucideArrowDownLeftFromSquare,
  default1783 as LucideArrowDownLeftSquare,
  default63 as LucideArrowDownNarrowWide,
  default62 as LucideArrowDownRight,
  default1539 as LucideArrowDownRightFromCircle,
  default1791 as LucideArrowDownRightFromSquare,
  default1785 as LucideArrowDownRightSquare,
  default1787 as LucideArrowDownSquare,
  default64 as LucideArrowDownToDot,
  default65 as LucideArrowDownToLine,
  default66 as LucideArrowDownUp,
  default1478 as LucideArrowDownWideNarrow,
  default1480 as LucideArrowDownZA,
  default1481 as LucideArrowDownZa,
  default71 as LucideArrowLeft,
  default1537 as LucideArrowLeftCircle,
  default68 as LucideArrowLeftFromLine,
  default69 as LucideArrowLeftRight,
  default1789 as LucideArrowLeftSquare,
  default70 as LucideArrowLeftToLine,
  default75 as LucideArrowRight,
  default1545 as LucideArrowRightCircle,
  default72 as LucideArrowRightFromLine,
  default73 as LucideArrowRightLeft,
  default1799 as LucideArrowRightSquare,
  default74 as LucideArrowRightToLine,
  default83 as LucideArrowUp,
  default1468 as LucideArrowUp01,
  default1469 as LucideArrowUp10,
  default1482 as LucideArrowUpAZ,
  default1483 as LucideArrowUpAz,
  default1547 as LucideArrowUpCircle,
  default76 as LucideArrowUpDown,
  default77 as LucideArrowUpFromDot,
  default78 as LucideArrowUpFromLine,
  default79 as LucideArrowUpLeft,
  default1543 as LucideArrowUpLeftFromCircle,
  default1793 as LucideArrowUpLeftFromSquare,
  default1801 as LucideArrowUpLeftSquare,
  default1484 as LucideArrowUpNarrowWide,
  default80 as LucideArrowUpRight,
  default1541 as LucideArrowUpRightFromCircle,
  default1797 as LucideArrowUpRightFromSquare,
  default1803 as LucideArrowUpRightSquare,
  default1805 as LucideArrowUpSquare,
  default81 as LucideArrowUpToLine,
  default82 as LucideArrowUpWideNarrow,
  default1486 as LucideArrowUpZA,
  default1487 as LucideArrowUpZa,
  default84 as LucideArrowsUpFromLine,
  default85 as LucideAsterisk,
  default1809 as LucideAsteriskSquare,
  default87 as LucideAtSign,
  default86 as LucideAtom,
  default88 as LucideAudioLines,
  default89 as LucideAudioWaveform,
  default91 as LucideAward,
  default90 as LucideAxe,
  default1489 as LucideAxis3D,
  default1488 as LucideAxis3d,
  default92 as LucideBaby,
  default93 as LucideBackpack,
  default109 as LucideBadge,
  default94 as LucideBadgeAlert,
  default95 as LucideBadgeCent,
  default1490 as LucideBadgeCheck,
  default96 as LucideBadgeDollarSign,
  default97 as LucideBadgeEuro,
  default1493 as LucideBadgeHelp,
  default98 as LucideBadgeIndianRupee,
  default99 as LucideBadgeInfo,
  default101 as LucideBadgeJapaneseYen,
  default100 as LucideBadgeMinus,
  default102 as LucideBadgePercent,
  default103 as LucideBadgePlus,
  default104 as LucideBadgePoundSterling,
  default1492 as LucideBadgeQuestionMark,
  default105 as LucideBadgeRussianRuble,
  default106 as LucideBadgeSwissFranc,
  default107 as LucideBadgeTurkishLira,
  default108 as LucideBadgeX,
  default110 as LucideBaggageClaim,
  default111 as LucideBalloon,
  default112 as LucideBan,
  default113 as LucideBanana,
  default114 as LucideBandage,
  default118 as LucideBanknote,
  default115 as LucideBanknoteArrowDown,
  default116 as LucideBanknoteArrowUp,
  default117 as LucideBanknoteX,
  default1519 as LucideBarChart,
  default1525 as LucideBarChart2,
  default1517 as LucideBarChart3,
  default1513 as LucideBarChart4,
  default1515 as LucideBarChartBig,
  default1509 as LucideBarChartHorizontal,
  default1507 as LucideBarChartHorizontalBig,
  default119 as LucideBarcode,
  default120 as LucideBarrel,
  default121 as LucideBaseline,
  default122 as LucideBath,
  default129 as LucideBattery,
  default123 as LucideBatteryCharging,
  default124 as LucideBatteryFull,
  default125 as LucideBatteryLow,
  default126 as LucideBatteryMedium,
  default127 as LucideBatteryPlus,
  default128 as LucideBatteryWarning,
  default130 as LucideBeaker,
  default132 as LucideBean,
  default131 as LucideBeanOff,
  default135 as LucideBed,
  default133 as LucideBedDouble,
  default134 as LucideBedSingle,
  default137 as LucideBeef,
  default136 as LucideBeefOff,
  default139 as LucideBeer,
  default138 as LucideBeerOff,
  default147 as LucideBell,
  default140 as LucideBellCheck,
  default141 as LucideBellDot,
  default142 as LucideBellElectric,
  default143 as LucideBellMinus,
  default144 as LucideBellOff,
  default145 as LucideBellPlus,
  default146 as LucideBellRing,
  default1495 as LucideBetweenHorizonalEnd,
  default1497 as LucideBetweenHorizonalStart,
  default1494 as LucideBetweenHorizontalEnd,
  default1496 as LucideBetweenHorizontalStart,
  default148 as LucideBetweenVerticalEnd,
  default149 as LucideBetweenVerticalStart,
  default150 as LucideBicepsFlexed,
  default151 as LucideBike,
  default152 as LucideBinary,
  default154 as LucideBinoculars,
  default153 as LucideBiohazard,
  default155 as LucideBird,
  default156 as LucideBirdhouse,
  default157 as LucideBitcoin,
  default158 as LucideBlend,
  default159 as LucideBlinds,
  default160 as LucideBlocks,
  default164 as LucideBluetooth,
  default161 as LucideBluetoothConnected,
  default162 as LucideBluetoothOff,
  default163 as LucideBluetoothSearching,
  default165 as LucideBold,
  default166 as LucideBolt,
  default167 as LucideBomb,
  default168 as LucideBone,
  default193 as LucideBook,
  default169 as LucideBookA,
  default170 as LucideBookAlert,
  default171 as LucideBookAudio,
  default172 as LucideBookCheck,
  default173 as LucideBookCopy,
  default1498 as LucideBookDashed,
  default174 as LucideBookDown,
  default175 as LucideBookHeadphones,
  default176 as LucideBookHeart,
  default177 as LucideBookImage,
  default178 as LucideBookKey,
  default179 as LucideBookLock,
  default180 as LucideBookMarked,
  default181 as LucideBookMinus,
  default184 as LucideBookOpen,
  default182 as LucideBookOpenCheck,
  default183 as LucideBookOpenText,
  default185 as LucideBookPlus,
  default186 as LucideBookSearch,
  default1499 as LucideBookTemplate,
  default187 as LucideBookText,
  default188 as LucideBookType,
  default190 as LucideBookUp,
  default189 as LucideBookUp2,
  default191 as LucideBookUser,
  default192 as LucideBookX,
  default199 as LucideBookmark,
  default194 as LucideBookmarkCheck,
  default195 as LucideBookmarkMinus,
  default196 as LucideBookmarkOff,
  default197 as LucideBookmarkPlus,
  default198 as LucideBookmarkX,
  default200 as LucideBoomBox,
  default203 as LucideBot,
  default201 as LucideBotMessageSquare,
  default202 as LucideBotOff,
  default204 as LucideBottleWine,
  default205 as LucideBowArrow,
  default206 as LucideBox,
  default1839 as LucideBoxSelect,
  default207 as LucideBoxes,
  default1500 as LucideBraces,
  default208 as LucideBrackets,
  default211 as LucideBrain,
  default209 as LucideBrainCircuit,
  default210 as LucideBrainCog,
  default214 as LucideBrickWall,
  default212 as LucideBrickWallFire,
  default213 as LucideBrickWallShield,
  default218 as LucideBriefcase,
  default215 as LucideBriefcaseBusiness,
  default216 as LucideBriefcaseConveyorBelt,
  default217 as LucideBriefcaseMedical,
  default219 as LucideBringToFront,
  default221 as LucideBrush,
  default220 as LucideBrushCleaning,
  default222 as LucideBubbles,
  default225 as LucideBug,
  default223 as LucideBugOff,
  default224 as LucideBugPlay,
  default227 as LucideBuilding,
  default226 as LucideBuilding2,
  default229 as LucideBus,
  default228 as LucideBusFront,
  default231 as LucideCable,
  default230 as LucideCableCar,
  default233 as LucideCake,
  default232 as LucideCakeSlice,
  default234 as LucideCalculator,
  default255 as LucideCalendar,
  default235 as LucideCalendar1,
  default236 as LucideCalendarArrowDown,
  default237 as LucideCalendarArrowUp,
  default239 as LucideCalendarCheck,
  default238 as LucideCalendarCheck2,
  default240 as LucideCalendarClock,
  default241 as LucideCalendarCog,
  default242 as LucideCalendarDays,
  default244 as LucideCalendarFold,
  default243 as LucideCalendarHeart,
  default246 as LucideCalendarMinus,
  default245 as LucideCalendarMinus2,
  default247 as LucideCalendarOff,
  default249 as LucideCalendarPlus,
  default248 as LucideCalendarPlus2,
  default250 as LucideCalendarRange,
  default252 as LucideCalendarSearch,
  default251 as LucideCalendarSync,
  default254 as LucideCalendarX,
  default253 as LucideCalendarX2,
  default256 as LucideCalendars,
  default258 as LucideCamera,
  default257 as LucideCameraOff,
  default1511 as LucideCandlestickChart,
  default261 as LucideCandy,
  default259 as LucideCandyCane,
  default260 as LucideCandyOff,
  default263 as LucideCannabis,
  default262 as LucideCannabisOff,
  default1502 as LucideCaptions,
  default264 as LucideCaptionsOff,
  default268 as LucideCar,
  default265 as LucideCarFront,
  default266 as LucideCarTaxiFront,
  default267 as LucideCaravan,
  default269 as LucideCardSim,
  default270 as LucideCarrot,
  default271 as LucideCaseLower,
  default272 as LucideCaseSensitive,
  default273 as LucideCaseUpper,
  default274 as LucideCassetteTape,
  default275 as LucideCast,
  default276 as LucideCastle,
  default277 as LucideCat,
  default279 as LucideCctv,
  default278 as LucideCctvOff,
  default1504 as LucideChartArea,
  default1508 as LucideChartBar,
  default1506 as LucideChartBarBig,
  default280 as LucideChartBarDecreasing,
  default281 as LucideChartBarIncreasing,
  default282 as LucideChartBarStacked,
  default1510 as LucideChartCandlestick,
  default1516 as LucideChartColumn,
  default1514 as LucideChartColumnBig,
  default283 as LucideChartColumnDecreasing,
  default1512 as LucideChartColumnIncreasing,
  default284 as LucideChartColumnStacked,
  default285 as LucideChartGantt,
  default1520 as LucideChartLine,
  default286 as LucideChartNetwork,
  default1524 as LucideChartNoAxesColumn,
  default287 as LucideChartNoAxesColumnDecreasing,
  default1518 as LucideChartNoAxesColumnIncreasing,
  default288 as LucideChartNoAxesCombined,
  default1522 as LucideChartNoAxesGantt,
  default1526 as LucideChartPie,
  default1528 as LucideChartScatter,
  default289 as LucideChartSpline,
  default292 as LucideCheck,
  default290 as LucideCheckCheck,
  default1549 as LucideCheckCircle,
  default1551 as LucideCheckCircle2,
  default291 as LucideCheckLine,
  default1820 as LucideCheckSquare,
  default1822 as LucideCheckSquare2,
  default293 as LucideChefHat,
  default294 as LucideCherry,
  default295 as LucideChessBishop,
  default296 as LucideChessKing,
  default297 as LucideChessKnight,
  default298 as LucideChessPawn,
  default299 as LucideChessQueen,
  default300 as LucideChessRook,
  default301 as LucideChevronDown,
  default1555 as LucideChevronDownCircle,
  default1818 as LucideChevronDownSquare,
  default302 as LucideChevronFirst,
  default303 as LucideChevronLast,
  default304 as LucideChevronLeft,
  default1553 as LucideChevronLeftCircle,
  default1824 as LucideChevronLeftSquare,
  default305 as LucideChevronRight,
  default1557 as LucideChevronRightCircle,
  default1826 as LucideChevronRightSquare,
  default306 as LucideChevronUp,
  default1559 as LucideChevronUpCircle,
  default1828 as LucideChevronUpSquare,
  default308 as LucideChevronsDown,
  default307 as LucideChevronsDownUp,
  default311 as LucideChevronsLeft,
  default310 as LucideChevronsLeftRight,
  default309 as LucideChevronsLeftRightEllipsis,
  default313 as LucideChevronsRight,
  default312 as LucideChevronsRightLeft,
  default315 as LucideChevronsUp,
  default314 as LucideChevronsUpDown,
  default316 as LucideChurch,
  default318 as LucideCigarette,
  default317 as LucideCigaretteOff,
  default333 as LucideCircle,
  default1530 as LucideCircleAlert,
  default1532 as LucideCircleArrowDown,
  default1536 as LucideCircleArrowLeft,
  default1534 as LucideCircleArrowOutDownLeft,
  default1538 as LucideCircleArrowOutDownRight,
  default1542 as LucideCircleArrowOutUpLeft,
  default1540 as LucideCircleArrowOutUpRight,
  default1544 as LucideCircleArrowRight,
  default1546 as LucideCircleArrowUp,
  default1550 as LucideCircleCheck,
  default1548 as LucideCircleCheckBig,
  default1554 as LucideCircleChevronDown,
  default1552 as LucideCircleChevronLeft,
  default1556 as LucideCircleChevronRight,
  default1558 as LucideCircleChevronUp,
  default319 as LucideCircleDashed,
  default1560 as LucideCircleDivide,
  default320 as LucideCircleDollarSign,
  default323 as LucideCircleDot,
  default321 as LucideCircleDotDashed,
  default322 as LucideCircleEllipsis,
  default325 as LucideCircleEqual,
  default324 as LucideCircleFadingArrowUp,
  default326 as LucideCircleFadingPlus,
  default1562 as LucideCircleGauge,
  default1582 as LucideCircleHelp,
  default1564 as LucideCircleMinus,
  default327 as LucideCircleOff,
  default1570 as LucideCircleParking,
  default1566 as LucideCircleParkingOff,
  default1568 as LucideCirclePause,
  default1572 as LucideCirclePercent,
  default328 as LucideCirclePile,
  default1578 as LucideCirclePlay,
  default1574 as LucideCirclePlus,
  default329 as LucideCirclePoundSterling,
  default1576 as LucideCirclePower,
  default1580 as LucideCircleQuestionMark,
  default330 as LucideCircleSlash,
  default1583 as LucideCircleSlash2,
  default1584 as LucideCircleSlashed,
  default331 as LucideCircleSmall,
  default332 as LucideCircleStar,
  default1585 as LucideCircleStop,
  default1589 as LucideCircleUser,
  default1587 as LucideCircleUserRound,
  default1591 as LucideCircleX,
  default335 as LucideCircuitBoard,
  default334 as LucideCitrus,
  default336 as LucideClapperboard,
  default346 as LucideClipboard,
  default337 as LucideClipboardCheck,
  default338 as LucideClipboardClock,
  default339 as LucideClipboardCopy,
  default1596 as LucideClipboardEdit,
  default340 as LucideClipboardList,
  default341 as LucideClipboardMinus,
  default342 as LucideClipboardPaste,
  default1595 as LucideClipboardPen,
  default1593 as LucideClipboardPenLine,
  default344 as LucideClipboardPlus,
  default1594 as LucideClipboardSignature,
  default343 as LucideClipboardType,
  default345 as LucideClipboardX,
  default365 as LucideClock,
  default347 as LucideClock1,
  default348 as LucideClock10,
  default349 as LucideClock11,
  default350 as LucideClock12,
  default351 as LucideClock2,
  default352 as LucideClock3,
  default353 as LucideClock4,
  default354 as LucideClock5,
  default355 as LucideClock6,
  default356 as LucideClock7,
  default357 as LucideClock8,
  default358 as LucideClock9,
  default359 as LucideClockAlert,
  default360 as LucideClockArrowDown,
  default361 as LucideClockArrowUp,
  default362 as LucideClockCheck,
  default363 as LucideClockFading,
  default364 as LucideClockPlus,
  default366 as LucideClosedCaption,
  default384 as LucideCloud,
  default367 as LucideCloudAlert,
  default368 as LucideCloudBackup,
  default369 as LucideCloudCheck,
  default370 as LucideCloudCog,
  default1597 as LucideCloudDownload,
  default371 as LucideCloudDrizzle,
  default372 as LucideCloudFog,
  default373 as LucideCloudHail,
  default374 as LucideCloudLightning,
  default376 as LucideCloudMoon,
  default375 as LucideCloudMoonRain,
  default377 as LucideCloudOff,
  default379 as LucideCloudRain,
  default378 as LucideCloudRainWind,
  default380 as LucideCloudSnow,
  default382 as LucideCloudSun,
  default381 as LucideCloudSunRain,
  default383 as LucideCloudSync,
  default1599 as LucideCloudUpload,
  default385 as LucideCloudy,
  default386 as LucideClover,
  default387 as LucideClub,
  default392 as LucideCode,
  default1602 as LucideCode2,
  default1830 as LucideCodeSquare,
  default1601 as LucideCodeXml,
  default388 as LucideCoffee,
  default389 as LucideCog,
  default390 as LucideCoins,
  default1604 as LucideColumns,
  default1603 as LucideColumns2,
  default1608 as LucideColumns3,
  default1605 as LucideColumns3Cog,
  default395 as LucideColumns4,
  default1606 as LucideColumnsSettings,
  default391 as LucideCombine,
  default393 as LucideCommand,
  default394 as LucideCompass,
  default396 as LucideComponent,
  default399 as LucideComputer,
  default397 as LucideConciergeBell,
  default398 as LucideCone,
  default400 as LucideConstruction,
  default401 as LucideContact,
  default1611 as LucideContact2,
  default1610 as LucideContactRound,
  default402 as LucideContainer,
  default403 as LucideContrast,
  default404 as LucideCookie,
  default405 as LucideCookingPot,
  default411 as LucideCopy,
  default406 as LucideCopyCheck,
  default407 as LucideCopyMinus,
  default408 as LucideCopyPlus,
  default409 as LucideCopySlash,
  default410 as LucideCopyX,
  default412 as LucideCopyleft,
  default413 as LucideCopyright,
  default414 as LucideCornerDownLeft,
  default415 as LucideCornerDownRight,
  default416 as LucideCornerLeftDown,
  default417 as LucideCornerLeftUp,
  default418 as LucideCornerRightDown,
  default419 as LucideCornerRightUp,
  default421 as LucideCornerUpLeft,
  default420 as LucideCornerUpRight,
  default422 as LucideCpu,
  default423 as LucideCreativeCommons,
  default424 as LucideCreditCard,
  default425 as LucideCroissant,
  default426 as LucideCrop,
  default427 as LucideCross,
  default428 as LucideCrosshair,
  default429 as LucideCrown,
  default430 as LucideCuboid,
  default431 as LucideCupSoda,
  default1501 as LucideCurlyBraces,
  default432 as LucideCurrency,
  default433 as LucideCylinder,
  default434 as LucideDam,
  default438 as LucideDatabase,
  default435 as LucideDatabaseBackup,
  default436 as LucideDatabaseSearch,
  default437 as LucideDatabaseZap,
  default439 as LucideDecimalsArrowLeft,
  default440 as LucideDecimalsArrowRight,
  default441 as LucideDelete,
  default442 as LucideDessert,
  default443 as LucideDiameter,
  default447 as LucideDiamond,
  default444 as LucideDiamondMinus,
  default1612 as LucideDiamondPercent,
  default445 as LucideDiamondPlus,
  default448 as LucideDice1,
  default446 as LucideDice2,
  default449 as LucideDice3,
  default450 as LucideDice4,
  default451 as LucideDice5,
  default452 as LucideDice6,
  default453 as LucideDices,
  default454 as LucideDiff,
  default458 as LucideDisc,
  default455 as LucideDisc2,
  default456 as LucideDisc3,
  default457 as LucideDiscAlbum,
  default459 as LucideDivide,
  default1561 as LucideDivideCircle,
  default1841 as LucideDivideSquare,
  default461 as LucideDna,
  default460 as LucideDnaOff,
  default462 as LucideDock,
  default463 as LucideDog,
  default464 as LucideDollarSign,
  default465 as LucideDonut,
  default467 as LucideDoorClosed,
  default466 as LucideDoorClosedLocked,
  default468 as LucideDoorOpen,
  default469 as LucideDot,
  default1843 as LucideDotSquare,
  default470 as LucideDownload,
  default1598 as LucideDownloadCloud,
  default471 as LucideDraftingCompass,
  default472 as LucideDrama,
  default473 as LucideDrill,
  default474 as LucideDrone,
  default476 as LucideDroplet,
  default475 as LucideDropletOff,
  default477 as LucideDroplets,
  default478 as LucideDrum,
  default479 as LucideDrumstick,
  default480 as LucideDumbbell,
  default482 as LucideEar,
  default481 as LucideEarOff,
  default1614 as LucideEarth,
  default483 as LucideEarthLock,
  default484 as LucideEclipse,
  default1866 as LucideEdit,
  default1757 as LucideEdit2,
  default1755 as LucideEdit3,
  default487 as LucideEgg,
  default485 as LucideEggFried,
  default486 as LucideEggOff,
  default488 as LucideEllipse,
  default1618 as LucideEllipsis,
  default1616 as LucideEllipsisVertical,
  default491 as LucideEqual,
  default489 as LucideEqualApproximately,
  default490 as LucideEqualNot,
  default1847 as LucideEqualSquare,
  default492 as LucideEraser,
  default493 as LucideEthernetPort,
  default494 as LucideEuro,
  default495 as LucideEvCharger,
  default496 as LucideExpand,
  default497 as LucideExternalLink,
  default500 as LucideEye,
  default498 as LucideEyeClosed,
  default499 as LucideEyeOff,
  default501 as LucideFactory,
  default502 as LucideFan,
  default503 as LucideFastForward,
  default504 as LucideFeather,
  default505 as LucideFence,
  default506 as LucideFerrisWheel,
  default535 as LucideFile,
  default507 as LucideFileArchive,
  default1645 as LucideFileAudio,
  default1646 as LucideFileAudio2,
  default1621 as LucideFileAxis3D,
  default1620 as LucideFileAxis3d,
  default1622 as LucideFileBadge,
  default1623 as LucideFileBadge2,
  default1629 as LucideFileBarChart,
  default1631 as LucideFileBarChart2,
  default508 as LucideFileBox,
  default1626 as LucideFileBraces,
  default1624 as LucideFileBracesCorner,
  default1630 as LucideFileChartColumn,
  default1628 as LucideFileChartColumnIncreasing,
  default1632 as LucideFileChartLine,
  default1634 as LucideFileChartPie,
  default510 as LucideFileCheck,
  default1637 as LucideFileCheck2,
  default1636 as LucideFileCheckCorner,
  default509 as LucideFileClock,
  default511 as LucideFileCode,
  default1639 as LucideFileCode2,
  default1638 as LucideFileCodeCorner,
  default1640 as LucideFileCog,
  default1641 as LucideFileCog2,
  default512 as LucideFileDiff,
  default513 as LucideFileDigit,
  default514 as LucideFileDown,
  default1656 as LucideFileEdit,
  default1642 as LucideFileExclamationPoint,
  default1644 as LucideFileHeadphone,
  default515 as LucideFileHeart,
  default516 as LucideFileImage,
  default517 as LucideFileInput,
  default1627 as LucideFileJson,
  default1625 as LucideFileJson2,
  default1651 as LucideFileKey,
  default1652 as LucideFileKey2,
  default1633 as LucideFileLineChart,
  default1647 as LucideFileLock,
  default1648 as LucideFileLock2,
  default518 as LucideFileMinus,
  default1650 as LucideFileMinus2,
  default1649 as LucideFileMinusCorner,
  default519 as LucideFileMusic,
  default520 as LucideFileOutput,
  default1655 as LucideFilePen,
  default1653 as LucideFilePenLine,
  default1635 as LucideFilePieChart,
  default1657 as LucideFilePlay,
  default521 as LucideFilePlus,
  default1660 as LucideFilePlus2,
  default1659 as LucideFilePlusCorner,
  default1664 as LucideFileQuestion,
  default1663 as LucideFileQuestionMark,
  default522 as LucideFileScan,
  default523 as LucideFileSearch,
  default1662 as LucideFileSearch2,
  default1661 as LucideFileSearchCorner,
  default1665 as LucideFileSignal,
  default1654 as LucideFileSignature,
  default524 as LucideFileSliders,
  default525 as LucideFileSpreadsheet,
  default526 as LucideFileStack,
  default527 as LucideFileSymlink,
  default528 as LucideFileTerminal,
  default529 as LucideFileText,
  default530 as LucideFileType,
  default1668 as LucideFileType2,
  default1667 as LucideFileTypeCorner,
  default531 as LucideFileUp,
  default533 as LucideFileUser,
  default1658 as LucideFileVideo,
  default1670 as LucideFileVideo2,
  default1669 as LucideFileVideoCamera,
  default532 as LucideFileVolume,
  default1666 as LucideFileVolume2,
  default1643 as LucideFileWarning,
  default534 as LucideFileX,
  default1672 as LucideFileX2,
  default1671 as LucideFileXCorner,
  default536 as LucideFiles,
  default537 as LucideFilm,
  default1680 as LucideFilter,
  default1678 as LucideFilterX,
  default1465 as LucideFingerprint,
  default1464 as LucideFingerprintPattern,
  default539 as LucideFireExtinguisher,
  default541 as LucideFish,
  default540 as LucideFishOff,
  default538 as LucideFishSymbol,
  default542 as LucideFishingHook,
  default543 as LucideFishingRod,
  default547 as LucideFlag,
  default544 as LucideFlagOff,
  default545 as LucideFlagTriangleLeft,
  default546 as LucideFlagTriangleRight,
  default549 as LucideFlame,
  default548 as LucideFlameKindling,
  default551 as LucideFlashlight,
  default550 as LucideFlashlightOff,
  default552 as LucideFlaskConical,
  default553 as LucideFlaskConicalOff,
  default554 as LucideFlaskRound,
  default1811 as LucideFlipHorizontal,
  default555 as LucideFlipHorizontal2,
  default1813 as LucideFlipVertical,
  default556 as LucideFlipVertical2,
  default558 as LucideFlower,
  default557 as LucideFlower2,
  default559 as LucideFocus,
  default560 as LucideFoldHorizontal,
  default561 as LucideFoldVertical,
  default589 as LucideFolder,
  default562 as LucideFolderArchive,
  default563 as LucideFolderCheck,
  default564 as LucideFolderClock,
  default565 as LucideFolderClosed,
  default566 as LucideFolderCode,
  default1673 as LucideFolderCog,
  default1674 as LucideFolderCog2,
  default567 as LucideFolderDot,
  default568 as LucideFolderDown,
  default1676 as LucideFolderEdit,
  default570 as LucideFolderGit,
  default569 as LucideFolderGit2,
  default571 as LucideFolderHeart,
  default572 as LucideFolderInput,
  default573 as LucideFolderKanban,
  default574 as LucideFolderKey,
  default576 as LucideFolderLock,
  default575 as LucideFolderMinus,
  default578 as LucideFolderOpen,
  default577 as LucideFolderOpenDot,
  default579 as LucideFolderOutput,
  default1675 as LucideFolderPen,
  default580 as LucideFolderPlus,
  default581 as LucideFolderRoot,
  default583 as LucideFolderSearch,
  default582 as LucideFolderSearch2,
  default584 as LucideFolderSymlink,
  default585 as LucideFolderSync,
  default586 as LucideFolderTree,
  default587 as LucideFolderUp,
  default588 as LucideFolderX,
  default590 as LucideFolders,
  default591 as LucideFootprints,
  default1940 as LucideForkKnife,
  default1938 as LucideForkKnifeCrossed,
  default592 as LucideForklift,
  default593 as LucideForm,
  default1761 as LucideFormInput,
  default594 as LucideForward,
  default595 as LucideFrame,
  default596 as LucideFrown,
  default597 as LucideFuel,
  default598 as LucideFullscreen,
  default1845 as LucideFunctionSquare,
  default1679 as LucideFunnel,
  default599 as LucideFunnelPlus,
  default1677 as LucideFunnelX,
  default601 as LucideGalleryHorizontal,
  default600 as LucideGalleryHorizontalEnd,
  default602 as LucideGalleryThumbnails,
  default603 as LucideGalleryVertical,
  default604 as LucideGalleryVerticalEnd,
  default607 as LucideGamepad,
  default605 as LucideGamepad2,
  default606 as LucideGamepadDirectional,
  default1523 as LucideGanttChart,
  default1815 as LucideGanttChartSquare,
  default608 as LucideGauge,
  default1563 as LucideGaugeCircle,
  default609 as LucideGavel,
  default610 as LucideGem,
  default611 as LucideGeorgianLari,
  default612 as LucideGhost,
  default613 as LucideGift,
  default616 as LucideGitBranch,
  default614 as LucideGitBranchMinus,
  default615 as LucideGitBranchPlus,
  default1682 as LucideGitCommit,
  default1681 as LucideGitCommitHorizontal,
  default617 as LucideGitCommitVertical,
  default619 as LucideGitCompare,
  default618 as LucideGitCompareArrows,
  default620 as LucideGitFork,
  default621 as LucideGitGraph,
  default623 as LucideGitMerge,
  default622 as LucideGitMergeConflict,
  default629 as LucideGitPullRequest,
  default624 as LucideGitPullRequestArrow,
  default626 as LucideGitPullRequestClosed,
  default627 as LucideGitPullRequestCreate,
  default625 as LucideGitPullRequestCreateArrow,
  default628 as LucideGitPullRequestDraft,
  default630 as LucideGlassWater,
  default631 as LucideGlasses,
  default635 as LucideGlobe,
  default1615 as LucideGlobe2,
  default632 as LucideGlobeLock,
  default633 as LucideGlobeOff,
  default634 as LucideGlobeX,
  default636 as LucideGoal,
  default637 as LucideGpu,
  default1697 as LucideGrab,
  default638 as LucideGraduationCap,
  default639 as LucideGrape,
  default1692 as LucideGrid,
  default1688 as LucideGrid2X2,
  default1684 as LucideGrid2X2Check,
  default1686 as LucideGrid2X2Plus,
  default1690 as LucideGrid2X2X,
  default1687 as LucideGrid2x2,
  default1683 as LucideGrid2x2Check,
  default1685 as LucideGrid2x2Plus,
  default1689 as LucideGrid2x2X,
  default1693 as LucideGrid3X3,
  default640 as LucideGrid3x2,
  default1691 as LucideGrid3x3,
  default643 as LucideGrip,
  default641 as LucideGripHorizontal,
  default642 as LucideGripVertical,
  default644 as LucideGroup,
  default645 as LucideGuitar,
  default646 as LucideHam,
  default649 as LucideHamburger,
  default647 as LucideHammer,
  default654 as LucideHand,
  default648 as LucideHandCoins,
  default650 as LucideHandFist,
  default1696 as LucideHandGrab,
  default651 as LucideHandHeart,
  default1694 as LucideHandHelping,
  default652 as LucideHandMetal,
  default653 as LucideHandPlatter,
  default655 as LucideHandbag,
  default656 as LucideHandshake,
  default659 as LucideHardDrive,
  default657 as LucideHardDriveDownload,
  default658 as LucideHardDriveUpload,
  default660 as LucideHardHat,
  default661 as LucideHash,
  default662 as LucideHatGlasses,
  default664 as LucideHaze,
  default663 as LucideHd,
  default665 as LucideHdmiPort,
  default672 as LucideHeading,
  default666 as LucideHeading1,
  default667 as LucideHeading2,
  default669 as LucideHeading3,
  default668 as LucideHeading4,
  default670 as LucideHeading5,
  default671 as LucideHeading6,
  default673 as LucideHeadphoneOff,
  default674 as LucideHeadphones,
  default675 as LucideHeadset,
  default683 as LucideHeart,
  default676 as LucideHeartCrack,
  default677 as LucideHeartHandshake,
  default678 as LucideHeartMinus,
  default679 as LucideHeartOff,
  default680 as LucideHeartPlus,
  default681 as LucideHeartPulse,
  default682 as LucideHeartX,
  default684 as LucideHeater,
  default685 as LucideHelicopter,
  default1581 as LucideHelpCircle,
  default1695 as LucideHelpingHand,
  default686 as LucideHexagon,
  default687 as LucideHighlighter,
  default688 as LucideHistory,
  default1699 as LucideHome,
  default690 as LucideHop,
  default689 as LucideHopOff,
  default691 as LucideHospital,
  default692 as LucideHotel,
  default693 as LucideHourglass,
  default1698 as LucideHouse,
  default694 as LucideHouseHeart,
  default697 as LucideHousePlug,
  default695 as LucideHousePlus,
  default696 as LucideHouseWifi,
  default1703 as LucideIceCream,
  default1701 as LucideIceCream2,
  default1700 as LucideIceCreamBowl,
  default1702 as LucideIceCreamCone,
  default699 as LucideIdCard,
  default698 as LucideIdCardLanyard,
  default707 as LucideImage,
  default700 as LucideImageDown,
  default701 as LucideImageMinus,
  default702 as LucideImageOff,
  default703 as LucideImagePlay,
  default704 as LucideImagePlus,
  default705 as LucideImageUp,
  default706 as LucideImageUpscale,
  default708 as LucideImages,
  default709 as LucideImport,
  default710 as LucideInbox,
  default1712 as LucideIndent,
  default1710 as LucideIndentDecrease,
  default1713 as LucideIndentIncrease,
  default711 as LucideIndianRupee,
  default712 as LucideInfinity,
  default713 as LucideInfo,
  default1859 as LucideInspect,
  default715 as LucideInspectionPanel,
  default714 as LucideItalic,
  default716 as LucideIterationCcw,
  default717 as LucideIterationCw,
  default718 as LucideJapaneseYen,
  default719 as LucideJoystick,
  default720 as LucideKanban,
  default1849 as LucideKanbanSquare,
  default1832 as LucideKanbanSquareDashed,
  default721 as LucideKayak,
  default724 as LucideKey,
  default722 as LucideKeyRound,
  default723 as LucideKeySquare,
  default727 as LucideKeyboard,
  default725 as LucideKeyboardMusic,
  default726 as LucideKeyboardOff,
  default733 as LucideLamp,
  default728 as LucideLampCeiling,
  default731 as LucideLampDesk,
  default729 as LucideLampFloor,
  default730 as LucideLampWallDown,
  default732 as LucideLampWallUp,
  default734 as LucideLandPlot,
  default735 as LucideLandmark,
  default736 as LucideLanguages,
  default738 as LucideLaptop,
  default1705 as LucideLaptop2,
  default1704 as LucideLaptopMinimal,
  default737 as LucideLaptopMinimalCheck,
  default740 as LucideLasso,
  default739 as LucideLassoSelect,
  default741 as LucideLaugh,
  default1706 as LucideLayers,
  default742 as LucideLayers2,
  default1707 as LucideLayers3,
  default743 as LucideLayersMinus,
  default744 as LucideLayersPlus,
  default1753 as LucideLayout,
  default745 as LucideLayoutDashboard,
  default746 as LucideLayoutGrid,
  default747 as LucideLayoutList,
  default748 as LucideLayoutPanelLeft,
  default749 as LucideLayoutPanelTop,
  default750 as LucideLayoutTemplate,
  default751 as LucideLeaf,
  default753 as LucideLeafyGreen,
  default752 as LucideLectern,
  default754 as LucideLensConcave,
  default756 as LucideLensConvex,
  default1910 as LucideLetterText,
  default757 as LucideLibrary,
  default755 as LucideLibraryBig,
  default1851 as LucideLibrarySquare,
  default758 as LucideLifeBuoy,
  default759 as LucideLigature,
  default761 as LucideLightbulb,
  default760 as LucideLightbulbOff,
  default1521 as LucideLineChart,
  default762 as LucideLineDotRightHorizontal,
  default763 as LucideLineSquiggle,
  default764 as LucideLineStyle,
  default767 as LucideLink,
  default766 as LucideLink2,
  default765 as LucideLink2Off,
  default786 as LucideList,
  default768 as LucideListCheck,
  default769 as LucideListChecks,
  default770 as LucideListChevronsDownUp,
  default771 as LucideListChevronsUpDown,
  default772 as LucideListCollapse,
  default773 as LucideListEnd,
  default775 as LucideListFilter,
  default774 as LucideListFilterPlus,
  default1708 as LucideListIndentDecrease,
  default1711 as LucideListIndentIncrease,
  default776 as LucideListMinus,
  default777 as LucideListMusic,
  default778 as LucideListOrdered,
  default779 as LucideListPlus,
  default780 as LucideListRestart,
  default781 as LucideListStart,
  default782 as LucideListTodo,
  default783 as LucideListTree,
  default784 as LucideListVideo,
  default785 as LucideListX,
  default788 as LucideLoader,
  default1715 as LucideLoader2,
  default1714 as LucideLoaderCircle,
  default787 as LucideLoaderPinwheel,
  default791 as LucideLocate,
  default789 as LucideLocateFixed,
  default790 as LucideLocateOff,
  default1723 as LucideLocationEdit,
  default793 as LucideLock,
  default792 as LucideLockKeyhole,
  default1718 as LucideLockKeyholeOpen,
  default1716 as LucideLockOpen,
  default794 as LucideLogIn,
  default795 as LucideLogOut,
  default796 as LucideLogs,
  default797 as LucideLollipop,
  default798 as LucideLuggage,
  default1853 as LucideMSquare,
  default799 as LucideMagnet,
  default807 as LucideMail,
  default800 as LucideMailCheck,
  default801 as LucideMailMinus,
  default805 as LucideMailOpen,
  default802 as LucideMailPlus,
  default1721 as LucideMailQuestion,
  default1720 as LucideMailQuestionMark,
  default803 as LucideMailSearch,
  default804 as LucideMailWarning,
  default806 as LucideMailX,
  default808 as LucideMailbox,
  default809 as LucideMails,
  default825 as LucideMap,
  default811 as LucideMapMinus,
  default822 as LucideMapPin,
  default812 as LucideMapPinCheck,
  default810 as LucideMapPinCheckInside,
  default813 as LucideMapPinHouse,
  default815 as LucideMapPinMinus,
  default814 as LucideMapPinMinusInside,
  default816 as LucideMapPinOff,
  default1722 as LucideMapPinPen,
  default818 as LucideMapPinPlus,
  default817 as LucideMapPinPlusInside,
  default819 as LucideMapPinSearch,
  default821 as LucideMapPinX,
  default820 as LucideMapPinXInside,
  default823 as LucideMapPinned,
  default824 as LucideMapPlus,
  default828 as LucideMars,
  default826 as LucideMarsStroke,
  default827 as LucideMartini,
  default832 as LucideMaximize,
  default829 as LucideMaximize2,
  default831 as LucideMedal,
  default833 as LucideMegaphone,
  default830 as LucideMegaphoneOff,
  default834 as LucideMeh,
  default835 as LucideMemoryStick,
  default836 as LucideMenu,
  default1855 as LucideMenuSquare,
  default837 as LucideMerge,
  default849 as LucideMessageCircle,
  default838 as LucideMessageCircleCheck,
  default839 as LucideMessageCircleCode,
  default840 as LucideMessageCircleDashed,
  default841 as LucideMessageCircleHeart,
  default842 as LucideMessageCircleMore,
  default843 as LucideMessageCircleOff,
  default844 as LucideMessageCirclePlus,
  default1725 as LucideMessageCircleQuestion,
  default1724 as LucideMessageCircleQuestionMark,
  default845 as LucideMessageCircleReply,
  default846 as LucideMessageCircleWarning,
  default847 as LucideMessageCircleX,
  default865 as LucideMessageSquare,
  default848 as LucideMessageSquareCheck,
  default850 as LucideMessageSquareCode,
  default851 as LucideMessageSquareDashed,
  default852 as LucideMessageSquareDiff,
  default853 as LucideMessageSquareDot,
  default854 as LucideMessageSquareHeart,
  default855 as LucideMessageSquareLock,
  default856 as LucideMessageSquareMore,
  default857 as LucideMessageSquareOff,
  default858 as LucideMessageSquarePlus,
  default859 as LucideMessageSquareQuote,
  default860 as LucideMessageSquareReply,
  default861 as LucideMessageSquareShare,
  default862 as LucideMessageSquareText,
  default863 as LucideMessageSquareWarning,
  default864 as LucideMessageSquareX,
  default866 as LucideMessagesSquare,
  default867 as LucideMetronome,
  default869 as LucideMic,
  default1727 as LucideMic2,
  default868 as LucideMicOff,
  default1726 as LucideMicVocal,
  default870 as LucideMicrochip,
  default871 as LucideMicroscope,
  default872 as LucideMicrowave,
  default873 as LucideMilestone,
  default875 as LucideMilk,
  default874 as LucideMilkOff,
  default877 as LucideMinimize,
  default876 as LucideMinimize2,
  default878 as LucideMinus,
  default1565 as LucideMinusCircle,
  default1857 as LucideMinusSquare,
  default879 as LucideMirrorRectangular,
  default880 as LucideMirrorRound,
  default894 as LucideMonitor,
  default881 as LucideMonitorCheck,
  default882 as LucideMonitorCloud,
  default883 as LucideMonitorCog,
  default884 as LucideMonitorDot,
  default885 as LucideMonitorDown,
  default886 as LucideMonitorOff,
  default887 as LucideMonitorPause,
  default888 as LucideMonitorPlay,
  default889 as LucideMonitorSmartphone,
  default890 as LucideMonitorSpeaker,
  default891 as LucideMonitorStop,
  default892 as LucideMonitorUp,
  default893 as LucideMonitorX,
  default896 as LucideMoon,
  default895 as LucideMoonStar,
  default1619 as LucideMoreHorizontal,
  default1617 as LucideMoreVertical,
  default897 as LucideMotorbike,
  default899 as LucideMountain,
  default898 as LucideMountainSnow,
  default908 as LucideMouse,
  default900 as LucideMouseLeft,
  default901 as LucideMouseOff,
  default906 as LucideMousePointer,
  default903 as LucideMousePointer2,
  default902 as LucideMousePointer2Off,
  default904 as LucideMousePointerBan,
  default905 as LucideMousePointerClick,
  default1834 as LucideMousePointerSquareDashed,
  default907 as LucideMouseRight,
  default921 as LucideMove,
  default1729 as LucideMove3D,
  default1728 as LucideMove3d,
  default910 as LucideMoveDiagonal,
  default909 as LucideMoveDiagonal2,
  default913 as LucideMoveDown,
  default911 as LucideMoveDownLeft,
  default912 as LucideMoveDownRight,
  default914 as LucideMoveHorizontal,
  default915 as LucideMoveLeft,
  default916 as LucideMoveRight,
  default919 as LucideMoveUp,
  default917 as LucideMoveUpLeft,
  default918 as LucideMoveUpRight,
  default920 as LucideMoveVertical,
  default927 as LucideMusic,
  default922 as LucideMusic2,
  default923 as LucideMusic3,
  default924 as LucideMusic4,
  default929 as LucideNavigation,
  default925 as LucideNavigation2,
  default926 as LucideNavigation2Off,
  default928 as LucideNavigationOff,
  default931 as LucideNetwork,
  default930 as LucideNewspaper,
  default932 as LucideNfc,
  default933 as LucideNonBinary,
  default937 as LucideNotebook,
  default934 as LucideNotebookPen,
  default935 as LucideNotebookTabs,
  default936 as LucideNotebookText,
  default939 as LucideNotepadText,
  default938 as LucideNotepadTextDashed,
  default942 as LucideNut,
  default940 as LucideNutOff,
  default1060 as LucideOctagon,
  default1730 as LucideOctagonAlert,
  default941 as LucideOctagonMinus,
  default1732 as LucideOctagonPause,
  default1734 as LucideOctagonX,
  default944 as LucideOmega,
  default943 as LucideOption,
  default945 as LucideOrbit,
  default946 as LucideOrigami,
  default1709 as LucideOutdent,
  default955 as LucidePackage,
  default947 as LucidePackage2,
  default948 as LucidePackageCheck,
  default949 as LucidePackageMinus,
  default950 as LucidePackageOpen,
  default951 as LucidePackagePlus,
  default952 as LucidePackageSearch,
  default953 as LucidePackageX,
  default954 as LucidePaintBucket,
  default956 as LucidePaintRoller,
  default957 as LucidePaintbrush,
  default1737 as LucidePaintbrush2,
  default1736 as LucidePaintbrushVertical,
  default958 as LucidePalette,
  default1916 as LucidePalmtree,
  default959 as LucidePanda,
  default962 as LucidePanelBottom,
  default961 as LucidePanelBottomClose,
  default1738 as LucidePanelBottomDashed,
  default1739 as LucidePanelBottomInactive,
  default960 as LucidePanelBottomOpen,
  default1744 as LucidePanelLeft,
  default1742 as LucidePanelLeftClose,
  default1740 as LucidePanelLeftDashed,
  default1741 as LucidePanelLeftInactive,
  default1746 as LucidePanelLeftOpen,
  default963 as LucidePanelLeftRightDashed,
  default966 as LucidePanelRight,
  default964 as LucidePanelRightClose,
  default1748 as LucidePanelRightDashed,
  default1749 as LucidePanelRightInactive,
  default965 as LucidePanelRightOpen,
  default970 as LucidePanelTop,
  default967 as LucidePanelTopBottomDashed,
  default968 as LucidePanelTopClose,
  default1750 as LucidePanelTopDashed,
  default1751 as LucidePanelTopInactive,
  default969 as LucidePanelTopOpen,
  default971 as LucidePanelsLeftBottom,
  default1609 as LucidePanelsLeftRight,
  default972 as LucidePanelsRightBottom,
  default1767 as LucidePanelsTopBottom,
  default1752 as LucidePanelsTopLeft,
  default973 as LucidePaperclip,
  default974 as LucideParentheses,
  default1571 as LucideParkingCircle,
  default1567 as LucideParkingCircleOff,
  default975 as LucideParkingMeter,
  default1861 as LucideParkingSquare,
  default1869 as LucideParkingSquareOff,
  default976 as LucidePartyPopper,
  default978 as LucidePause,
  default1569 as LucidePauseCircle,
  default1733 as LucidePauseOctagon,
  default977 as LucidePawPrint,
  default979 as LucidePcCase,
  default1756 as LucidePen,
  default1865 as LucidePenBox,
  default1754 as LucidePenLine,
  default980 as LucidePenOff,
  default1867 as LucidePenSquare,
  default982 as LucidePenTool,
  default985 as LucidePencil,
  default981 as LucidePencilLine,
  default983 as LucidePencilOff,
  default984 as LucidePencilRuler,
  default986 as LucidePentagon,
  default987 as LucidePercent,
  default1573 as LucidePercentCircle,
  default1613 as LucidePercentDiamond,
  default1863 as LucidePercentSquare,
  default988 as LucidePersonStanding,
  default989 as LucidePhilippinePeso,
  default996 as LucidePhone,
  default990 as LucidePhoneCall,
  default991 as LucidePhoneForwarded,
  default992 as LucidePhoneIncoming,
  default993 as LucidePhoneMissed,
  default994 as LucidePhoneOff,
  default995 as LucidePhoneOutgoing,
  default997 as LucidePi,
  default1871 as LucidePiSquare,
  default998 as LucidePiano,
  default999 as LucidePickaxe,
  default1001 as LucidePictureInPicture,
  default1000 as LucidePictureInPicture2,
  default1527 as LucidePieChart,
  default1002 as LucidePiggyBank,
  default1005 as LucidePilcrow,
  default1003 as LucidePilcrowLeft,
  default1004 as LucidePilcrowRight,
  default1873 as LucidePilcrowSquare,
  default1007 as LucidePill,
  default1006 as LucidePillBottle,
  default1008 as LucidePin,
  default1011 as LucidePinOff,
  default1009 as LucidePipette,
  default1010 as LucidePizza,
  default1014 as LucidePlane,
  default1012 as LucidePlaneLanding,
  default1013 as LucidePlaneTakeoff,
  default1017 as LucidePlay,
  default1579 as LucidePlayCircle,
  default1875 as LucidePlaySquare,
  default1016 as LucidePlug,
  default1015 as LucidePlug2,
  default1758 as LucidePlugZap,
  default1759 as LucidePlugZap2,
  default1018 as LucidePlus,
  default1575 as LucidePlusCircle,
  default1877 as LucidePlusSquare,
  default1019 as LucidePocketKnife,
  default1020 as LucidePodcast,
  default1022 as LucidePointer,
  default1021 as LucidePointerOff,
  default1023 as LucidePopcorn,
  default1024 as LucidePopsicle,
  default1025 as LucidePoundSterling,
  default1027 as LucidePower,
  default1577 as LucidePowerCircle,
  default1026 as LucidePowerOff,
  default1883 as LucidePowerSquare,
  default1029 as LucidePresentation,
  default1032 as LucidePrinter,
  default1028 as LucidePrinterCheck,
  default1030 as LucidePrinterX,
  default1031 as LucideProjector,
  default1033 as LucideProportions,
  default1034 as LucidePuzzle,
  default1035 as LucidePyramid,
  default1036 as LucideQrCode,
  default1037 as LucideQuote,
  default1038 as LucideRabbit,
  default1039 as LucideRadar,
  default1040 as LucideRadiation,
  default1041 as LucideRadical,
  default1045 as LucideRadio,
  default1042 as LucideRadioOff,
  default1043 as LucideRadioReceiver,
  default1044 as LucideRadioTower,
  default1046 as LucideRadius,
  default1047 as LucideRainbow,
  default1048 as LucideRat,
  default1049 as LucideRatio,
  default1059 as LucideReceipt,
  default1050 as LucideReceiptCent,
  default1051 as LucideReceiptEuro,
  default1052 as LucideReceiptIndianRupee,
  default1053 as LucideReceiptJapaneseYen,
  default1054 as LucideReceiptPoundSterling,
  default1055 as LucideReceiptRussianRuble,
  default1056 as LucideReceiptSwissFranc,
  default1057 as LucideReceiptText,
  default1058 as LucideReceiptTurkishLira,
  default1061 as LucideRectangleCircle,
  default1760 as LucideRectangleEllipsis,
  default1062 as LucideRectangleGoggles,
  default1063 as LucideRectangleHorizontal,
  default1064 as LucideRectangleVertical,
  default1065 as LucideRecycle,
  default1068 as LucideRedo,
  default1066 as LucideRedo2,
  default1067 as LucideRedoDot,
  default1069 as LucideRefreshCcw,
  default1070 as LucideRefreshCcwDot,
  default1072 as LucideRefreshCw,
  default1071 as LucideRefreshCwOff,
  default1073 as LucideRefrigerator,
  default1074 as LucideRegex,
  default1075 as LucideRemoveFormatting,
  default1078 as LucideRepeat,
  default1076 as LucideRepeat1,
  default1077 as LucideRepeat2,
  default1080 as LucideReplace,
  default1079 as LucideReplaceAll,
  default1082 as LucideReply,
  default1081 as LucideReplyAll,
  default1083 as LucideRewind,
  default1084 as LucideRibbon,
  default1085 as LucideRoad,
  default1086 as LucideRocket,
  default1087 as LucideRockingChair,
  default1088 as LucideRollerCoaster,
  default1089 as LucideRose,
  default1763 as LucideRotate3D,
  default1762 as LucideRotate3d,
  default1092 as LucideRotateCcw,
  default1090 as LucideRotateCcwKey,
  default1091 as LucideRotateCcwSquare,
  default1094 as LucideRotateCw,
  default1093 as LucideRotateCwSquare,
  default1096 as LucideRoute,
  default1095 as LucideRouteOff,
  default1097 as LucideRouter,
  default1765 as LucideRows,
  default1764 as LucideRows2,
  default1766 as LucideRows3,
  default1098 as LucideRows4,
  default1099 as LucideRss,
  default1101 as LucideRuler,
  default1100 as LucideRulerDimensionLine,
  default1102 as LucideRussianRuble,
  default1103 as LucideSailboat,
  default1104 as LucideSalad,
  default1105 as LucideSandwich,
  default1107 as LucideSatellite,
  default1106 as LucideSatelliteDish,
  default1108 as LucideSaudiRiyal,
  default1111 as LucideSave,
  default1109 as LucideSaveAll,
  default1110 as LucideSaveOff,
  default1112 as LucideScale,
  default1769 as LucideScale3D,
  default1768 as LucideScale3d,
  default1113 as LucideScaling,
  default1122 as LucideScan,
  default1114 as LucideScanBarcode,
  default1116 as LucideScanEye,
  default1115 as LucideScanFace,
  default1117 as LucideScanHeart,
  default1118 as LucideScanLine,
  default1119 as LucideScanQrCode,
  default1120 as LucideScanSearch,
  default1121 as LucideScanText,
  default1529 as LucideScatterChart,
  default1123 as LucideSchool,
  default1922 as LucideSchool2,
  default1125 as LucideScissors,
  default1124 as LucideScissorsLineDashed,
  default1879 as LucideScissorsSquare,
  default1807 as LucideScissorsSquareDashedBottom,
  default1126 as LucideScooter,
  default1128 as LucideScreenShare,
  default1127 as LucideScreenShareOff,
  default1130 as LucideScroll,
  default1129 as LucideScrollText,
  default1136 as LucideSearch,
  default1131 as LucideSearchAlert,
  default1132 as LucideSearchCheck,
  default1133 as LucideSearchCode,
  default1134 as LucideSearchSlash,
  default1135 as LucideSearchX,
  default1137 as LucideSection,
  default1139 as LucideSend,
  default1771 as LucideSendHorizonal,
  default1770 as LucideSendHorizontal,
  default1138 as LucideSendToBack,
  default1140 as LucideSeparatorHorizontal,
  default1142 as LucideSeparatorVertical,
  default1145 as LucideServer,
  default1141 as LucideServerCog,
  default1143 as LucideServerCrash,
  default1144 as LucideServerOff,
  default1147 as LucideSettings,
  default1146 as LucideSettings2,
  default1150 as LucideShapes,
  default1148 as LucideShare,
  default1149 as LucideShare2,
  default1152 as LucideSheet,
  default1151 as LucideShell,
  default1153 as LucideShelvingUnit,
  default1165 as LucideShield,
  default1154 as LucideShieldAlert,
  default1155 as LucideShieldBan,
  default1156 as LucideShieldCheck,
  default1775 as LucideShieldClose,
  default1158 as LucideShieldCog,
  default1157 as LucideShieldCogCorner,
  default1159 as LucideShieldEllipsis,
  default1160 as LucideShieldHalf,
  default1161 as LucideShieldMinus,
  default1162 as LucideShieldOff,
  default1163 as LucideShieldPlus,
  default1773 as LucideShieldQuestion,
  default1772 as LucideShieldQuestionMark,
  default1164 as LucideShieldUser,
  default1774 as LucideShieldX,
  default1167 as LucideShip,
  default1166 as LucideShipWheel,
  default1168 as LucideShirt,
  default1169 as LucideShoppingBag,
  default1170 as LucideShoppingBasket,
  default1171 as LucideShoppingCart,
  default1172 as LucideShovel,
  default1173 as LucideShowerHead,
  default1174 as LucideShredder,
  default1175 as LucideShrimp,
  default1176 as LucideShrink,
  default1177 as LucideShrub,
  default1178 as LucideShuffle,
  default1745 as LucideSidebar,
  default1743 as LucideSidebarClose,
  default1747 as LucideSidebarOpen,
  default1179 as LucideSigma,
  default1881 as LucideSigmaSquare,
  default1184 as LucideSignal,
  default1180 as LucideSignalHigh,
  default1181 as LucideSignalLow,
  default1182 as LucideSignalMedium,
  default1183 as LucideSignalZero,
  default1185 as LucideSignature,
  default1187 as LucideSignpost,
  default1186 as LucideSignpostBig,
  default1188 as LucideSiren,
  default1189 as LucideSkipBack,
  default1190 as LucideSkipForward,
  default1191 as LucideSkull,
  default1192 as LucideSlash,
  default1885 as LucideSlashSquare,
  default1193 as LucideSlice,
  default1777 as LucideSliders,
  default1194 as LucideSlidersHorizontal,
  default1776 as LucideSlidersVertical,
  default1197 as LucideSmartphone,
  default1195 as LucideSmartphoneCharging,
  default1196 as LucideSmartphoneNfc,
  default1199 as LucideSmile,
  default1198 as LucideSmilePlus,
  default1200 as LucideSnail,
  default1201 as LucideSnowflake,
  default1202 as LucideSoapDispenserDroplet,
  default1203 as LucideSofa,
  default1204 as LucideSolarPanel,
  default1485 as LucideSortAsc,
  default1479 as LucideSortDesc,
  default1205 as LucideSoup,
  default1206 as LucideSpace,
  default1207 as LucideSpade,
  default1208 as LucideSparkle,
  default1778 as LucideSparkles,
  default1209 as LucideSpeaker,
  default1210 as LucideSpeech,
  default1212 as LucideSpellCheck,
  default1211 as LucideSpellCheck2,
  default1214 as LucideSpline,
  default1213 as LucideSplinePointer,
  default1215 as LucideSplit,
  default1887 as LucideSplitSquareHorizontal,
  default1889 as LucideSplitSquareVertical,
  default1216 as LucideSpool,
  default1217 as LucideSportShoe,
  default1218 as LucideSpotlight,
  default1219 as LucideSprayCan,
  default1220 as LucideSprout,
  default1233 as LucideSquare,
  default1780 as LucideSquareActivity,
  default1786 as LucideSquareArrowDown,
  default1782 as LucideSquareArrowDownLeft,
  default1784 as LucideSquareArrowDownRight,
  default1788 as LucideSquareArrowLeft,
  default1794 as LucideSquareArrowOutDownLeft,
  default1790 as LucideSquareArrowOutDownRight,
  default1792 as LucideSquareArrowOutUpLeft,
  default1796 as LucideSquareArrowOutUpRight,
  default1798 as LucideSquareArrowRight,
  default1221 as LucideSquareArrowRightEnter,
  default1222 as LucideSquareArrowRightExit,
  default1804 as LucideSquareArrowUp,
  default1800 as LucideSquareArrowUpLeft,
  default1802 as LucideSquareArrowUpRight,
  default1808 as LucideSquareAsterisk,
  default1806 as LucideSquareBottomDashedScissors,
  default1810 as LucideSquareCenterlineDashedHorizontal,
  default1812 as LucideSquareCenterlineDashedVertical,
  default1814 as LucideSquareChartGantt,
  default1821 as LucideSquareCheck,
  default1819 as LucideSquareCheckBig,
  default1817 as LucideSquareChevronDown,
  default1823 as LucideSquareChevronLeft,
  default1825 as LucideSquareChevronRight,
  default1827 as LucideSquareChevronUp,
  default1829 as LucideSquareCode,
  default1838 as LucideSquareDashed,
  default1224 as LucideSquareDashedBottom,
  default1223 as LucideSquareDashedBottomCode,
  default1831 as LucideSquareDashedKanban,
  default1833 as LucideSquareDashedMousePointer,
  default1835 as LucideSquareDashedText,
  default1225 as LucideSquareDashedTopSolid,
  default1840 as LucideSquareDivide,
  default1842 as LucideSquareDot,
  default1846 as LucideSquareEqual,
  default1844 as LucideSquareFunction,
  default1816 as LucideSquareGanttChart,
  default1848 as LucideSquareKanban,
  default1850 as LucideSquareLibrary,
  default1852 as LucideSquareM,
  default1854 as LucideSquareMenu,
  default1856 as LucideSquareMinus,
  default1858 as LucideSquareMousePointer,
  default1860 as LucideSquareParking,
  default1868 as LucideSquareParkingOff,
  default1226 as LucideSquarePause,
  default1864 as LucideSquarePen,
  default1862 as LucideSquarePercent,
  default1870 as LucideSquarePi,
  default1872 as LucideSquarePilcrow,
  default1874 as LucideSquarePlay,
  default1876 as LucideSquarePlus,
  default1882 as LucideSquarePower,
  default1227 as LucideSquareRadical,
  default1228 as LucideSquareRoundCorner,
  default1878 as LucideSquareScissors,
  default1880 as LucideSquareSigma,
  default1884 as LucideSquareSlash,
  default1886 as LucideSquareSplitHorizontal,
  default1888 as LucideSquareSplitVertical,
  default1229 as LucideSquareSquare,
  default1230 as LucideSquareStack,
  default1231 as LucideSquareStar,
  default1232 as LucideSquareStop,
  default1890 as LucideSquareTerminal,
  default1894 as LucideSquareUser,
  default1892 as LucideSquareUserRound,
  default1896 as LucideSquareX,
  default1234 as LucideSquaresExclude,
  default1235 as LucideSquaresIntersect,
  default1236 as LucideSquaresSubtract,
  default1237 as LucideSquaresUnite,
  default1239 as LucideSquircle,
  default1238 as LucideSquircleDashed,
  default1240 as LucideSquirrel,
  default1241 as LucideStamp,
  default1244 as LucideStar,
  default1242 as LucideStarHalf,
  default1243 as LucideStarOff,
  default1779 as LucideStars,
  default1245 as LucideStepBack,
  default1246 as LucideStepForward,
  default1248 as LucideStethoscope,
  default1247 as LucideSticker,
  default1249 as LucideStickyNote,
  default1252 as LucideStone,
  default1586 as LucideStopCircle,
  default1250 as LucideStore,
  default1251 as LucideStretchHorizontal,
  default1253 as LucideStretchVertical,
  default1254 as LucideStrikethrough,
  default1255 as LucideSubscript,
  default1503 as LucideSubtitles,
  default1260 as LucideSun,
  default1256 as LucideSunDim,
  default1257 as LucideSunMedium,
  default1258 as LucideSunMoon,
  default1259 as LucideSunSnow,
  default1261 as LucideSunrise,
  default1262 as LucideSunset,
  default1263 as LucideSuperscript,
  default1264 as LucideSwatchBook,
  default1265 as LucideSwissFranc,
  default1266 as LucideSwitchCamera,
  default1267 as LucideSword,
  default1268 as LucideSwords,
  default1269 as LucideSyringe,
  default1277 as LucideTable,
  default1271 as LucideTable2,
  default1270 as LucideTableCellsMerge,
  default1272 as LucideTableCellsSplit,
  default1273 as LucideTableColumnsSplit,
  default1607 as LucideTableConfig,
  default1274 as LucideTableOfContents,
  default1275 as LucideTableProperties,
  default1276 as LucideTableRowsSplit,
  default1279 as LucideTablet,
  default1278 as LucideTabletSmartphone,
  default1280 as LucideTablets,
  default1281 as LucideTag,
  default1283 as LucideTags,
  default1282 as LucideTally1,
  default1284 as LucideTally2,
  default1285 as LucideTally3,
  default1286 as LucideTally4,
  default1287 as LucideTally5,
  default1288 as LucideTangent,
  default1289 as LucideTarget,
  default1290 as LucideTelescope,
  default1292 as LucideTent,
  default1291 as LucideTentTree,
  default1293 as LucideTerminal,
  default1891 as LucideTerminalSquare,
  default1294 as LucideTestTube,
  default1899 as LucideTestTube2,
  default1898 as LucideTestTubeDiagonal,
  default1295 as LucideTestTubes,
  default1907 as LucideText,
  default1900 as LucideTextAlignCenter,
  default1902 as LucideTextAlignEnd,
  default1904 as LucideTextAlignJustify,
  default1906 as LucideTextAlignStart,
  default1297 as LucideTextCursor,
  default1296 as LucideTextCursorInput,
  default1909 as LucideTextInitial,
  default1298 as LucideTextQuote,
  default1299 as LucideTextSearch,
  default1837 as LucideTextSelect,
  default1836 as LucideTextSelection,
  default1911 as LucideTextWrap,
  default1300 as LucideTheater,
  default1303 as LucideThermometer,
  default1301 as LucideThermometerSnowflake,
  default1302 as LucideThermometerSun,
  default1304 as LucideThumbsDown,
  default1305 as LucideThumbsUp,
  default1312 as LucideTicket,
  default1306 as LucideTicketCheck,
  default1307 as LucideTicketMinus,
  default1308 as LucideTicketPercent,
  default1309 as LucideTicketPlus,
  default1310 as LucideTicketSlash,
  default1311 as LucideTicketX,
  default1314 as LucideTickets,
  default1313 as LucideTicketsPlane,
  default1315 as LucideTimeline,
  default1317 as LucideTimer,
  default1316 as LucideTimerOff,
  default1318 as LucideTimerReset,
  default1319 as LucideToggleLeft,
  default1320 as LucideToggleRight,
  default1321 as LucideToilet,
  default1322 as LucideToolCase,
  default1323 as LucideToolbox,
  default1324 as LucideTornado,
  default1325 as LucideTorus,
  default1327 as LucideTouchpad,
  default1326 as LucideTouchpadOff,
  default1328 as LucideTowelRack,
  default1329 as LucideTowerControl,
  default1330 as LucideToyBrick,
  default1331 as LucideTractor,
  default1333 as LucideTrafficCone,
  default1914 as LucideTrain,
  default1334 as LucideTrainFront,
  default1332 as LucideTrainFrontTunnel,
  default1335 as LucideTrainTrack,
  default1913 as LucideTramFront,
  default1336 as LucideTransgender,
  default1338 as LucideTrash,
  default1337 as LucideTrash2,
  default1339 as LucideTreeDeciduous,
  default1915 as LucideTreePalm,
  default1340 as LucideTreePine,
  default1341 as LucideTrees,
  default1342 as LucideTrendingDown,
  default1344 as LucideTrendingUp,
  default1343 as LucideTrendingUpDown,
  default1347 as LucideTriangle,
  default1917 as LucideTriangleAlert,
  default1345 as LucideTriangleDashed,
  default1346 as LucideTriangleRight,
  default1348 as LucideTrophy,
  default1350 as LucideTruck,
  default1349 as LucideTruckElectric,
  default1351 as LucideTurkishLira,
  default1352 as LucideTurntable,
  default1353 as LucideTurtle,
  default1355 as LucideTv,
  default1920 as LucideTv2,
  default1919 as LucideTvMinimal,
  default1354 as LucideTvMinimalPlay,
  default1357 as LucideType,
  default1356 as LucideTypeOutline,
  default1359 as LucideUmbrella,
  default1358 as LucideUmbrellaOff,
  default1360 as LucideUnderline,
  default1363 as LucideUndo,
  default1361 as LucideUndo2,
  default1362 as LucideUndoDot,
  default1364 as LucideUnfoldHorizontal,
  default1365 as LucideUnfoldVertical,
  default1366 as LucideUngroup,
  default1921 as LucideUniversity,
  default1368 as LucideUnlink,
  default1367 as LucideUnlink2,
  default1717 as LucideUnlock,
  default1719 as LucideUnlockKeyhole,
  default1369 as LucideUnplug,
  default1370 as LucideUpload,
  default1600 as LucideUploadCloud,
  default1371 as LucideUsb,
  default1385 as LucideUser,
  default1934 as LucideUser2,
  default1372 as LucideUserCheck,
  default1924 as LucideUserCheck2,
  default1590 as LucideUserCircle,
  default1588 as LucideUserCircle2,
  default1373 as LucideUserCog,
  default1926 as LucideUserCog2,
  default1374 as LucideUserKey,
  default1375 as LucideUserLock,
  default1376 as LucideUserMinus,
  default1928 as LucideUserMinus2,
  default1377 as LucideUserPen,
  default1378 as LucideUserPlus,
  default1930 as LucideUserPlus2,
  default1933 as LucideUserRound,
  default1923 as LucideUserRoundCheck,
  default1925 as LucideUserRoundCog,
  default1379 as LucideUserRoundKey,
  default1927 as LucideUserRoundMinus,
  default1380 as LucideUserRoundPen,
  default1929 as LucideUserRoundPlus,
  default1381 as LucideUserRoundSearch,
  default1931 as LucideUserRoundX,
  default1382 as LucideUserSearch,
  default1895 as LucideUserSquare,
  default1893 as LucideUserSquare2,
  default1383 as LucideUserStar,
  default1384 as LucideUserX,
  default1932 as LucideUserX2,
  default1386 as LucideUsers,
  default1936 as LucideUsers2,
  default1935 as LucideUsersRound,
  default1939 as LucideUtensils,
  default1937 as LucideUtensilsCrossed,
  default1387 as LucideUtilityPole,
  default1388 as LucideVan,
  default1389 as LucideVariable,
  default1390 as LucideVault,
  default1391 as LucideVectorSquare,
  default1392 as LucideVegan,
  default1393 as LucideVenetianMask,
  default1395 as LucideVenus,
  default1394 as LucideVenusAndMars,
  default1491 as LucideVerified,
  default1397 as LucideVibrate,
  default1396 as LucideVibrateOff,
  default1399 as LucideVideo,
  default1398 as LucideVideoOff,
  default1400 as LucideVideotape,
  default1401 as LucideView,
  default1403 as LucideVoicemail,
  default1402 as LucideVolleyball,
  default1408 as LucideVolume,
  default1404 as LucideVolume1,
  default1405 as LucideVolume2,
  default1406 as LucideVolumeOff,
  default1407 as LucideVolumeX,
  default1409 as LucideVote,
  default1411 as LucideWallet,
  default1942 as LucideWallet2,
  default1410 as LucideWalletCards,
  default1941 as LucideWalletMinimal,
  default1412 as LucideWallpaper,
  default1413 as LucideWand,
  default1944 as LucideWand2,
  default1943 as LucideWandSparkles,
  default1414 as LucideWarehouse,
  default1415 as LucideWashingMachine,
  default1416 as LucideWatch,
  default1420 as LucideWaves,
  default1417 as LucideWavesArrowDown,
  default1418 as LucideWavesArrowUp,
  default1419 as LucideWavesLadder,
  default1421 as LucideWaypoints,
  default1422 as LucideWebcam,
  default1424 as LucideWebhook,
  default1423 as LucideWebhookOff,
  default1426 as LucideWeight,
  default1425 as LucideWeightTilde,
  default1427 as LucideWheat,
  default1428 as LucideWheatOff,
  default1429 as LucideWholeWord,
  default1437 as LucideWifi,
  default1430 as LucideWifiCog,
  default1431 as LucideWifiHigh,
  default1432 as LucideWifiLow,
  default1433 as LucideWifiOff,
  default1434 as LucideWifiPen,
  default1435 as LucideWifiSync,
  default1436 as LucideWifiZero,
  default1439 as LucideWind,
  default1438 as LucideWindArrowDown,
  default1441 as LucideWine,
  default1440 as LucideWineOff,
  default1442 as LucideWorkflow,
  default1444 as LucideWorm,
  default1912 as LucideWrapText,
  default1443 as LucideWrench,
  default1445 as LucideX,
  default1592 as LucideXCircle,
  default1446 as LucideXLineTop,
  default1735 as LucideXOctagon,
  default1897 as LucideXSquare,
  default1447 as LucideZap,
  default1448 as LucideZapOff,
  default1449 as LucideZodiacAquarius,
  default1450 as LucideZodiacAries,
  default1451 as LucideZodiacCancer,
  default1452 as LucideZodiacCapricorn,
  default1453 as LucideZodiacGemini,
  default1454 as LucideZodiacLeo,
  default1455 as LucideZodiacLibra,
  default1456 as LucideZodiacOphiuchus,
  default1457 as LucideZodiacPisces,
  default1458 as LucideZodiacSagittarius,
  default1459 as LucideZodiacScorpio,
  default1460 as LucideZodiacTaurus,
  default1462 as LucideZodiacVirgo,
  default1461 as LucideZoomIn,
  default1463 as LucideZoomOut
};
