// lucide-solid/dist/source/aliases/suffixed.jsx
/**
* @license lucide-solid v1.11.0 - ISC
*
* This source code is licensed under the ISC license.
* See the LICENSE file in the root directory of this source tree.
*/

// src/aliases/suffixed.ts
import { default as default2 } from "../icons/a-arrow-down";
import { default as default3 } from "../icons/a-arrow-up";
import { default as default4 } from "../icons/a-large-small";
import { default as default5 } from "../icons/accessibility";
import { default as default6 } from "../icons/activity";
import { default as default7 } from "../icons/airplay";
import { default as default8 } from "../icons/air-vent";
import { default as default9 } from "../icons/alarm-clock-off";
import { default as default10 } from "../icons/alarm-clock";
import { default as default11 } from "../icons/alarm-smoke";
import { default as default12 } from "../icons/album";
import { default as default13 } from "../icons/align-center-horizontal";
import { default as default14 } from "../icons/align-center-vertical";
import { default as default15 } from "../icons/align-end-horizontal";
import { default as default16 } from "../icons/align-end-vertical";
import { default as default17 } from "../icons/align-horizontal-distribute-center";
import { default as default18 } from "../icons/align-horizontal-distribute-end";
import { default as default19 } from "../icons/align-horizontal-distribute-start";
import { default as default20 } from "../icons/align-horizontal-justify-center";
import { default as default21 } from "../icons/align-horizontal-justify-end";
import { default as default22 } from "../icons/align-horizontal-justify-start";
import { default as default23 } from "../icons/align-horizontal-space-around";
import { default as default24 } from "../icons/align-horizontal-space-between";
import { default as default25 } from "../icons/align-start-horizontal";
import { default as default26 } from "../icons/align-start-vertical";
import { default as default27 } from "../icons/align-vertical-distribute-center";
import { default as default28 } from "../icons/align-vertical-distribute-end";
import { default as default29 } from "../icons/align-vertical-distribute-start";
import { default as default30 } from "../icons/align-vertical-justify-center";
import { default as default31 } from "../icons/align-vertical-justify-end";
import { default as default32 } from "../icons/align-vertical-justify-start";
import { default as default33 } from "../icons/align-vertical-space-around";
import { default as default34 } from "../icons/align-vertical-space-between";
import { default as default35 } from "../icons/ambulance";
import { default as default36 } from "../icons/ampersand";
import { default as default37 } from "../icons/ampersands";
import { default as default38 } from "../icons/amphora";
import { default as default39 } from "../icons/angry";
import { default as default40 } from "../icons/anchor";
import { default as default41 } from "../icons/annoyed";
import { default as default42 } from "../icons/antenna";
import { default as default43 } from "../icons/anvil";
import { default as default44 } from "../icons/aperture";
import { default as default45 } from "../icons/app-window-mac";
import { default as default46 } from "../icons/app-window";
import { default as default47 } from "../icons/apple";
import { default as default48 } from "../icons/archive-restore";
import { default as default49 } from "../icons/archive-x";
import { default as default50 } from "../icons/armchair";
import { default as default51 } from "../icons/archive";
import { default as default52 } from "../icons/arrow-big-down-dash";
import { default as default53 } from "../icons/arrow-big-down";
import { default as default54 } from "../icons/arrow-big-left-dash";
import { default as default55 } from "../icons/arrow-big-right-dash";
import { default as default56 } from "../icons/arrow-big-left";
import { default as default57 } from "../icons/arrow-big-right";
import { default as default58 } from "../icons/arrow-big-up-dash";
import { default as default59 } from "../icons/arrow-big-up";
import { default as default60 } from "../icons/arrow-down-from-line";
import { default as default61 } from "../icons/arrow-down-left";
import { default as default62 } from "../icons/arrow-down-right";
import { default as default63 } from "../icons/arrow-down-narrow-wide";
import { default as default64 } from "../icons/arrow-down-to-dot";
import { default as default65 } from "../icons/arrow-down-to-line";
import { default as default66 } from "../icons/arrow-down-up";
import { default as default67 } from "../icons/arrow-down";
import { default as default68 } from "../icons/arrow-left-from-line";
import { default as default69 } from "../icons/arrow-left-right";
import { default as default70 } from "../icons/arrow-left-to-line";
import { default as default71 } from "../icons/arrow-left";
import { default as default72 } from "../icons/arrow-right-from-line";
import { default as default73 } from "../icons/arrow-right-left";
import { default as default74 } from "../icons/arrow-right-to-line";
import { default as default75 } from "../icons/arrow-right";
import { default as default76 } from "../icons/arrow-up-down";
import { default as default77 } from "../icons/arrow-up-from-dot";
import { default as default78 } from "../icons/arrow-up-left";
import { default as default79 } from "../icons/arrow-up-from-line";
import { default as default80 } from "../icons/arrow-up-right";
import { default as default81 } from "../icons/arrow-up-wide-narrow";
import { default as default82 } from "../icons/arrow-up-to-line";
import { default as default83 } from "../icons/arrow-up";
import { default as default84 } from "../icons/arrows-up-from-line";
import { default as default85 } from "../icons/asterisk";
import { default as default86 } from "../icons/atom";
import { default as default87 } from "../icons/at-sign";
import { default as default88 } from "../icons/audio-lines";
import { default as default89 } from "../icons/audio-waveform";
import { default as default90 } from "../icons/axe";
import { default as default91 } from "../icons/award";
import { default as default92 } from "../icons/backpack";
import { default as default93 } from "../icons/baby";
import { default as default94 } from "../icons/badge-alert";
import { default as default95 } from "../icons/badge-cent";
import { default as default96 } from "../icons/badge-dollar-sign";
import { default as default97 } from "../icons/badge-euro";
import { default as default98 } from "../icons/badge-indian-rupee";
import { default as default99 } from "../icons/badge-info";
import { default as default100 } from "../icons/badge-minus";
import { default as default101 } from "../icons/badge-percent";
import { default as default102 } from "../icons/badge-pound-sterling";
import { default as default103 } from "../icons/badge-japanese-yen";
import { default as default104 } from "../icons/badge-russian-ruble";
import { default as default105 } from "../icons/badge-plus";
import { default as default106 } from "../icons/badge-swiss-franc";
import { default as default107 } from "../icons/badge-turkish-lira";
import { default as default108 } from "../icons/badge-x";
import { default as default109 } from "../icons/baggage-claim";
import { default as default110 } from "../icons/badge";
import { default as default111 } from "../icons/ban";
import { default as default112 } from "../icons/balloon";
import { default as default113 } from "../icons/banana";
import { default as default114 } from "../icons/bandage";
import { default as default115 } from "../icons/banknote-arrow-down";
import { default as default116 } from "../icons/banknote-arrow-up";
import { default as default117 } from "../icons/banknote-x";
import { default as default118 } from "../icons/banknote";
import { default as default119 } from "../icons/barcode";
import { default as default120 } from "../icons/barrel";
import { default as default121 } from "../icons/baseline";
import { default as default122 } from "../icons/bath";
import { default as default123 } from "../icons/battery-charging";
import { default as default124 } from "../icons/battery-full";
import { default as default125 } from "../icons/battery-low";
import { default as default126 } from "../icons/battery-medium";
import { default as default127 } from "../icons/battery-plus";
import { default as default128 } from "../icons/battery-warning";
import { default as default129 } from "../icons/battery";
import { default as default130 } from "../icons/beaker";
import { default as default131 } from "../icons/bean-off";
import { default as default132 } from "../icons/bean";
import { default as default133 } from "../icons/bed-double";
import { default as default134 } from "../icons/bed-single";
import { default as default135 } from "../icons/bed";
import { default as default136 } from "../icons/beef";
import { default as default137 } from "../icons/beef-off";
import { default as default138 } from "../icons/beer-off";
import { default as default139 } from "../icons/beer";
import { default as default140 } from "../icons/bell-check";
import { default as default141 } from "../icons/bell-dot";
import { default as default142 } from "../icons/bell-electric";
import { default as default143 } from "../icons/bell-minus";
import { default as default144 } from "../icons/bell-off";
import { default as default145 } from "../icons/bell-ring";
import { default as default146 } from "../icons/bell-plus";
import { default as default147 } from "../icons/bell";
import { default as default148 } from "../icons/between-vertical-end";
import { default as default149 } from "../icons/between-vertical-start";
import { default as default150 } from "../icons/biceps-flexed";
import { default as default151 } from "../icons/bike";
import { default as default152 } from "../icons/binary";
import { default as default153 } from "../icons/biohazard";
import { default as default154 } from "../icons/binoculars";
import { default as default155 } from "../icons/bird";
import { default as default156 } from "../icons/birdhouse";
import { default as default157 } from "../icons/bitcoin";
import { default as default158 } from "../icons/blend";
import { default as default159 } from "../icons/blinds";
import { default as default160 } from "../icons/blocks";
import { default as default161 } from "../icons/bluetooth-connected";
import { default as default162 } from "../icons/bluetooth-off";
import { default as default163 } from "../icons/bluetooth-searching";
import { default as default164 } from "../icons/bluetooth";
import { default as default165 } from "../icons/bold";
import { default as default166 } from "../icons/bolt";
import { default as default167 } from "../icons/bomb";
import { default as default168 } from "../icons/bone";
import { default as default169 } from "../icons/book-a";
import { default as default170 } from "../icons/book-alert";
import { default as default171 } from "../icons/book-audio";
import { default as default172 } from "../icons/book-check";
import { default as default173 } from "../icons/book-copy";
import { default as default174 } from "../icons/book-down";
import { default as default175 } from "../icons/book-headphones";
import { default as default176 } from "../icons/book-heart";
import { default as default177 } from "../icons/book-image";
import { default as default178 } from "../icons/book-key";
import { default as default179 } from "../icons/book-lock";
import { default as default180 } from "../icons/book-marked";
import { default as default181 } from "../icons/book-minus";
import { default as default182 } from "../icons/book-open-check";
import { default as default183 } from "../icons/book-open-text";
import { default as default184 } from "../icons/book-open";
import { default as default185 } from "../icons/book-plus";
import { default as default186 } from "../icons/book-search";
import { default as default187 } from "../icons/book-text";
import { default as default188 } from "../icons/book-type";
import { default as default189 } from "../icons/book-up-2";
import { default as default190 } from "../icons/book-up";
import { default as default191 } from "../icons/book-user";
import { default as default192 } from "../icons/book-x";
import { default as default193 } from "../icons/bookmark-check";
import { default as default194 } from "../icons/bookmark-minus";
import { default as default195 } from "../icons/bookmark-off";
import { default as default196 } from "../icons/book";
import { default as default197 } from "../icons/bookmark-x";
import { default as default198 } from "../icons/bookmark-plus";
import { default as default199 } from "../icons/bookmark";
import { default as default200 } from "../icons/boom-box";
import { default as default201 } from "../icons/bot-message-square";
import { default as default202 } from "../icons/bot-off";
import { default as default203 } from "../icons/bot";
import { default as default204 } from "../icons/bottle-wine";
import { default as default205 } from "../icons/bow-arrow";
import { default as default206 } from "../icons/box";
import { default as default207 } from "../icons/boxes";
import { default as default208 } from "../icons/brackets";
import { default as default209 } from "../icons/brain-circuit";
import { default as default210 } from "../icons/brain-cog";
import { default as default211 } from "../icons/brain";
import { default as default212 } from "../icons/brick-wall-fire";
import { default as default213 } from "../icons/brick-wall-shield";
import { default as default214 } from "../icons/brick-wall";
import { default as default215 } from "../icons/briefcase-business";
import { default as default216 } from "../icons/briefcase-conveyor-belt";
import { default as default217 } from "../icons/briefcase-medical";
import { default as default218 } from "../icons/briefcase";
import { default as default219 } from "../icons/bring-to-front";
import { default as default220 } from "../icons/brush-cleaning";
import { default as default221 } from "../icons/brush";
import { default as default222 } from "../icons/bubbles";
import { default as default223 } from "../icons/bug-off";
import { default as default224 } from "../icons/bug-play";
import { default as default225 } from "../icons/bug";
import { default as default226 } from "../icons/building-2";
import { default as default227 } from "../icons/building";
import { default as default228 } from "../icons/bus-front";
import { default as default229 } from "../icons/bus";
import { default as default230 } from "../icons/cable-car";
import { default as default231 } from "../icons/cake-slice";
import { default as default232 } from "../icons/cable";
import { default as default233 } from "../icons/cake";
import { default as default234 } from "../icons/calculator";
import { default as default235 } from "../icons/calendar-1";
import { default as default236 } from "../icons/calendar-arrow-down";
import { default as default237 } from "../icons/calendar-arrow-up";
import { default as default238 } from "../icons/calendar-check-2";
import { default as default239 } from "../icons/calendar-check";
import { default as default240 } from "../icons/calendar-clock";
import { default as default241 } from "../icons/calendar-cog";
import { default as default242 } from "../icons/calendar-days";
import { default as default243 } from "../icons/calendar-heart";
import { default as default244 } from "../icons/calendar-fold";
import { default as default245 } from "../icons/calendar-minus-2";
import { default as default246 } from "../icons/calendar-minus";
import { default as default247 } from "../icons/calendar-off";
import { default as default248 } from "../icons/calendar-plus-2";
import { default as default249 } from "../icons/calendar-plus";
import { default as default250 } from "../icons/calendar-range";
import { default as default251 } from "../icons/calendar-sync";
import { default as default252 } from "../icons/calendar-search";
import { default as default253 } from "../icons/calendar-x-2";
import { default as default254 } from "../icons/calendar-x";
import { default as default255 } from "../icons/calendar";
import { default as default256 } from "../icons/calendars";
import { default as default257 } from "../icons/camera-off";
import { default as default258 } from "../icons/camera";
import { default as default259 } from "../icons/candy-cane";
import { default as default260 } from "../icons/candy-off";
import { default as default261 } from "../icons/candy";
import { default as default262 } from "../icons/cannabis";
import { default as default263 } from "../icons/cannabis-off";
import { default as default264 } from "../icons/car-front";
import { default as default265 } from "../icons/car-taxi-front";
import { default as default266 } from "../icons/captions-off";
import { default as default267 } from "../icons/caravan";
import { default as default268 } from "../icons/card-sim";
import { default as default269 } from "../icons/car";
import { default as default270 } from "../icons/carrot";
import { default as default271 } from "../icons/case-sensitive";
import { default as default272 } from "../icons/case-upper";
import { default as default273 } from "../icons/case-lower";
import { default as default274 } from "../icons/cassette-tape";
import { default as default275 } from "../icons/cast";
import { default as default276 } from "../icons/castle";
import { default as default277 } from "../icons/cat";
import { default as default278 } from "../icons/cctv-off";
import { default as default279 } from "../icons/cctv";
import { default as default280 } from "../icons/chart-bar-decreasing";
import { default as default281 } from "../icons/chart-bar-increasing";
import { default as default282 } from "../icons/chart-bar-stacked";
import { default as default283 } from "../icons/chart-column-decreasing";
import { default as default284 } from "../icons/chart-column-stacked";
import { default as default285 } from "../icons/chart-gantt";
import { default as default286 } from "../icons/chart-network";
import { default as default287 } from "../icons/chart-no-axes-column-decreasing";
import { default as default288 } from "../icons/chart-no-axes-combined";
import { default as default289 } from "../icons/chart-spline";
import { default as default290 } from "../icons/check-check";
import { default as default291 } from "../icons/check-line";
import { default as default292 } from "../icons/check";
import { default as default293 } from "../icons/cherry";
import { default as default294 } from "../icons/chef-hat";
import { default as default295 } from "../icons/chess-bishop";
import { default as default296 } from "../icons/chess-king";
import { default as default297 } from "../icons/chess-knight";
import { default as default298 } from "../icons/chess-pawn";
import { default as default299 } from "../icons/chess-queen";
import { default as default300 } from "../icons/chess-rook";
import { default as default301 } from "../icons/chevron-down";
import { default as default302 } from "../icons/chevron-first";
import { default as default303 } from "../icons/chevron-left";
import { default as default304 } from "../icons/chevron-right";
import { default as default305 } from "../icons/chevron-last";
import { default as default306 } from "../icons/chevron-up";
import { default as default307 } from "../icons/chevrons-down-up";
import { default as default308 } from "../icons/chevrons-down";
import { default as default309 } from "../icons/chevrons-left-right-ellipsis";
import { default as default310 } from "../icons/chevrons-left-right";
import { default as default311 } from "../icons/chevrons-left";
import { default as default312 } from "../icons/chevrons-right-left";
import { default as default313 } from "../icons/chevrons-up-down";
import { default as default314 } from "../icons/chevrons-right";
import { default as default315 } from "../icons/church";
import { default as default316 } from "../icons/chevrons-up";
import { default as default317 } from "../icons/cigarette-off";
import { default as default318 } from "../icons/cigarette";
import { default as default319 } from "../icons/circle-dashed";
import { default as default320 } from "../icons/circle-dollar-sign";
import { default as default321 } from "../icons/circle-ellipsis";
import { default as default322 } from "../icons/circle-dot-dashed";
import { default as default323 } from "../icons/circle-dot";
import { default as default324 } from "../icons/circle-fading-arrow-up";
import { default as default325 } from "../icons/circle-fading-plus";
import { default as default326 } from "../icons/circle-off";
import { default as default327 } from "../icons/circle-equal";
import { default as default328 } from "../icons/circle-pile";
import { default as default329 } from "../icons/circle-pound-sterling";
import { default as default330 } from "../icons/circle-slash";
import { default as default331 } from "../icons/circle-small";
import { default as default332 } from "../icons/circle-star";
import { default as default333 } from "../icons/circle";
import { default as default334 } from "../icons/citrus";
import { default as default335 } from "../icons/circuit-board";
import { default as default336 } from "../icons/clapperboard";
import { default as default337 } from "../icons/clipboard-check";
import { default as default338 } from "../icons/clipboard-copy";
import { default as default339 } from "../icons/clipboard-list";
import { default as default340 } from "../icons/clipboard-clock";
import { default as default341 } from "../icons/clipboard-paste";
import { default as default342 } from "../icons/clipboard-minus";
import { default as default343 } from "../icons/clipboard-type";
import { default as default344 } from "../icons/clipboard-plus";
import { default as default345 } from "../icons/clipboard-x";
import { default as default346 } from "../icons/clipboard";
import { default as default347 } from "../icons/clock-1";
import { default as default348 } from "../icons/clock-11";
import { default as default349 } from "../icons/clock-10";
import { default as default350 } from "../icons/clock-12";
import { default as default351 } from "../icons/clock-2";
import { default as default352 } from "../icons/clock-3";
import { default as default353 } from "../icons/clock-4";
import { default as default354 } from "../icons/clock-5";
import { default as default355 } from "../icons/clock-6";
import { default as default356 } from "../icons/clock-7";
import { default as default357 } from "../icons/clock-8";
import { default as default358 } from "../icons/clock-9";
import { default as default359 } from "../icons/clock-alert";
import { default as default360 } from "../icons/clock-arrow-up";
import { default as default361 } from "../icons/clock-arrow-down";
import { default as default362 } from "../icons/clock-check";
import { default as default363 } from "../icons/clock-fading";
import { default as default364 } from "../icons/clock-plus";
import { default as default365 } from "../icons/clock";
import { default as default366 } from "../icons/closed-caption";
import { default as default367 } from "../icons/cloud-backup";
import { default as default368 } from "../icons/cloud-alert";
import { default as default369 } from "../icons/cloud-drizzle";
import { default as default370 } from "../icons/cloud-cog";
import { default as default371 } from "../icons/cloud-check";
import { default as default372 } from "../icons/cloud-fog";
import { default as default373 } from "../icons/cloud-hail";
import { default as default374 } from "../icons/cloud-lightning";
import { default as default375 } from "../icons/cloud-moon-rain";
import { default as default376 } from "../icons/cloud-moon";
import { default as default377 } from "../icons/cloud-off";
import { default as default378 } from "../icons/cloud-rain-wind";
import { default as default379 } from "../icons/cloud-snow";
import { default as default380 } from "../icons/cloud-sun-rain";
import { default as default381 } from "../icons/cloud-rain";
import { default as default382 } from "../icons/cloud-sun";
import { default as default383 } from "../icons/cloud-sync";
import { default as default384 } from "../icons/cloud";
import { default as default385 } from "../icons/cloudy";
import { default as default386 } from "../icons/clover";
import { default as default387 } from "../icons/club";
import { default as default388 } from "../icons/coffee";
import { default as default389 } from "../icons/cog";
import { default as default390 } from "../icons/coins";
import { default as default391 } from "../icons/combine";
import { default as default392 } from "../icons/code";
import { default as default393 } from "../icons/command";
import { default as default394 } from "../icons/compass";
import { default as default395 } from "../icons/columns-4";
import { default as default396 } from "../icons/component";
import { default as default397 } from "../icons/concierge-bell";
import { default as default398 } from "../icons/cone";
import { default as default399 } from "../icons/computer";
import { default as default400 } from "../icons/construction";
import { default as default401 } from "../icons/contact";
import { default as default402 } from "../icons/container";
import { default as default403 } from "../icons/contrast";
import { default as default404 } from "../icons/cookie";
import { default as default405 } from "../icons/cooking-pot";
import { default as default406 } from "../icons/copy-check";
import { default as default407 } from "../icons/copy-minus";
import { default as default408 } from "../icons/copy-plus";
import { default as default409 } from "../icons/copy-slash";
import { default as default410 } from "../icons/copy-x";
import { default as default411 } from "../icons/copy";
import { default as default412 } from "../icons/copyleft";
import { default as default413 } from "../icons/copyright";
import { default as default414 } from "../icons/corner-down-left";
import { default as default415 } from "../icons/corner-down-right";
import { default as default416 } from "../icons/corner-left-down";
import { default as default417 } from "../icons/corner-left-up";
import { default as default418 } from "../icons/corner-right-down";
import { default as default419 } from "../icons/corner-right-up";
import { default as default420 } from "../icons/corner-up-left";
import { default as default421 } from "../icons/corner-up-right";
import { default as default422 } from "../icons/cpu";
import { default as default423 } from "../icons/creative-commons";
import { default as default424 } from "../icons/credit-card";
import { default as default425 } from "../icons/croissant";
import { default as default426 } from "../icons/crop";
import { default as default427 } from "../icons/cross";
import { default as default428 } from "../icons/crosshair";
import { default as default429 } from "../icons/crown";
import { default as default430 } from "../icons/cuboid";
import { default as default431 } from "../icons/cup-soda";
import { default as default432 } from "../icons/currency";
import { default as default433 } from "../icons/cylinder";
import { default as default434 } from "../icons/dam";
import { default as default435 } from "../icons/database-backup";
import { default as default436 } from "../icons/database-search";
import { default as default437 } from "../icons/database-zap";
import { default as default438 } from "../icons/decimals-arrow-left";
import { default as default439 } from "../icons/decimals-arrow-right";
import { default as default440 } from "../icons/delete";
import { default as default441 } from "../icons/database";
import { default as default442 } from "../icons/dessert";
import { default as default443 } from "../icons/diameter";
import { default as default444 } from "../icons/diamond-minus";
import { default as default445 } from "../icons/diamond-plus";
import { default as default446 } from "../icons/dice-2";
import { default as default447 } from "../icons/diamond";
import { default as default448 } from "../icons/dice-1";
import { default as default449 } from "../icons/dice-3";
import { default as default450 } from "../icons/dice-4";
import { default as default451 } from "../icons/dice-5";
import { default as default452 } from "../icons/dice-6";
import { default as default453 } from "../icons/dices";
import { default as default454 } from "../icons/diff";
import { default as default455 } from "../icons/disc-2";
import { default as default456 } from "../icons/disc-3";
import { default as default457 } from "../icons/disc";
import { default as default458 } from "../icons/divide";
import { default as default459 } from "../icons/disc-album";
import { default as default460 } from "../icons/dna-off";
import { default as default461 } from "../icons/dna";
import { default as default462 } from "../icons/dock";
import { default as default463 } from "../icons/dog";
import { default as default464 } from "../icons/dollar-sign";
import { default as default465 } from "../icons/donut";
import { default as default466 } from "../icons/door-closed-locked";
import { default as default467 } from "../icons/door-closed";
import { default as default468 } from "../icons/door-open";
import { default as default469 } from "../icons/dot";
import { default as default470 } from "../icons/download";
import { default as default471 } from "../icons/drafting-compass";
import { default as default472 } from "../icons/drama";
import { default as default473 } from "../icons/drill";
import { default as default474 } from "../icons/drone";
import { default as default475 } from "../icons/droplet-off";
import { default as default476 } from "../icons/droplet";
import { default as default477 } from "../icons/droplets";
import { default as default478 } from "../icons/drum";
import { default as default479 } from "../icons/drumstick";
import { default as default480 } from "../icons/dumbbell";
import { default as default481 } from "../icons/ear-off";
import { default as default482 } from "../icons/ear";
import { default as default483 } from "../icons/earth-lock";
import { default as default484 } from "../icons/eclipse";
import { default as default485 } from "../icons/egg-fried";
import { default as default486 } from "../icons/egg-off";
import { default as default487 } from "../icons/egg";
import { default as default488 } from "../icons/ellipse";
import { default as default489 } from "../icons/equal-approximately";
import { default as default490 } from "../icons/equal-not";
import { default as default491 } from "../icons/equal";
import { default as default492 } from "../icons/eraser";
import { default as default493 } from "../icons/ethernet-port";
import { default as default494 } from "../icons/euro";
import { default as default495 } from "../icons/ev-charger";
import { default as default496 } from "../icons/expand";
import { default as default497 } from "../icons/external-link";
import { default as default498 } from "../icons/eye-closed";
import { default as default499 } from "../icons/eye-off";
import { default as default500 } from "../icons/eye";
import { default as default501 } from "../icons/factory";
import { default as default502 } from "../icons/fan";
import { default as default503 } from "../icons/fast-forward";
import { default as default504 } from "../icons/feather";
import { default as default505 } from "../icons/fence";
import { default as default506 } from "../icons/ferris-wheel";
import { default as default507 } from "../icons/file-archive";
import { default as default508 } from "../icons/file-box";
import { default as default509 } from "../icons/file-clock";
import { default as default510 } from "../icons/file-check";
import { default as default511 } from "../icons/file-code";
import { default as default512 } from "../icons/file-digit";
import { default as default513 } from "../icons/file-diff";
import { default as default514 } from "../icons/file-down";
import { default as default515 } from "../icons/file-heart";
import { default as default516 } from "../icons/file-image";
import { default as default517 } from "../icons/file-input";
import { default as default518 } from "../icons/file-minus";
import { default as default519 } from "../icons/file-music";
import { default as default520 } from "../icons/file-output";
import { default as default521 } from "../icons/file-plus";
import { default as default522 } from "../icons/file-scan";
import { default as default523 } from "../icons/file-search";
import { default as default524 } from "../icons/file-sliders";
import { default as default525 } from "../icons/file-stack";
import { default as default526 } from "../icons/file-spreadsheet";
import { default as default527 } from "../icons/file-symlink";
import { default as default528 } from "../icons/file-terminal";
import { default as default529 } from "../icons/file-type";
import { default as default530 } from "../icons/file-up";
import { default as default531 } from "../icons/file-text";
import { default as default532 } from "../icons/file-volume";
import { default as default533 } from "../icons/file-user";
import { default as default534 } from "../icons/file-x";
import { default as default535 } from "../icons/file";
import { default as default536 } from "../icons/files";
import { default as default537 } from "../icons/film";
import { default as default538 } from "../icons/fish-symbol";
import { default as default539 } from "../icons/fire-extinguisher";
import { default as default540 } from "../icons/fish-off";
import { default as default541 } from "../icons/fish";
import { default as default542 } from "../icons/fishing-rod";
import { default as default543 } from "../icons/fishing-hook";
import { default as default544 } from "../icons/flag-off";
import { default as default545 } from "../icons/flag-triangle-left";
import { default as default546 } from "../icons/flag-triangle-right";
import { default as default547 } from "../icons/flag";
import { default as default548 } from "../icons/flame-kindling";
import { default as default549 } from "../icons/flame";
import { default as default550 } from "../icons/flashlight-off";
import { default as default551 } from "../icons/flask-conical";
import { default as default552 } from "../icons/flashlight";
import { default as default553 } from "../icons/flask-conical-off";
import { default as default554 } from "../icons/flask-round";
import { default as default555 } from "../icons/flip-horizontal-2";
import { default as default556 } from "../icons/flip-vertical-2";
import { default as default557 } from "../icons/flower-2";
import { default as default558 } from "../icons/flower";
import { default as default559 } from "../icons/fold-horizontal";
import { default as default560 } from "../icons/focus";
import { default as default561 } from "../icons/fold-vertical";
import { default as default562 } from "../icons/folder-archive";
import { default as default563 } from "../icons/folder-check";
import { default as default564 } from "../icons/folder-clock";
import { default as default565 } from "../icons/folder-closed";
import { default as default566 } from "../icons/folder-code";
import { default as default567 } from "../icons/folder-dot";
import { default as default568 } from "../icons/folder-down";
import { default as default569 } from "../icons/folder-git-2";
import { default as default570 } from "../icons/folder-git";
import { default as default571 } from "../icons/folder-input";
import { default as default572 } from "../icons/folder-heart";
import { default as default573 } from "../icons/folder-kanban";
import { default as default574 } from "../icons/folder-key";
import { default as default575 } from "../icons/folder-minus";
import { default as default576 } from "../icons/folder-open-dot";
import { default as default577 } from "../icons/folder-lock";
import { default as default578 } from "../icons/folder-open";
import { default as default579 } from "../icons/folder-output";
import { default as default580 } from "../icons/folder-plus";
import { default as default581 } from "../icons/folder-root";
import { default as default582 } from "../icons/folder-search-2";
import { default as default583 } from "../icons/folder-search";
import { default as default584 } from "../icons/folder-symlink";
import { default as default585 } from "../icons/folder-sync";
import { default as default586 } from "../icons/folder-tree";
import { default as default587 } from "../icons/folder-x";
import { default as default588 } from "../icons/folder-up";
import { default as default589 } from "../icons/folder";
import { default as default590 } from "../icons/folders";
import { default as default591 } from "../icons/footprints";
import { default as default592 } from "../icons/forklift";
import { default as default593 } from "../icons/form";
import { default as default594 } from "../icons/forward";
import { default as default595 } from "../icons/frame";
import { default as default596 } from "../icons/frown";
import { default as default597 } from "../icons/fuel";
import { default as default598 } from "../icons/fullscreen";
import { default as default599 } from "../icons/funnel-plus";
import { default as default600 } from "../icons/gallery-horizontal-end";
import { default as default601 } from "../icons/gallery-horizontal";
import { default as default602 } from "../icons/gallery-vertical";
import { default as default603 } from "../icons/gallery-thumbnails";
import { default as default604 } from "../icons/gallery-vertical-end";
import { default as default605 } from "../icons/gamepad-2";
import { default as default606 } from "../icons/gamepad-directional";
import { default as default607 } from "../icons/gamepad";
import { default as default608 } from "../icons/gauge";
import { default as default609 } from "../icons/gavel";
import { default as default610 } from "../icons/gem";
import { default as default611 } from "../icons/georgian-lari";
import { default as default612 } from "../icons/ghost";
import { default as default613 } from "../icons/gift";
import { default as default614 } from "../icons/git-branch-plus";
import { default as default615 } from "../icons/git-branch-minus";
import { default as default616 } from "../icons/git-branch";
import { default as default617 } from "../icons/git-commit-vertical";
import { default as default618 } from "../icons/git-compare-arrows";
import { default as default619 } from "../icons/git-compare";
import { default as default620 } from "../icons/git-fork";
import { default as default621 } from "../icons/git-graph";
import { default as default622 } from "../icons/git-merge-conflict";
import { default as default623 } from "../icons/git-pull-request-arrow";
import { default as default624 } from "../icons/git-merge";
import { default as default625 } from "../icons/git-pull-request-create-arrow";
import { default as default626 } from "../icons/git-pull-request-closed";
import { default as default627 } from "../icons/git-pull-request-create";
import { default as default628 } from "../icons/git-pull-request-draft";
import { default as default629 } from "../icons/git-pull-request";
import { default as default630 } from "../icons/glass-water";
import { default as default631 } from "../icons/glasses";
import { default as default632 } from "../icons/globe-lock";
import { default as default633 } from "../icons/globe-off";
import { default as default634 } from "../icons/globe-x";
import { default as default635 } from "../icons/globe";
import { default as default636 } from "../icons/goal";
import { default as default637 } from "../icons/gpu";
import { default as default638 } from "../icons/graduation-cap";
import { default as default639 } from "../icons/grape";
import { default as default640 } from "../icons/grid-3x2";
import { default as default641 } from "../icons/grip-horizontal";
import { default as default642 } from "../icons/grip";
import { default as default643 } from "../icons/group";
import { default as default644 } from "../icons/grip-vertical";
import { default as default645 } from "../icons/guitar";
import { default as default646 } from "../icons/ham";
import { default as default647 } from "../icons/hammer";
import { default as default648 } from "../icons/hand-coins";
import { default as default649 } from "../icons/hamburger";
import { default as default650 } from "../icons/hand-fist";
import { default as default651 } from "../icons/hand-heart";
import { default as default652 } from "../icons/hand-metal";
import { default as default653 } from "../icons/hand-platter";
import { default as default654 } from "../icons/hand";
import { default as default655 } from "../icons/handshake";
import { default as default656 } from "../icons/handbag";
import { default as default657 } from "../icons/hard-drive-download";
import { default as default658 } from "../icons/hard-drive-upload";
import { default as default659 } from "../icons/hard-drive";
import { default as default660 } from "../icons/hard-hat";
import { default as default661 } from "../icons/hash";
import { default as default662 } from "../icons/hd";
import { default as default663 } from "../icons/hat-glasses";
import { default as default664 } from "../icons/haze";
import { default as default665 } from "../icons/heading-1";
import { default as default666 } from "../icons/hdmi-port";
import { default as default667 } from "../icons/heading-2";
import { default as default668 } from "../icons/heading-4";
import { default as default669 } from "../icons/heading-3";
import { default as default670 } from "../icons/heading-5";
import { default as default671 } from "../icons/heading-6";
import { default as default672 } from "../icons/heading";
import { default as default673 } from "../icons/headphone-off";
import { default as default674 } from "../icons/headphones";
import { default as default675 } from "../icons/headset";
import { default as default676 } from "../icons/heart-crack";
import { default as default677 } from "../icons/heart-handshake";
import { default as default678 } from "../icons/heart-minus";
import { default as default679 } from "../icons/heart-off";
import { default as default680 } from "../icons/heart-plus";
import { default as default681 } from "../icons/heart-pulse";
import { default as default682 } from "../icons/heart-x";
import { default as default683 } from "../icons/heart";
import { default as default684 } from "../icons/helicopter";
import { default as default685 } from "../icons/heater";
import { default as default686 } from "../icons/hexagon";
import { default as default687 } from "../icons/highlighter";
import { default as default688 } from "../icons/history";
import { default as default689 } from "../icons/hop-off";
import { default as default690 } from "../icons/hop";
import { default as default691 } from "../icons/hospital";
import { default as default692 } from "../icons/hotel";
import { default as default693 } from "../icons/hourglass";
import { default as default694 } from "../icons/house-heart";
import { default as default695 } from "../icons/house-plus";
import { default as default696 } from "../icons/house-wifi";
import { default as default697 } from "../icons/house-plug";
import { default as default698 } from "../icons/id-card-lanyard";
import { default as default699 } from "../icons/id-card";
import { default as default700 } from "../icons/image-down";
import { default as default701 } from "../icons/image-minus";
import { default as default702 } from "../icons/image-off";
import { default as default703 } from "../icons/image-play";
import { default as default704 } from "../icons/image-plus";
import { default as default705 } from "../icons/image-up";
import { default as default706 } from "../icons/image-upscale";
import { default as default707 } from "../icons/image";
import { default as default708 } from "../icons/import";
import { default as default709 } from "../icons/images";
import { default as default710 } from "../icons/inbox";
import { default as default711 } from "../icons/indian-rupee";
import { default as default712 } from "../icons/infinity";
import { default as default713 } from "../icons/info";
import { default as default714 } from "../icons/italic";
import { default as default715 } from "../icons/inspection-panel";
import { default as default716 } from "../icons/iteration-ccw";
import { default as default717 } from "../icons/iteration-cw";
import { default as default718 } from "../icons/joystick";
import { default as default719 } from "../icons/japanese-yen";
import { default as default720 } from "../icons/kayak";
import { default as default721 } from "../icons/kanban";
import { default as default722 } from "../icons/key-square";
import { default as default723 } from "../icons/key-round";
import { default as default724 } from "../icons/key";
import { default as default725 } from "../icons/keyboard-music";
import { default as default726 } from "../icons/keyboard-off";
import { default as default727 } from "../icons/keyboard";
import { default as default728 } from "../icons/lamp-ceiling";
import { default as default729 } from "../icons/lamp-floor";
import { default as default730 } from "../icons/lamp-wall-down";
import { default as default731 } from "../icons/lamp-desk";
import { default as default732 } from "../icons/lamp-wall-up";
import { default as default733 } from "../icons/lamp";
import { default as default734 } from "../icons/land-plot";
import { default as default735 } from "../icons/landmark";
import { default as default736 } from "../icons/languages";
import { default as default737 } from "../icons/laptop-minimal-check";
import { default as default738 } from "../icons/laptop";
import { default as default739 } from "../icons/lasso-select";
import { default as default740 } from "../icons/lasso";
import { default as default741 } from "../icons/laugh";
import { default as default742 } from "../icons/layers-2";
import { default as default743 } from "../icons/layers-minus";
import { default as default744 } from "../icons/layers-plus";
import { default as default745 } from "../icons/layout-grid";
import { default as default746 } from "../icons/layout-dashboard";
import { default as default747 } from "../icons/layout-list";
import { default as default748 } from "../icons/layout-panel-left";
import { default as default749 } from "../icons/layout-panel-top";
import { default as default750 } from "../icons/layout-template";
import { default as default751 } from "../icons/leaf";
import { default as default752 } from "../icons/lectern";
import { default as default753 } from "../icons/leafy-green";
import { default as default754 } from "../icons/lens-concave";
import { default as default755 } from "../icons/library-big";
import { default as default756 } from "../icons/lens-convex";
import { default as default757 } from "../icons/library";
import { default as default758 } from "../icons/life-buoy";
import { default as default759 } from "../icons/ligature";
import { default as default760 } from "../icons/lightbulb-off";
import { default as default761 } from "../icons/lightbulb";
import { default as default762 } from "../icons/line-dot-right-horizontal";
import { default as default763 } from "../icons/line-squiggle";
import { default as default764 } from "../icons/line-style";
import { default as default765 } from "../icons/link-2-off";
import { default as default766 } from "../icons/link-2";
import { default as default767 } from "../icons/link";
import { default as default768 } from "../icons/list-check";
import { default as default769 } from "../icons/list-chevrons-down-up";
import { default as default770 } from "../icons/list-chevrons-up-down";
import { default as default771 } from "../icons/list-checks";
import { default as default772 } from "../icons/list-collapse";
import { default as default773 } from "../icons/list-end";
import { default as default774 } from "../icons/list-filter-plus";
import { default as default775 } from "../icons/list-filter";
import { default as default776 } from "../icons/list-minus";
import { default as default777 } from "../icons/list-music";
import { default as default778 } from "../icons/list-ordered";
import { default as default779 } from "../icons/list-plus";
import { default as default780 } from "../icons/list-restart";
import { default as default781 } from "../icons/list-start";
import { default as default782 } from "../icons/list-tree";
import { default as default783 } from "../icons/list-todo";
import { default as default784 } from "../icons/list-video";
import { default as default785 } from "../icons/list-x";
import { default as default786 } from "../icons/list";
import { default as default787 } from "../icons/loader-pinwheel";
import { default as default788 } from "../icons/loader";
import { default as default789 } from "../icons/locate-fixed";
import { default as default790 } from "../icons/locate-off";
import { default as default791 } from "../icons/locate";
import { default as default792 } from "../icons/lock-keyhole";
import { default as default793 } from "../icons/lock";
import { default as default794 } from "../icons/log-in";
import { default as default795 } from "../icons/log-out";
import { default as default796 } from "../icons/lollipop";
import { default as default797 } from "../icons/logs";
import { default as default798 } from "../icons/luggage";
import { default as default799 } from "../icons/magnet";
import { default as default800 } from "../icons/mail-check";
import { default as default801 } from "../icons/mail-minus";
import { default as default802 } from "../icons/mail-plus";
import { default as default803 } from "../icons/mail-search";
import { default as default804 } from "../icons/mail-warning";
import { default as default805 } from "../icons/mail-open";
import { default as default806 } from "../icons/mail-x";
import { default as default807 } from "../icons/mailbox";
import { default as default808 } from "../icons/mail";
import { default as default809 } from "../icons/mails";
import { default as default810 } from "../icons/map-pin-check-inside";
import { default as default811 } from "../icons/map-minus";
import { default as default812 } from "../icons/map-pin-check";
import { default as default813 } from "../icons/map-pin-house";
import { default as default814 } from "../icons/map-pin-minus";
import { default as default815 } from "../icons/map-pin-minus-inside";
import { default as default816 } from "../icons/map-pin-off";
import { default as default817 } from "../icons/map-pin-plus-inside";
import { default as default818 } from "../icons/map-pin-plus";
import { default as default819 } from "../icons/map-pin-search";
import { default as default820 } from "../icons/map-pin-x-inside";
import { default as default821 } from "../icons/map-pin-x";
import { default as default822 } from "../icons/map-pin";
import { default as default823 } from "../icons/map-pinned";
import { default as default824 } from "../icons/map-plus";
import { default as default825 } from "../icons/map";
import { default as default826 } from "../icons/mars-stroke";
import { default as default827 } from "../icons/mars";
import { default as default828 } from "../icons/martini";
import { default as default829 } from "../icons/maximize-2";
import { default as default830 } from "../icons/megaphone-off";
import { default as default831 } from "../icons/maximize";
import { default as default832 } from "../icons/medal";
import { default as default833 } from "../icons/megaphone";
import { default as default834 } from "../icons/meh";
import { default as default835 } from "../icons/memory-stick";
import { default as default836 } from "../icons/merge";
import { default as default837 } from "../icons/menu";
import { default as default838 } from "../icons/message-circle-check";
import { default as default839 } from "../icons/message-circle-code";
import { default as default840 } from "../icons/message-circle-dashed";
import { default as default841 } from "../icons/message-circle-heart";
import { default as default842 } from "../icons/message-circle-more";
import { default as default843 } from "../icons/message-circle-off";
import { default as default844 } from "../icons/message-circle-plus";
import { default as default845 } from "../icons/message-circle-reply";
import { default as default846 } from "../icons/message-circle-warning";
import { default as default847 } from "../icons/message-circle-x";
import { default as default848 } from "../icons/message-square-check";
import { default as default849 } from "../icons/message-circle";
import { default as default850 } from "../icons/message-square-code";
import { default as default851 } from "../icons/message-square-diff";
import { default as default852 } from "../icons/message-square-dashed";
import { default as default853 } from "../icons/message-square-dot";
import { default as default854 } from "../icons/message-square-heart";
import { default as default855 } from "../icons/message-square-lock";
import { default as default856 } from "../icons/message-square-more";
import { default as default857 } from "../icons/message-square-off";
import { default as default858 } from "../icons/message-square-plus";
import { default as default859 } from "../icons/message-square-quote";
import { default as default860 } from "../icons/message-square-reply";
import { default as default861 } from "../icons/message-square-share";
import { default as default862 } from "../icons/message-square-text";
import { default as default863 } from "../icons/message-square-warning";
import { default as default864 } from "../icons/message-square-x";
import { default as default865 } from "../icons/message-square";
import { default as default866 } from "../icons/messages-square";
import { default as default867 } from "../icons/metronome";
import { default as default868 } from "../icons/mic-off";
import { default as default869 } from "../icons/mic";
import { default as default870 } from "../icons/microchip";
import { default as default871 } from "../icons/microscope";
import { default as default872 } from "../icons/microwave";
import { default as default873 } from "../icons/milestone";
import { default as default874 } from "../icons/milk-off";
import { default as default875 } from "../icons/milk";
import { default as default876 } from "../icons/minimize-2";
import { default as default877 } from "../icons/minimize";
import { default as default878 } from "../icons/minus";
import { default as default879 } from "../icons/mirror-rectangular";
import { default as default880 } from "../icons/mirror-round";
import { default as default881 } from "../icons/monitor-check";
import { default as default882 } from "../icons/monitor-cloud";
import { default as default883 } from "../icons/monitor-cog";
import { default as default884 } from "../icons/monitor-dot";
import { default as default885 } from "../icons/monitor-down";
import { default as default886 } from "../icons/monitor-off";
import { default as default887 } from "../icons/monitor-play";
import { default as default888 } from "../icons/monitor-smartphone";
import { default as default889 } from "../icons/monitor-pause";
import { default as default890 } from "../icons/monitor-speaker";
import { default as default891 } from "../icons/monitor-up";
import { default as default892 } from "../icons/monitor-stop";
import { default as default893 } from "../icons/monitor-x";
import { default as default894 } from "../icons/monitor";
import { default as default895 } from "../icons/moon-star";
import { default as default896 } from "../icons/moon";
import { default as default897 } from "../icons/motorbike";
import { default as default898 } from "../icons/mountain";
import { default as default899 } from "../icons/mountain-snow";
import { default as default900 } from "../icons/mouse-left";
import { default as default901 } from "../icons/mouse-off";
import { default as default902 } from "../icons/mouse-pointer-2-off";
import { default as default903 } from "../icons/mouse-pointer-2";
import { default as default904 } from "../icons/mouse-pointer-ban";
import { default as default905 } from "../icons/mouse-pointer-click";
import { default as default906 } from "../icons/mouse-pointer";
import { default as default907 } from "../icons/mouse-right";
import { default as default908 } from "../icons/move-diagonal-2";
import { default as default909 } from "../icons/mouse";
import { default as default910 } from "../icons/move-diagonal";
import { default as default911 } from "../icons/move-down-left";
import { default as default912 } from "../icons/move-down-right";
import { default as default913 } from "../icons/move-down";
import { default as default914 } from "../icons/move-horizontal";
import { default as default915 } from "../icons/move-left";
import { default as default916 } from "../icons/move-right";
import { default as default917 } from "../icons/move-up-left";
import { default as default918 } from "../icons/move-up-right";
import { default as default919 } from "../icons/move-up";
import { default as default920 } from "../icons/move-vertical";
import { default as default921 } from "../icons/move";
import { default as default922 } from "../icons/music-2";
import { default as default923 } from "../icons/music-3";
import { default as default924 } from "../icons/music-4";
import { default as default925 } from "../icons/navigation-2";
import { default as default926 } from "../icons/navigation-2-off";
import { default as default927 } from "../icons/music";
import { default as default928 } from "../icons/navigation-off";
import { default as default929 } from "../icons/navigation";
import { default as default930 } from "../icons/newspaper";
import { default as default931 } from "../icons/network";
import { default as default932 } from "../icons/nfc";
import { default as default933 } from "../icons/non-binary";
import { default as default934 } from "../icons/notebook-pen";
import { default as default935 } from "../icons/notebook-tabs";
import { default as default936 } from "../icons/notebook-text";
import { default as default937 } from "../icons/notebook";
import { default as default938 } from "../icons/notepad-text-dashed";
import { default as default939 } from "../icons/notepad-text";
import { default as default940 } from "../icons/nut-off";
import { default as default941 } from "../icons/octagon-minus";
import { default as default942 } from "../icons/nut";
import { default as default943 } from "../icons/option";
import { default as default944 } from "../icons/omega";
import { default as default945 } from "../icons/origami";
import { default as default946 } from "../icons/orbit";
import { default as default947 } from "../icons/package-2";
import { default as default948 } from "../icons/package-check";
import { default as default949 } from "../icons/package-minus";
import { default as default950 } from "../icons/package-open";
import { default as default951 } from "../icons/package-plus";
import { default as default952 } from "../icons/package-search";
import { default as default953 } from "../icons/paint-bucket";
import { default as default954 } from "../icons/package-x";
import { default as default955 } from "../icons/paint-roller";
import { default as default956 } from "../icons/package";
import { default as default957 } from "../icons/paintbrush";
import { default as default958 } from "../icons/palette";
import { default as default959 } from "../icons/panda";
import { default as default960 } from "../icons/panel-bottom-open";
import { default as default961 } from "../icons/panel-bottom";
import { default as default962 } from "../icons/panel-bottom-close";
import { default as default963 } from "../icons/panel-left-right-dashed";
import { default as default964 } from "../icons/panel-right-close";
import { default as default965 } from "../icons/panel-right-open";
import { default as default966 } from "../icons/panel-right";
import { default as default967 } from "../icons/panel-top-bottom-dashed";
import { default as default968 } from "../icons/panel-top-close";
import { default as default969 } from "../icons/panel-top-open";
import { default as default970 } from "../icons/panel-top";
import { default as default971 } from "../icons/panels-right-bottom";
import { default as default972 } from "../icons/panels-left-bottom";
import { default as default973 } from "../icons/paperclip";
import { default as default974 } from "../icons/parentheses";
import { default as default975 } from "../icons/parking-meter";
import { default as default976 } from "../icons/party-popper";
import { default as default977 } from "../icons/paw-print";
import { default as default978 } from "../icons/pause";
import { default as default979 } from "../icons/pc-case";
import { default as default980 } from "../icons/pen-off";
import { default as default981 } from "../icons/pencil-line";
import { default as default982 } from "../icons/pen-tool";
import { default as default983 } from "../icons/pencil-ruler";
import { default as default984 } from "../icons/pencil";
import { default as default985 } from "../icons/pentagon";
import { default as default986 } from "../icons/pencil-off";
import { default as default987 } from "../icons/percent";
import { default as default988 } from "../icons/philippine-peso";
import { default as default989 } from "../icons/phone-call";
import { default as default990 } from "../icons/person-standing";
import { default as default991 } from "../icons/phone-forwarded";
import { default as default992 } from "../icons/phone-incoming";
import { default as default993 } from "../icons/phone-missed";
import { default as default994 } from "../icons/phone-off";
import { default as default995 } from "../icons/phone-outgoing";
import { default as default996 } from "../icons/phone";
import { default as default997 } from "../icons/pi";
import { default as default998 } from "../icons/piano";
import { default as default999 } from "../icons/pickaxe";
import { default as default1000 } from "../icons/picture-in-picture-2";
import { default as default1001 } from "../icons/picture-in-picture";
import { default as default1002 } from "../icons/piggy-bank";
import { default as default1003 } from "../icons/pilcrow-left";
import { default as default1004 } from "../icons/pilcrow-right";
import { default as default1005 } from "../icons/pilcrow";
import { default as default1006 } from "../icons/pill-bottle";
import { default as default1007 } from "../icons/pill";
import { default as default1008 } from "../icons/pin";
import { default as default1009 } from "../icons/pipette";
import { default as default1010 } from "../icons/pizza";
import { default as default1011 } from "../icons/plane-landing";
import { default as default1012 } from "../icons/pin-off";
import { default as default1013 } from "../icons/plane-takeoff";
import { default as default1014 } from "../icons/plane";
import { default as default1015 } from "../icons/plug-2";
import { default as default1016 } from "../icons/plug";
import { default as default1017 } from "../icons/play";
import { default as default1018 } from "../icons/plus";
import { default as default1019 } from "../icons/pocket-knife";
import { default as default1020 } from "../icons/podcast";
import { default as default1021 } from "../icons/pointer-off";
import { default as default1022 } from "../icons/pointer";
import { default as default1023 } from "../icons/popcorn";
import { default as default1024 } from "../icons/popsicle";
import { default as default1025 } from "../icons/pound-sterling";
import { default as default1026 } from "../icons/power-off";
import { default as default1027 } from "../icons/presentation";
import { default as default1028 } from "../icons/printer-check";
import { default as default1029 } from "../icons/printer-x";
import { default as default1030 } from "../icons/projector";
import { default as default1031 } from "../icons/power";
import { default as default1032 } from "../icons/proportions";
import { default as default1033 } from "../icons/printer";
import { default as default1034 } from "../icons/pyramid";
import { default as default1035 } from "../icons/qr-code";
import { default as default1036 } from "../icons/quote";
import { default as default1037 } from "../icons/puzzle";
import { default as default1038 } from "../icons/rabbit";
import { default as default1039 } from "../icons/radar";
import { default as default1040 } from "../icons/radical";
import { default as default1041 } from "../icons/radio-off";
import { default as default1042 } from "../icons/radiation";
import { default as default1043 } from "../icons/radio-receiver";
import { default as default1044 } from "../icons/radio-tower";
import { default as default1045 } from "../icons/radio";
import { default as default1046 } from "../icons/radius";
import { default as default1047 } from "../icons/rainbow";
import { default as default1048 } from "../icons/rat";
import { default as default1049 } from "../icons/ratio";
import { default as default1050 } from "../icons/receipt-cent";
import { default as default1051 } from "../icons/receipt-euro";
import { default as default1052 } from "../icons/receipt-indian-rupee";
import { default as default1053 } from "../icons/receipt-japanese-yen";
import { default as default1054 } from "../icons/receipt-pound-sterling";
import { default as default1055 } from "../icons/receipt-russian-ruble";
import { default as default1056 } from "../icons/receipt-swiss-franc";
import { default as default1057 } from "../icons/receipt-text";
import { default as default1058 } from "../icons/receipt-turkish-lira";
import { default as default1059 } from "../icons/receipt";
import { default as default1060 } from "../icons/octagon";
import { default as default1061 } from "../icons/rectangle-goggles";
import { default as default1062 } from "../icons/rectangle-circle";
import { default as default1063 } from "../icons/rectangle-horizontal";
import { default as default1064 } from "../icons/rectangle-vertical";
import { default as default1065 } from "../icons/recycle";
import { default as default1066 } from "../icons/redo-2";
import { default as default1067 } from "../icons/redo-dot";
import { default as default1068 } from "../icons/redo";
import { default as default1069 } from "../icons/refresh-ccw";
import { default as default1070 } from "../icons/refresh-ccw-dot";
import { default as default1071 } from "../icons/refresh-cw-off";
import { default as default1072 } from "../icons/refresh-cw";
import { default as default1073 } from "../icons/refrigerator";
import { default as default1074 } from "../icons/regex";
import { default as default1075 } from "../icons/remove-formatting";
import { default as default1076 } from "../icons/repeat-1";
import { default as default1077 } from "../icons/repeat-2";
import { default as default1078 } from "../icons/repeat";
import { default as default1079 } from "../icons/replace-all";
import { default as default1080 } from "../icons/reply-all";
import { default as default1081 } from "../icons/replace";
import { default as default1082 } from "../icons/reply";
import { default as default1083 } from "../icons/rewind";
import { default as default1084 } from "../icons/ribbon";
import { default as default1085 } from "../icons/road";
import { default as default1086 } from "../icons/rocket";
import { default as default1087 } from "../icons/rocking-chair";
import { default as default1088 } from "../icons/roller-coaster";
import { default as default1089 } from "../icons/rose";
import { default as default1090 } from "../icons/rotate-ccw-key";
import { default as default1091 } from "../icons/rotate-ccw-square";
import { default as default1092 } from "../icons/rotate-ccw";
import { default as default1093 } from "../icons/rotate-cw";
import { default as default1094 } from "../icons/rotate-cw-square";
import { default as default1095 } from "../icons/route-off";
import { default as default1096 } from "../icons/route";
import { default as default1097 } from "../icons/router";
import { default as default1098 } from "../icons/rows-4";
import { default as default1099 } from "../icons/ruler-dimension-line";
import { default as default1100 } from "../icons/ruler";
import { default as default1101 } from "../icons/rss";
import { default as default1102 } from "../icons/russian-ruble";
import { default as default1103 } from "../icons/sailboat";
import { default as default1104 } from "../icons/salad";
import { default as default1105 } from "../icons/sandwich";
import { default as default1106 } from "../icons/satellite-dish";
import { default as default1107 } from "../icons/satellite";
import { default as default1108 } from "../icons/saudi-riyal";
import { default as default1109 } from "../icons/save-all";
import { default as default1110 } from "../icons/save-off";
import { default as default1111 } from "../icons/save";
import { default as default1112 } from "../icons/scale";
import { default as default1113 } from "../icons/scaling";
import { default as default1114 } from "../icons/scan-barcode";
import { default as default1115 } from "../icons/scan-eye";
import { default as default1116 } from "../icons/scan-face";
import { default as default1117 } from "../icons/scan-heart";
import { default as default1118 } from "../icons/scan-line";
import { default as default1119 } from "../icons/scan-qr-code";
import { default as default1120 } from "../icons/scan-search";
import { default as default1121 } from "../icons/scan-text";
import { default as default1122 } from "../icons/scan";
import { default as default1123 } from "../icons/school";
import { default as default1124 } from "../icons/scissors-line-dashed";
import { default as default1125 } from "../icons/scissors";
import { default as default1126 } from "../icons/scooter";
import { default as default1127 } from "../icons/screen-share-off";
import { default as default1128 } from "../icons/scroll-text";
import { default as default1129 } from "../icons/screen-share";
import { default as default1130 } from "../icons/scroll";
import { default as default1131 } from "../icons/search-alert";
import { default as default1132 } from "../icons/search-check";
import { default as default1133 } from "../icons/search-code";
import { default as default1134 } from "../icons/search-slash";
import { default as default1135 } from "../icons/search-x";
import { default as default1136 } from "../icons/search";
import { default as default1137 } from "../icons/send-to-back";
import { default as default1138 } from "../icons/section";
import { default as default1139 } from "../icons/send";
import { default as default1140 } from "../icons/separator-horizontal";
import { default as default1141 } from "../icons/server-cog";
import { default as default1142 } from "../icons/separator-vertical";
import { default as default1143 } from "../icons/server-crash";
import { default as default1144 } from "../icons/server-off";
import { default as default1145 } from "../icons/server";
import { default as default1146 } from "../icons/settings-2";
import { default as default1147 } from "../icons/settings";
import { default as default1148 } from "../icons/share";
import { default as default1149 } from "../icons/shapes";
import { default as default1150 } from "../icons/share-2";
import { default as default1151 } from "../icons/sheet";
import { default as default1152 } from "../icons/shell";
import { default as default1153 } from "../icons/shelving-unit";
import { default as default1154 } from "../icons/shield-alert";
import { default as default1155 } from "../icons/shield-ban";
import { default as default1156 } from "../icons/shield-check";
import { default as default1157 } from "../icons/shield-cog-corner";
import { default as default1158 } from "../icons/shield-cog";
import { default as default1159 } from "../icons/shield-ellipsis";
import { default as default1160 } from "../icons/shield-half";
import { default as default1161 } from "../icons/shield-minus";
import { default as default1162 } from "../icons/shield-off";
import { default as default1163 } from "../icons/shield-plus";
import { default as default1164 } from "../icons/shield-user";
import { default as default1165 } from "../icons/shield";
import { default as default1166 } from "../icons/ship-wheel";
import { default as default1167 } from "../icons/ship";
import { default as default1168 } from "../icons/shirt";
import { default as default1169 } from "../icons/shopping-bag";
import { default as default1170 } from "../icons/shopping-basket";
import { default as default1171 } from "../icons/shopping-cart";
import { default as default1172 } from "../icons/shower-head";
import { default as default1173 } from "../icons/shovel";
import { default as default1174 } from "../icons/shredder";
import { default as default1175 } from "../icons/shrimp";
import { default as default1176 } from "../icons/shrink";
import { default as default1177 } from "../icons/shrub";
import { default as default1178 } from "../icons/shuffle";
import { default as default1179 } from "../icons/sigma";
import { default as default1180 } from "../icons/signal-high";
import { default as default1181 } from "../icons/signal-low";
import { default as default1182 } from "../icons/signal-medium";
import { default as default1183 } from "../icons/signal-zero";
import { default as default1184 } from "../icons/signature";
import { default as default1185 } from "../icons/signpost-big";
import { default as default1186 } from "../icons/signpost";
import { default as default1187 } from "../icons/signal";
import { default as default1188 } from "../icons/siren";
import { default as default1189 } from "../icons/skip-back";
import { default as default1190 } from "../icons/skip-forward";
import { default as default1191 } from "../icons/skull";
import { default as default1192 } from "../icons/slash";
import { default as default1193 } from "../icons/slice";
import { default as default1194 } from "../icons/sliders-horizontal";
import { default as default1195 } from "../icons/smartphone-charging";
import { default as default1196 } from "../icons/smartphone-nfc";
import { default as default1197 } from "../icons/smartphone";
import { default as default1198 } from "../icons/smile-plus";
import { default as default1199 } from "../icons/smile";
import { default as default1200 } from "../icons/snail";
import { default as default1201 } from "../icons/soap-dispenser-droplet";
import { default as default1202 } from "../icons/sofa";
import { default as default1203 } from "../icons/snowflake";
import { default as default1204 } from "../icons/solar-panel";
import { default as default1205 } from "../icons/soup";
import { default as default1206 } from "../icons/spade";
import { default as default1207 } from "../icons/sparkle";
import { default as default1208 } from "../icons/space";
import { default as default1209 } from "../icons/speaker";
import { default as default1210 } from "../icons/speech";
import { default as default1211 } from "../icons/spell-check-2";
import { default as default1212 } from "../icons/spell-check";
import { default as default1213 } from "../icons/spline-pointer";
import { default as default1214 } from "../icons/spline";
import { default as default1215 } from "../icons/split";
import { default as default1216 } from "../icons/spool";
import { default as default1217 } from "../icons/sport-shoe";
import { default as default1218 } from "../icons/spotlight";
import { default as default1219 } from "../icons/spray-can";
import { default as default1220 } from "../icons/sprout";
import { default as default1221 } from "../icons/square-arrow-right-enter";
import { default as default1222 } from "../icons/square-arrow-right-exit";
import { default as default1223 } from "../icons/square-dashed-bottom-code";
import { default as default1224 } from "../icons/square-dashed-bottom";
import { default as default1225 } from "../icons/square-dashed-top-solid";
import { default as default1226 } from "../icons/square-pause";
import { default as default1227 } from "../icons/square-radical";
import { default as default1228 } from "../icons/square-round-corner";
import { default as default1229 } from "../icons/square-square";
import { default as default1230 } from "../icons/square-stack";
import { default as default1231 } from "../icons/square-star";
import { default as default1232 } from "../icons/square-stop";
import { default as default1233 } from "../icons/square";
import { default as default1234 } from "../icons/squares-exclude";
import { default as default1235 } from "../icons/squares-intersect";
import { default as default1236 } from "../icons/squares-subtract";
import { default as default1237 } from "../icons/squares-unite";
import { default as default1238 } from "../icons/squircle-dashed";
import { default as default1239 } from "../icons/squircle";
import { default as default1240 } from "../icons/squirrel";
import { default as default1241 } from "../icons/stamp";
import { default as default1242 } from "../icons/star-half";
import { default as default1243 } from "../icons/star-off";
import { default as default1244 } from "../icons/step-back";
import { default as default1245 } from "../icons/star";
import { default as default1246 } from "../icons/step-forward";
import { default as default1247 } from "../icons/sticker";
import { default as default1248 } from "../icons/stethoscope";
import { default as default1249 } from "../icons/sticky-note";
import { default as default1250 } from "../icons/store";
import { default as default1251 } from "../icons/stretch-horizontal";
import { default as default1252 } from "../icons/stone";
import { default as default1253 } from "../icons/stretch-vertical";
import { default as default1254 } from "../icons/strikethrough";
import { default as default1255 } from "../icons/subscript";
import { default as default1256 } from "../icons/sun-dim";
import { default as default1257 } from "../icons/sun-medium";
import { default as default1258 } from "../icons/sun-moon";
import { default as default1259 } from "../icons/sun-snow";
import { default as default1260 } from "../icons/sun";
import { default as default1261 } from "../icons/sunrise";
import { default as default1262 } from "../icons/sunset";
import { default as default1263 } from "../icons/superscript";
import { default as default1264 } from "../icons/swatch-book";
import { default as default1265 } from "../icons/swiss-franc";
import { default as default1266 } from "../icons/switch-camera";
import { default as default1267 } from "../icons/sword";
import { default as default1268 } from "../icons/swords";
import { default as default1269 } from "../icons/syringe";
import { default as default1270 } from "../icons/table-cells-merge";
import { default as default1271 } from "../icons/table-2";
import { default as default1272 } from "../icons/table-cells-split";
import { default as default1273 } from "../icons/table-columns-split";
import { default as default1274 } from "../icons/table-of-contents";
import { default as default1275 } from "../icons/table-properties";
import { default as default1276 } from "../icons/table-rows-split";
import { default as default1277 } from "../icons/table";
import { default as default1278 } from "../icons/tablet-smartphone";
import { default as default1279 } from "../icons/tablets";
import { default as default1280 } from "../icons/tablet";
import { default as default1281 } from "../icons/tag";
import { default as default1282 } from "../icons/tally-1";
import { default as default1283 } from "../icons/tags";
import { default as default1284 } from "../icons/tally-2";
import { default as default1285 } from "../icons/tally-3";
import { default as default1286 } from "../icons/tally-4";
import { default as default1287 } from "../icons/tally-5";
import { default as default1288 } from "../icons/tangent";
import { default as default1289 } from "../icons/target";
import { default as default1290 } from "../icons/telescope";
import { default as default1291 } from "../icons/tent-tree";
import { default as default1292 } from "../icons/terminal";
import { default as default1293 } from "../icons/test-tube";
import { default as default1294 } from "../icons/tent";
import { default as default1295 } from "../icons/test-tubes";
import { default as default1296 } from "../icons/text-cursor-input";
import { default as default1297 } from "../icons/text-cursor";
import { default as default1298 } from "../icons/text-quote";
import { default as default1299 } from "../icons/text-search";
import { default as default1300 } from "../icons/theater";
import { default as default1301 } from "../icons/thermometer-snowflake";
import { default as default1302 } from "../icons/thermometer-sun";
import { default as default1303 } from "../icons/thermometer";
import { default as default1304 } from "../icons/thumbs-down";
import { default as default1305 } from "../icons/thumbs-up";
import { default as default1306 } from "../icons/ticket-check";
import { default as default1307 } from "../icons/ticket-minus";
import { default as default1308 } from "../icons/ticket-percent";
import { default as default1309 } from "../icons/ticket-plus";
import { default as default1310 } from "../icons/ticket-slash";
import { default as default1311 } from "../icons/ticket";
import { default as default1312 } from "../icons/ticket-x";
import { default as default1313 } from "../icons/tickets-plane";
import { default as default1314 } from "../icons/tickets";
import { default as default1315 } from "../icons/timeline";
import { default as default1316 } from "../icons/timer-off";
import { default as default1317 } from "../icons/timer";
import { default as default1318 } from "../icons/timer-reset";
import { default as default1319 } from "../icons/toggle-left";
import { default as default1320 } from "../icons/toggle-right";
import { default as default1321 } from "../icons/toilet";
import { default as default1322 } from "../icons/tool-case";
import { default as default1323 } from "../icons/toolbox";
import { default as default1324 } from "../icons/tornado";
import { default as default1325 } from "../icons/touchpad-off";
import { default as default1326 } from "../icons/torus";
import { default as default1327 } from "../icons/touchpad";
import { default as default1328 } from "../icons/towel-rack";
import { default as default1329 } from "../icons/tower-control";
import { default as default1330 } from "../icons/toy-brick";
import { default as default1331 } from "../icons/tractor";
import { default as default1332 } from "../icons/train-front-tunnel";
import { default as default1333 } from "../icons/traffic-cone";
import { default as default1334 } from "../icons/train-front";
import { default as default1335 } from "../icons/train-track";
import { default as default1336 } from "../icons/transgender";
import { default as default1337 } from "../icons/trash-2";
import { default as default1338 } from "../icons/trash";
import { default as default1339 } from "../icons/tree-deciduous";
import { default as default1340 } from "../icons/tree-pine";
import { default as default1341 } from "../icons/trees";
import { default as default1342 } from "../icons/trending-down";
import { default as default1343 } from "../icons/trending-up-down";
import { default as default1344 } from "../icons/trending-up";
import { default as default1345 } from "../icons/triangle-dashed";
import { default as default1346 } from "../icons/triangle-right";
import { default as default1347 } from "../icons/triangle";
import { default as default1348 } from "../icons/trophy";
import { default as default1349 } from "../icons/truck-electric";
import { default as default1350 } from "../icons/turkish-lira";
import { default as default1351 } from "../icons/truck";
import { default as default1352 } from "../icons/turntable";
import { default as default1353 } from "../icons/turtle";
import { default as default1354 } from "../icons/tv-minimal-play";
import { default as default1355 } from "../icons/tv";
import { default as default1356 } from "../icons/type-outline";
import { default as default1357 } from "../icons/type";
import { default as default1358 } from "../icons/umbrella-off";
import { default as default1359 } from "../icons/umbrella";
import { default as default1360 } from "../icons/underline";
import { default as default1361 } from "../icons/undo-2";
import { default as default1362 } from "../icons/undo-dot";
import { default as default1363 } from "../icons/undo";
import { default as default1364 } from "../icons/unfold-horizontal";
import { default as default1365 } from "../icons/unfold-vertical";
import { default as default1366 } from "../icons/unlink-2";
import { default as default1367 } from "../icons/ungroup";
import { default as default1368 } from "../icons/unlink";
import { default as default1369 } from "../icons/unplug";
import { default as default1370 } from "../icons/upload";
import { default as default1371 } from "../icons/user-check";
import { default as default1372 } from "../icons/user-cog";
import { default as default1373 } from "../icons/usb";
import { default as default1374 } from "../icons/user-lock";
import { default as default1375 } from "../icons/user-key";
import { default as default1376 } from "../icons/user-minus";
import { default as default1377 } from "../icons/user-pen";
import { default as default1378 } from "../icons/user-plus";
import { default as default1379 } from "../icons/user-round-key";
import { default as default1380 } from "../icons/user-round-pen";
import { default as default1381 } from "../icons/user-round-search";
import { default as default1382 } from "../icons/user-search";
import { default as default1383 } from "../icons/user-star";
import { default as default1384 } from "../icons/user-x";
import { default as default1385 } from "../icons/user";
import { default as default1386 } from "../icons/users";
import { default as default1387 } from "../icons/van";
import { default as default1388 } from "../icons/utility-pole";
import { default as default1389 } from "../icons/variable";
import { default as default1390 } from "../icons/vault";
import { default as default1391 } from "../icons/vector-square";
import { default as default1392 } from "../icons/vegan";
import { default as default1393 } from "../icons/venetian-mask";
import { default as default1394 } from "../icons/venus";
import { default as default1395 } from "../icons/venus-and-mars";
import { default as default1396 } from "../icons/vibrate-off";
import { default as default1397 } from "../icons/vibrate";
import { default as default1398 } from "../icons/video-off";
import { default as default1399 } from "../icons/video";
import { default as default1400 } from "../icons/videotape";
import { default as default1401 } from "../icons/view";
import { default as default1402 } from "../icons/volleyball";
import { default as default1403 } from "../icons/voicemail";
import { default as default1404 } from "../icons/volume-1";
import { default as default1405 } from "../icons/volume-2";
import { default as default1406 } from "../icons/volume-off";
import { default as default1407 } from "../icons/volume-x";
import { default as default1408 } from "../icons/volume";
import { default as default1409 } from "../icons/vote";
import { default as default1410 } from "../icons/wallet-cards";
import { default as default1411 } from "../icons/wallet";
import { default as default1412 } from "../icons/wallpaper";
import { default as default1413 } from "../icons/wand";
import { default as default1414 } from "../icons/warehouse";
import { default as default1415 } from "../icons/washing-machine";
import { default as default1416 } from "../icons/watch";
import { default as default1417 } from "../icons/waves-arrow-down";
import { default as default1418 } from "../icons/waves-arrow-up";
import { default as default1419 } from "../icons/waves-ladder";
import { default as default1420 } from "../icons/waypoints";
import { default as default1421 } from "../icons/waves";
import { default as default1422 } from "../icons/webcam";
import { default as default1423 } from "../icons/webhook-off";
import { default as default1424 } from "../icons/webhook";
import { default as default1425 } from "../icons/weight-tilde";
import { default as default1426 } from "../icons/weight";
import { default as default1427 } from "../icons/wheat";
import { default as default1428 } from "../icons/wheat-off";
import { default as default1429 } from "../icons/whole-word";
import { default as default1430 } from "../icons/wifi-cog";
import { default as default1431 } from "../icons/wifi-high";
import { default as default1432 } from "../icons/wifi-low";
import { default as default1433 } from "../icons/wifi-off";
import { default as default1434 } from "../icons/wifi-pen";
import { default as default1435 } from "../icons/wifi-sync";
import { default as default1436 } from "../icons/wifi-zero";
import { default as default1437 } from "../icons/wifi";
import { default as default1438 } from "../icons/wind-arrow-down";
import { default as default1439 } from "../icons/wind";
import { default as default1440 } from "../icons/wine-off";
import { default as default1441 } from "../icons/wine";
import { default as default1442 } from "../icons/workflow";
import { default as default1443 } from "../icons/x";
import { default as default1444 } from "../icons/worm";
import { default as default1445 } from "../icons/wrench";
import { default as default1446 } from "../icons/x-line-top";
import { default as default1447 } from "../icons/zap";
import { default as default1448 } from "../icons/zap-off";
import { default as default1449 } from "../icons/zodiac-aquarius";
import { default as default1450 } from "../icons/zodiac-aries";
import { default as default1451 } from "../icons/zodiac-cancer";
import { default as default1452 } from "../icons/zodiac-capricorn";
import { default as default1453 } from "../icons/zodiac-gemini";
import { default as default1454 } from "../icons/zodiac-leo";
import { default as default1455 } from "../icons/zodiac-libra";
import { default as default1456 } from "../icons/zodiac-ophiuchus";
import { default as default1457 } from "../icons/zodiac-sagittarius";
import { default as default1458 } from "../icons/zodiac-pisces";
import { default as default1459 } from "../icons/zodiac-scorpio";
import { default as default1460 } from "../icons/zoom-in";
import { default as default1461 } from "../icons/zodiac-taurus";
import { default as default1462 } from "../icons/zodiac-virgo";
import { default as default1463 } from "../icons/zoom-out";
import { default as default1464 } from "../icons/fingerprint-pattern";
import {
  default as default1465
} from "../icons/fingerprint-pattern";
import { default as default1466 } from "../icons/arrow-down-0-1";
import { default as default1467 } from "../icons/arrow-down-1-0";
import { default as default1468 } from "../icons/arrow-up-0-1";
import { default as default1469 } from "../icons/arrow-up-1-0";
import { default as default1470 } from "../icons/alarm-clock-check";
import {
  default as default1471
} from "../icons/alarm-check";
import { default as default1472 } from "../icons/alarm-clock-plus";
import {
  default as default1473
} from "../icons/alarm-plus";
import { default as default1474 } from "../icons/alarm-clock-minus";
import {
  default as default1475
} from "../icons/alarm-minus";
import { default as default1476 } from "../icons/arrow-down-a-z";
import {
  default as default1477
} from "../icons/arrow-down-az";
import { default as default1478 } from "../icons/arrow-down-wide-narrow";
import {
  default as default1479
} from "../icons/sort-desc";
import { default as default1480 } from "../icons/arrow-down-z-a";
import {
  default as default1481
} from "../icons/arrow-down-za";
import { default as default1482 } from "../icons/arrow-up-a-z";
import {
  default as default1483
} from "../icons/arrow-up-az";
import { default as default1484 } from "../icons/arrow-up-narrow-wide";
import {
  default as default1485
} from "../icons/sort-asc";
import { default as default1486 } from "../icons/arrow-up-z-a";
import {
  default as default1487
} from "../icons/arrow-up-za";
import { default as default1488 } from "../icons/axis-3d";
import {
  default as default1489
} from "../icons/axis-3-d";
import { default as default1490 } from "../icons/badge-check";
import {
  default as default1491
} from "../icons/verified";
import { default as default1492 } from "../icons/badge-question-mark";
import {
  default as default1493
} from "../icons/badge-help";
import { default as default1494 } from "../icons/between-horizontal-end";
import {
  default as default1495
} from "../icons/between-horizonal-end";
import { default as default1496 } from "../icons/between-horizontal-start";
import {
  default as default1497
} from "../icons/between-horizonal-start";
import { default as default1498 } from "../icons/book-dashed";
import {
  default as default1499
} from "../icons/book-template";
import { default as default1500 } from "../icons/braces";
import {
  default as default1501
} from "../icons/curly-braces";
import { default as default1502 } from "../icons/captions";
import {
  default as default1503
} from "../icons/subtitles";
import { default as default1504 } from "../icons/chart-area";
import {
  default as default1505
} from "../icons/area-chart";
import { default as default1506 } from "../icons/chart-bar-big";
import {
  default as default1507
} from "../icons/bar-chart-horizontal-big";
import { default as default1508 } from "../icons/chart-bar";
import {
  default as default1509
} from "../icons/bar-chart-horizontal";
import { default as default1510 } from "../icons/chart-candlestick";
import {
  default as default1511
} from "../icons/candlestick-chart";
import { default as default1512 } from "../icons/chart-column-increasing";
import {
  default as default1513
} from "../icons/bar-chart-4";
import { default as default1514 } from "../icons/chart-column-big";
import {
  default as default1515
} from "../icons/bar-chart-big";
import { default as default1516 } from "../icons/chart-column";
import {
  default as default1517
} from "../icons/bar-chart-3";
import { default as default1518 } from "../icons/chart-no-axes-column-increasing";
import {
  default as default1519
} from "../icons/bar-chart";
import { default as default1520 } from "../icons/chart-line";
import {
  default as default1521
} from "../icons/line-chart";
import { default as default1522 } from "../icons/chart-no-axes-gantt";
import {
  default as default1523
} from "../icons/gantt-chart";
import { default as default1524 } from "../icons/chart-no-axes-column";
import {
  default as default1525
} from "../icons/bar-chart-2";
import { default as default1526 } from "../icons/chart-pie";
import {
  default as default1527
} from "../icons/pie-chart";
import { default as default1528 } from "../icons/chart-scatter";
import {
  default as default1529
} from "../icons/scatter-chart";
import { default as default1530 } from "../icons/circle-alert";
import {
  default as default1531
} from "../icons/alert-circle";
import { default as default1532 } from "../icons/circle-arrow-down";
import {
  default as default1533
} from "../icons/arrow-down-circle";
import { default as default1534 } from "../icons/circle-arrow-left";
import {
  default as default1535
} from "../icons/arrow-left-circle";
import { default as default1536 } from "../icons/circle-arrow-out-down-left";
import {
  default as default1537
} from "../icons/arrow-down-left-from-circle";
import { default as default1538 } from "../icons/circle-arrow-out-down-right";
import {
  default as default1539
} from "../icons/arrow-down-right-from-circle";
import { default as default1540 } from "../icons/circle-arrow-out-up-right";
import {
  default as default1541
} from "../icons/arrow-up-right-from-circle";
import { default as default1542 } from "../icons/circle-arrow-out-up-left";
import {
  default as default1543
} from "../icons/arrow-up-left-from-circle";
import { default as default1544 } from "../icons/circle-arrow-right";
import {
  default as default1545
} from "../icons/arrow-right-circle";
import { default as default1546 } from "../icons/circle-arrow-up";
import {
  default as default1547
} from "../icons/arrow-up-circle";
import { default as default1548 } from "../icons/circle-check-big";
import {
  default as default1549
} from "../icons/check-circle";
import { default as default1550 } from "../icons/circle-check";
import {
  default as default1551
} from "../icons/check-circle-2";
import { default as default1552 } from "../icons/circle-chevron-left";
import {
  default as default1553
} from "../icons/chevron-left-circle";
import { default as default1554 } from "../icons/circle-chevron-down";
import {
  default as default1555
} from "../icons/chevron-down-circle";
import { default as default1556 } from "../icons/circle-chevron-right";
import {
  default as default1557
} from "../icons/chevron-right-circle";
import { default as default1558 } from "../icons/circle-chevron-up";
import {
  default as default1559
} from "../icons/chevron-up-circle";
import { default as default1560 } from "../icons/circle-divide";
import {
  default as default1561
} from "../icons/divide-circle";
import { default as default1562 } from "../icons/circle-gauge";
import {
  default as default1563
} from "../icons/gauge-circle";
import { default as default1564 } from "../icons/circle-minus";
import {
  default as default1565
} from "../icons/minus-circle";
import { default as default1566 } from "../icons/circle-parking-off";
import {
  default as default1567
} from "../icons/parking-circle-off";
import { default as default1568 } from "../icons/circle-pause";
import {
  default as default1569
} from "../icons/pause-circle";
import { default as default1570 } from "../icons/circle-parking";
import {
  default as default1571
} from "../icons/parking-circle";
import { default as default1572 } from "../icons/circle-percent";
import {
  default as default1573
} from "../icons/percent-circle";
import { default as default1574 } from "../icons/circle-plus";
import {
  default as default1575
} from "../icons/plus-circle";
import { default as default1576 } from "../icons/circle-power";
import {
  default as default1577
} from "../icons/power-circle";
import { default as default1578 } from "../icons/circle-play";
import {
  default as default1579
} from "../icons/play-circle";
import { default as default1580 } from "../icons/circle-question-mark";
import {
  default as default1581
} from "../icons/help-circle";
import {
  default as default1582
} from "../icons/circle-help";
import { default as default1583 } from "../icons/circle-slash-2";
import {
  default as default1584
} from "../icons/circle-slashed";
import { default as default1585 } from "../icons/circle-stop";
import {
  default as default1586
} from "../icons/stop-circle";
import { default as default1587 } from "../icons/circle-user-round";
import {
  default as default1588
} from "../icons/user-circle-2";
import { default as default1589 } from "../icons/circle-x";
import {
  default as default1590
} from "../icons/x-circle";
import { default as default1591 } from "../icons/circle-user";
import {
  default as default1592
} from "../icons/user-circle";
import { default as default1593 } from "../icons/clipboard-pen-line";
import {
  default as default1594
} from "../icons/clipboard-signature";
import { default as default1595 } from "../icons/clipboard-pen";
import {
  default as default1596
} from "../icons/clipboard-edit";
import { default as default1597 } from "../icons/cloud-download";
import {
  default as default1598
} from "../icons/download-cloud";
import { default as default1599 } from "../icons/cloud-upload";
import {
  default as default1600
} from "../icons/upload-cloud";
import { default as default1601 } from "../icons/code-xml";
import {
  default as default1602
} from "../icons/code-2";
import { default as default1603 } from "../icons/columns-2";
import {
  default as default1604
} from "../icons/columns";
import { default as default1605 } from "../icons/columns-3-cog";
import {
  default as default1606
} from "../icons/columns-settings";
import {
  default as default1607
} from "../icons/table-config";
import { default as default1608 } from "../icons/columns-3";
import {
  default as default1609
} from "../icons/panels-left-right";
import { default as default1610 } from "../icons/contact-round";
import {
  default as default1611
} from "../icons/contact-2";
import { default as default1612 } from "../icons/diamond-percent";
import {
  default as default1613
} from "../icons/percent-diamond";
import { default as default1614 } from "../icons/ellipsis-vertical";
import {
  default as default1615
} from "../icons/more-vertical";
import { default as default1616 } from "../icons/earth";
import {
  default as default1617
} from "../icons/globe-2";
import { default as default1618 } from "../icons/ellipsis";
import {
  default as default1619
} from "../icons/more-horizontal";
import { default as default1620 } from "../icons/file-axis-3d";
import {
  default as default1621
} from "../icons/file-axis-3-d";
import { default as default1622 } from "../icons/file-badge";
import {
  default as default1623
} from "../icons/file-badge-2";
import { default as default1624 } from "../icons/file-braces-corner";
import {
  default as default1625
} from "../icons/file-json-2";
import { default as default1626 } from "../icons/file-braces";
import {
  default as default1627
} from "../icons/file-json";
import { default as default1628 } from "../icons/file-chart-column-increasing";
import {
  default as default1629
} from "../icons/file-bar-chart";
import { default as default1630 } from "../icons/file-chart-column";
import {
  default as default1631
} from "../icons/file-bar-chart-2";
import { default as default1632 } from "../icons/file-chart-pie";
import {
  default as default1633
} from "../icons/file-pie-chart";
import { default as default1634 } from "../icons/file-chart-line";
import {
  default as default1635
} from "../icons/file-line-chart";
import { default as default1636 } from "../icons/file-code-corner";
import {
  default as default1637
} from "../icons/file-code-2";
import { default as default1638 } from "../icons/file-cog";
import {
  default as default1639
} from "../icons/file-cog-2";
import { default as default1640 } from "../icons/file-check-corner";
import {
  default as default1641
} from "../icons/file-check-2";
import { default as default1642 } from "../icons/file-exclamation-point";
import {
  default as default1643
} from "../icons/file-warning";
import { default as default1644 } from "../icons/file-headphone";
import {
  default as default1645
} from "../icons/file-audio";
import {
  default as default1646
} from "../icons/file-audio-2";
import { default as default1647 } from "../icons/file-lock";
import {
  default as default1648
} from "../icons/file-lock-2";
import { default as default1649 } from "../icons/file-minus-corner";
import {
  default as default1650
} from "../icons/file-minus-2";
import { default as default1651 } from "../icons/file-key";
import {
  default as default1652
} from "../icons/file-key-2";
import { default as default1653 } from "../icons/file-pen-line";
import {
  default as default1654
} from "../icons/file-signature";
import { default as default1655 } from "../icons/file-pen";
import {
  default as default1656
} from "../icons/file-edit";
import { default as default1657 } from "../icons/file-play";
import {
  default as default1658
} from "../icons/file-video";
import { default as default1659 } from "../icons/file-plus-corner";
import {
  default as default1660
} from "../icons/file-plus-2";
import { default as default1661 } from "../icons/file-search-corner";
import {
  default as default1662
} from "../icons/file-search-2";
import { default as default1663 } from "../icons/file-question-mark";
import {
  default as default1664
} from "../icons/file-question";
import { default as default1665 } from "../icons/file-signal";
import {
  default as default1666
} from "../icons/file-volume-2";
import { default as default1667 } from "../icons/file-type-corner";
import {
  default as default1668
} from "../icons/file-type-2";
import { default as default1669 } from "../icons/file-video-camera";
import {
  default as default1670
} from "../icons/file-video-2";
import { default as default1671 } from "../icons/file-x-corner";
import {
  default as default1672
} from "../icons/file-x-2";
import { default as default1673 } from "../icons/folder-cog";
import {
  default as default1674
} from "../icons/folder-cog-2";
import { default as default1675 } from "../icons/folder-pen";
import {
  default as default1676
} from "../icons/folder-edit";
import { default as default1677 } from "../icons/funnel-x";
import {
  default as default1678
} from "../icons/filter-x";
import { default as default1679 } from "../icons/funnel";
import {
  default as default1680
} from "../icons/filter";
import { default as default1681 } from "../icons/git-commit-horizontal";
import {
  default as default1682
} from "../icons/git-commit";
import { default as default1683 } from "../icons/grid-2x2-check";
import {
  default as default1684
} from "../icons/grid-2-x-2-check";
import { default as default1685 } from "../icons/grid-2x2-plus";
import {
  default as default1686
} from "../icons/grid-2-x-2-plus";
import { default as default1687 } from "../icons/grid-2x2";
import {
  default as default1688
} from "../icons/grid-2-x-2";
import { default as default1689 } from "../icons/grid-2x2-x";
import {
  default as default1690
} from "../icons/grid-2-x-2-x";
import { default as default1691 } from "../icons/grid-3x3";
import {
  default as default1692
} from "../icons/grid";
import {
  default as default1693
} from "../icons/grid-3-x-3";
import { default as default1694 } from "../icons/hand-helping";
import {
  default as default1695
} from "../icons/helping-hand";
import { default as default1696 } from "../icons/hand-grab";
import {
  default as default1697
} from "../icons/grab";
import { default as default1698 } from "../icons/house";
import {
  default as default1699
} from "../icons/home";
import { default as default1700 } from "../icons/ice-cream-bowl";
import {
  default as default1701
} from "../icons/ice-cream-2";
import { default as default1702 } from "../icons/ice-cream-cone";
import {
  default as default1703
} from "../icons/ice-cream";
import { default as default1704 } from "../icons/laptop-minimal";
import {
  default as default1705
} from "../icons/laptop-2";
import { default as default1706 } from "../icons/layers";
import {
  default as default1707
} from "../icons/layers-3";
import { default as default1708 } from "../icons/list-indent-decrease";
import {
  default as default1709
} from "../icons/outdent";
import {
  default as default1710
} from "../icons/indent-decrease";
import { default as default1711 } from "../icons/list-indent-increase";
import {
  default as default1712
} from "../icons/indent";
import {
  default as default1713
} from "../icons/indent-increase";
import { default as default1714 } from "../icons/lock-open";
import {
  default as default1715
} from "../icons/unlock";
import { default as default1716 } from "../icons/loader-circle";
import {
  default as default1717
} from "../icons/loader-2";
import { default as default1718 } from "../icons/lock-keyhole-open";
import {
  default as default1719
} from "../icons/unlock-keyhole";
import { default as default1720 } from "../icons/mail-question-mark";
import {
  default as default1721
} from "../icons/mail-question";
import { default as default1722 } from "../icons/map-pin-pen";
import {
  default as default1723
} from "../icons/location-edit";
import { default as default1724 } from "../icons/message-circle-question-mark";
import {
  default as default1725
} from "../icons/message-circle-question";
import { default as default1726 } from "../icons/mic-vocal";
import {
  default as default1727
} from "../icons/mic-2";
import { default as default1728 } from "../icons/move-3d";
import {
  default as default1729
} from "../icons/move-3-d";
import { default as default1730 } from "../icons/octagon-alert";
import {
  default as default1731
} from "../icons/alert-octagon";
import { default as default1732 } from "../icons/octagon-pause";
import {
  default as default1733
} from "../icons/pause-octagon";
import { default as default1734 } from "../icons/octagon-x";
import {
  default as default1735
} from "../icons/x-octagon";
import { default as default1736 } from "../icons/panel-bottom-dashed";
import {
  default as default1737
} from "../icons/panel-bottom-inactive";
import { default as default1738 } from "../icons/panel-left-dashed";
import {
  default as default1739
} from "../icons/panel-left-inactive";
import { default as default1740 } from "../icons/paintbrush-vertical";
import {
  default as default1741
} from "../icons/paintbrush-2";
import { default as default1742 } from "../icons/panel-left-close";
import {
  default as default1743
} from "../icons/sidebar-close";
import { default as default1744 } from "../icons/panel-left";
import {
  default as default1745
} from "../icons/sidebar";
import { default as default1746 } from "../icons/panel-left-open";
import {
  default as default1747
} from "../icons/sidebar-open";
import { default as default1748 } from "../icons/panel-right-dashed";
import {
  default as default1749
} from "../icons/panel-right-inactive";
import { default as default1750 } from "../icons/panel-top-dashed";
import {
  default as default1751
} from "../icons/panel-top-inactive";
import { default as default1752 } from "../icons/panels-top-left";
import {
  default as default1753
} from "../icons/layout";
import { default as default1754 } from "../icons/pen";
import {
  default as default1755
} from "../icons/edit-2";
import { default as default1756 } from "../icons/pen-line";
import {
  default as default1757
} from "../icons/edit-3";
import { default as default1758 } from "../icons/rectangle-ellipsis";
import {
  default as default1759
} from "../icons/form-input";
import { default as default1760 } from "../icons/plug-zap";
import {
  default as default1761
} from "../icons/plug-zap-2";
import { default as default1762 } from "../icons/rotate-3d";
import {
  default as default1763
} from "../icons/rotate-3-d";
import { default as default1764 } from "../icons/rows-2";
import {
  default as default1765
} from "../icons/rows";
import { default as default1766 } from "../icons/rows-3";
import {
  default as default1767
} from "../icons/panels-top-bottom";
import { default as default1768 } from "../icons/scale-3d";
import {
  default as default1769
} from "../icons/scale-3-d";
import { default as default1770 } from "../icons/send-horizontal";
import {
  default as default1771
} from "../icons/send-horizonal";
import { default as default1772 } from "../icons/shield-question-mark";
import {
  default as default1773
} from "../icons/shield-question";
import { default as default1774 } from "../icons/shield-x";
import {
  default as default1775
} from "../icons/shield-close";
import { default as default1776 } from "../icons/sliders-vertical";
import {
  default as default1777
} from "../icons/sliders";
import { default as default1778 } from "../icons/sparkles";
import {
  default as default1779
} from "../icons/stars";
import { default as default1780 } from "../icons/square-activity";
import {
  default as default1781
} from "../icons/activity-square";
import { default as default1782 } from "../icons/square-arrow-down-left";
import {
  default as default1783
} from "../icons/arrow-down-left-square";
import { default as default1784 } from "../icons/square-arrow-down-right";
import {
  default as default1785
} from "../icons/arrow-down-right-square";
import { default as default1786 } from "../icons/square-arrow-down";
import {
  default as default1787
} from "../icons/arrow-down-square";
import { default as default1788 } from "../icons/square-arrow-left";
import {
  default as default1789
} from "../icons/arrow-left-square";
import { default as default1790 } from "../icons/square-arrow-out-down-right";
import {
  default as default1791
} from "../icons/arrow-down-right-from-square";
import { default as default1792 } from "../icons/square-arrow-out-down-left";
import {
  default as default1793
} from "../icons/arrow-down-left-from-square";
import { default as default1794 } from "../icons/square-arrow-out-up-left";
import {
  default as default1795
} from "../icons/arrow-up-left-from-square";
import { default as default1796 } from "../icons/square-arrow-out-up-right";
import {
  default as default1797
} from "../icons/arrow-up-right-from-square";
import { default as default1798 } from "../icons/square-arrow-right";
import {
  default as default1799
} from "../icons/arrow-right-square";
import { default as default1800 } from "../icons/square-arrow-up-left";
import {
  default as default1801
} from "../icons/arrow-up-left-square";
import { default as default1802 } from "../icons/square-arrow-up-right";
import {
  default as default1803
} from "../icons/arrow-up-right-square";
import { default as default1804 } from "../icons/square-arrow-up";
import {
  default as default1805
} from "../icons/arrow-up-square";
import { default as default1806 } from "../icons/square-bottom-dashed-scissors";
import {
  default as default1807
} from "../icons/scissors-square-dashed-bottom";
import { default as default1808 } from "../icons/square-asterisk";
import {
  default as default1809
} from "../icons/asterisk-square";
import { default as default1810 } from "../icons/square-centerline-dashed-horizontal";
import {
  default as default1811
} from "../icons/flip-horizontal";
import { default as default1812 } from "../icons/square-centerline-dashed-vertical";
import {
  default as default1813
} from "../icons/flip-vertical";
import { default as default1814 } from "../icons/square-chart-gantt";
import {
  default as default1815
} from "../icons/gantt-chart-square";
import {
  default as default1816
} from "../icons/square-gantt-chart";
import { default as default1817 } from "../icons/square-chevron-down";
import {
  default as default1818
} from "../icons/chevron-down-square";
import { default as default1819 } from "../icons/square-check-big";
import {
  default as default1820
} from "../icons/check-square";
import { default as default1821 } from "../icons/square-check";
import {
  default as default1822
} from "../icons/check-square-2";
import { default as default1823 } from "../icons/square-chevron-left";
import {
  default as default1824
} from "../icons/chevron-left-square";
import { default as default1825 } from "../icons/square-chevron-up";
import {
  default as default1826
} from "../icons/chevron-up-square";
import { default as default1827 } from "../icons/square-code";
import {
  default as default1828
} from "../icons/code-square";
import { default as default1829 } from "../icons/square-chevron-right";
import {
  default as default1830
} from "../icons/chevron-right-square";
import { default as default1831 } from "../icons/square-dashed-kanban";
import {
  default as default1832
} from "../icons/kanban-square-dashed";
import { default as default1833 } from "../icons/square-dashed-mouse-pointer";
import {
  default as default1834
} from "../icons/mouse-pointer-square-dashed";
import { default as default1835 } from "../icons/square-dashed-text";
import {
  default as default1836
} from "../icons/text-selection";
import {
  default as default1837
} from "../icons/text-select";
import { default as default1838 } from "../icons/square-dashed";
import {
  default as default1839
} from "../icons/box-select";
import { default as default1840 } from "../icons/square-divide";
import {
  default as default1841
} from "../icons/divide-square";
import { default as default1842 } from "../icons/square-dot";
import {
  default as default1843
} from "../icons/dot-square";
import { default as default1844 } from "../icons/square-function";
import {
  default as default1845
} from "../icons/function-square";
import { default as default1846 } from "../icons/square-equal";
import {
  default as default1847
} from "../icons/equal-square";
import { default as default1848 } from "../icons/square-kanban";
import {
  default as default1849
} from "../icons/kanban-square";
import { default as default1850 } from "../icons/square-library";
import {
  default as default1851
} from "../icons/library-square";
import { default as default1852 } from "../icons/square-m";
import {
  default as default1853
} from "../icons/m-square";
import { default as default1854 } from "../icons/square-menu";
import {
  default as default1855
} from "../icons/menu-square";
import { default as default1856 } from "../icons/square-minus";
import {
  default as default1857
} from "../icons/minus-square";
import { default as default1858 } from "../icons/square-mouse-pointer";
import {
  default as default1859
} from "../icons/inspect";
import { default as default1860 } from "../icons/square-parking";
import {
  default as default1861
} from "../icons/parking-square";
import { default as default1862 } from "../icons/square-percent";
import {
  default as default1863
} from "../icons/percent-square";
import { default as default1864 } from "../icons/square-pen";
import {
  default as default1865
} from "../icons/pen-box";
import {
  default as default1866
} from "../icons/edit";
import {
  default as default1867
} from "../icons/pen-square";
import { default as default1868 } from "../icons/square-parking-off";
import {
  default as default1869
} from "../icons/parking-square-off";
import { default as default1870 } from "../icons/square-pi";
import {
  default as default1871
} from "../icons/pi-square";
import { default as default1872 } from "../icons/square-pilcrow";
import {
  default as default1873
} from "../icons/pilcrow-square";
import { default as default1874 } from "../icons/square-play";
import {
  default as default1875
} from "../icons/play-square";
import { default as default1876 } from "../icons/square-plus";
import {
  default as default1877
} from "../icons/plus-square";
import { default as default1878 } from "../icons/square-scissors";
import {
  default as default1879
} from "../icons/scissors-square";
import { default as default1880 } from "../icons/square-sigma";
import {
  default as default1881
} from "../icons/sigma-square";
import { default as default1882 } from "../icons/square-power";
import {
  default as default1883
} from "../icons/power-square";
import { default as default1884 } from "../icons/square-slash";
import {
  default as default1885
} from "../icons/slash-square";
import { default as default1886 } from "../icons/square-split-horizontal";
import {
  default as default1887
} from "../icons/split-square-horizontal";
import { default as default1888 } from "../icons/square-split-vertical";
import {
  default as default1889
} from "../icons/split-square-vertical";
import { default as default1890 } from "../icons/square-terminal";
import {
  default as default1891
} from "../icons/terminal-square";
import { default as default1892 } from "../icons/square-user-round";
import {
  default as default1893
} from "../icons/user-square-2";
import { default as default1894 } from "../icons/square-user";
import {
  default as default1895
} from "../icons/user-square";
import { default as default1896 } from "../icons/square-x";
import {
  default as default1897
} from "../icons/x-square";
import { default as default1898 } from "../icons/test-tube-diagonal";
import {
  default as default1899
} from "../icons/test-tube-2";
import { default as default1900 } from "../icons/text-align-center";
import {
  default as default1901
} from "../icons/align-center";
import { default as default1902 } from "../icons/text-align-end";
import {
  default as default1903
} from "../icons/align-right";
import { default as default1904 } from "../icons/text-align-justify";
import {
  default as default1905
} from "../icons/align-justify";
import { default as default1906 } from "../icons/text-align-start";
import {
  default as default1907
} from "../icons/text";
import {
  default as default1908
} from "../icons/align-left";
import { default as default1909 } from "../icons/text-initial";
import {
  default as default1910
} from "../icons/letter-text";
import { default as default1911 } from "../icons/text-wrap";
import {
  default as default1912
} from "../icons/wrap-text";
import { default as default1913 } from "../icons/tram-front";
import {
  default as default1914
} from "../icons/train";
import { default as default1915 } from "../icons/tree-palm";
import {
  default as default1916
} from "../icons/palmtree";
import { default as default1917 } from "../icons/triangle-alert";
import {
  default as default1918
} from "../icons/alert-triangle";
import { default as default1919 } from "../icons/tv-minimal";
import {
  default as default1920
} from "../icons/tv-2";
import { default as default1921 } from "../icons/university";
import {
  default as default1922
} from "../icons/school-2";
import { default as default1923 } from "../icons/user-round-check";
import {
  default as default1924
} from "../icons/user-check-2";
import { default as default1925 } from "../icons/user-round-cog";
import {
  default as default1926
} from "../icons/user-cog-2";
import { default as default1927 } from "../icons/user-round-minus";
import {
  default as default1928
} from "../icons/user-minus-2";
import { default as default1929 } from "../icons/user-round-plus";
import {
  default as default1930
} from "../icons/user-plus-2";
import { default as default1931 } from "../icons/user-round-x";
import {
  default as default1932
} from "../icons/user-x-2";
import { default as default1933 } from "../icons/user-round";
import {
  default as default1934
} from "../icons/user-2";
import { default as default1935 } from "../icons/users-round";
import {
  default as default1936
} from "../icons/users-2";
import { default as default1937 } from "../icons/utensils-crossed";
import {
  default as default1938
} from "../icons/fork-knife-crossed";
import { default as default1939 } from "../icons/utensils";
import {
  default as default1940
} from "../icons/fork-knife";
import { default as default1941 } from "../icons/wallet-minimal";
import {
  default as default1942
} from "../icons/wallet-2";
import { default as default1943 } from "../icons/wand-sparkles";
import {
  default as default1944
} from "../icons/wand-2";
export {
  default2 as AArrowDownIcon,
  default3 as AArrowUpIcon,
  default4 as ALargeSmallIcon,
  default5 as AccessibilityIcon,
  default6 as ActivityIcon,
  default1781 as ActivitySquareIcon,
  default8 as AirVentIcon,
  default7 as AirplayIcon,
  default1471 as AlarmCheckIcon,
  default1470 as AlarmClockCheckIcon,
  default10 as AlarmClockIcon,
  default1474 as AlarmClockMinusIcon,
  default9 as AlarmClockOffIcon,
  default1472 as AlarmClockPlusIcon,
  default1475 as AlarmMinusIcon,
  default1473 as AlarmPlusIcon,
  default11 as AlarmSmokeIcon,
  default12 as AlbumIcon,
  default1531 as AlertCircleIcon,
  default1731 as AlertOctagonIcon,
  default1918 as AlertTriangleIcon,
  default13 as AlignCenterHorizontalIcon,
  default1901 as AlignCenterIcon,
  default14 as AlignCenterVerticalIcon,
  default15 as AlignEndHorizontalIcon,
  default16 as AlignEndVerticalIcon,
  default17 as AlignHorizontalDistributeCenterIcon,
  default18 as AlignHorizontalDistributeEndIcon,
  default19 as AlignHorizontalDistributeStartIcon,
  default20 as AlignHorizontalJustifyCenterIcon,
  default21 as AlignHorizontalJustifyEndIcon,
  default22 as AlignHorizontalJustifyStartIcon,
  default23 as AlignHorizontalSpaceAroundIcon,
  default24 as AlignHorizontalSpaceBetweenIcon,
  default1905 as AlignJustifyIcon,
  default1908 as AlignLeftIcon,
  default1903 as AlignRightIcon,
  default25 as AlignStartHorizontalIcon,
  default26 as AlignStartVerticalIcon,
  default27 as AlignVerticalDistributeCenterIcon,
  default28 as AlignVerticalDistributeEndIcon,
  default29 as AlignVerticalDistributeStartIcon,
  default30 as AlignVerticalJustifyCenterIcon,
  default31 as AlignVerticalJustifyEndIcon,
  default32 as AlignVerticalJustifyStartIcon,
  default33 as AlignVerticalSpaceAroundIcon,
  default34 as AlignVerticalSpaceBetweenIcon,
  default35 as AmbulanceIcon,
  default36 as AmpersandIcon,
  default37 as AmpersandsIcon,
  default38 as AmphoraIcon,
  default40 as AnchorIcon,
  default39 as AngryIcon,
  default41 as AnnoyedIcon,
  default42 as AntennaIcon,
  default43 as AnvilIcon,
  default44 as ApertureIcon,
  default46 as AppWindowIcon,
  default45 as AppWindowMacIcon,
  default47 as AppleIcon,
  default51 as ArchiveIcon,
  default48 as ArchiveRestoreIcon,
  default49 as ArchiveXIcon,
  default1505 as AreaChartIcon,
  default50 as ArmchairIcon,
  default52 as ArrowBigDownDashIcon,
  default53 as ArrowBigDownIcon,
  default54 as ArrowBigLeftDashIcon,
  default56 as ArrowBigLeftIcon,
  default55 as ArrowBigRightDashIcon,
  default57 as ArrowBigRightIcon,
  default58 as ArrowBigUpDashIcon,
  default59 as ArrowBigUpIcon,
  default1466 as ArrowDown01Icon,
  default1467 as ArrowDown10Icon,
  default1476 as ArrowDownAZIcon,
  default1477 as ArrowDownAzIcon,
  default1533 as ArrowDownCircleIcon,
  default60 as ArrowDownFromLineIcon,
  default67 as ArrowDownIcon,
  default1537 as ArrowDownLeftFromCircleIcon,
  default1793 as ArrowDownLeftFromSquareIcon,
  default61 as ArrowDownLeftIcon,
  default1783 as ArrowDownLeftSquareIcon,
  default63 as ArrowDownNarrowWideIcon,
  default1539 as ArrowDownRightFromCircleIcon,
  default1791 as ArrowDownRightFromSquareIcon,
  default62 as ArrowDownRightIcon,
  default1785 as ArrowDownRightSquareIcon,
  default1787 as ArrowDownSquareIcon,
  default64 as ArrowDownToDotIcon,
  default65 as ArrowDownToLineIcon,
  default66 as ArrowDownUpIcon,
  default1478 as ArrowDownWideNarrowIcon,
  default1480 as ArrowDownZAIcon,
  default1481 as ArrowDownZaIcon,
  default1535 as ArrowLeftCircleIcon,
  default68 as ArrowLeftFromLineIcon,
  default71 as ArrowLeftIcon,
  default69 as ArrowLeftRightIcon,
  default1789 as ArrowLeftSquareIcon,
  default70 as ArrowLeftToLineIcon,
  default1545 as ArrowRightCircleIcon,
  default72 as ArrowRightFromLineIcon,
  default75 as ArrowRightIcon,
  default73 as ArrowRightLeftIcon,
  default1799 as ArrowRightSquareIcon,
  default74 as ArrowRightToLineIcon,
  default1468 as ArrowUp01Icon,
  default1469 as ArrowUp10Icon,
  default1482 as ArrowUpAZIcon,
  default1483 as ArrowUpAzIcon,
  default1547 as ArrowUpCircleIcon,
  default76 as ArrowUpDownIcon,
  default77 as ArrowUpFromDotIcon,
  default79 as ArrowUpFromLineIcon,
  default83 as ArrowUpIcon,
  default1543 as ArrowUpLeftFromCircleIcon,
  default1795 as ArrowUpLeftFromSquareIcon,
  default78 as ArrowUpLeftIcon,
  default1801 as ArrowUpLeftSquareIcon,
  default1484 as ArrowUpNarrowWideIcon,
  default1541 as ArrowUpRightFromCircleIcon,
  default1797 as ArrowUpRightFromSquareIcon,
  default80 as ArrowUpRightIcon,
  default1803 as ArrowUpRightSquareIcon,
  default1805 as ArrowUpSquareIcon,
  default82 as ArrowUpToLineIcon,
  default81 as ArrowUpWideNarrowIcon,
  default1486 as ArrowUpZAIcon,
  default1487 as ArrowUpZaIcon,
  default84 as ArrowsUpFromLineIcon,
  default85 as AsteriskIcon,
  default1809 as AsteriskSquareIcon,
  default87 as AtSignIcon,
  default86 as AtomIcon,
  default88 as AudioLinesIcon,
  default89 as AudioWaveformIcon,
  default91 as AwardIcon,
  default90 as AxeIcon,
  default1489 as Axis3DIcon,
  default1488 as Axis3dIcon,
  default93 as BabyIcon,
  default92 as BackpackIcon,
  default94 as BadgeAlertIcon,
  default95 as BadgeCentIcon,
  default1490 as BadgeCheckIcon,
  default96 as BadgeDollarSignIcon,
  default97 as BadgeEuroIcon,
  default1493 as BadgeHelpIcon,
  default110 as BadgeIcon,
  default98 as BadgeIndianRupeeIcon,
  default99 as BadgeInfoIcon,
  default103 as BadgeJapaneseYenIcon,
  default100 as BadgeMinusIcon,
  default101 as BadgePercentIcon,
  default105 as BadgePlusIcon,
  default102 as BadgePoundSterlingIcon,
  default1492 as BadgeQuestionMarkIcon,
  default104 as BadgeRussianRubleIcon,
  default106 as BadgeSwissFrancIcon,
  default107 as BadgeTurkishLiraIcon,
  default108 as BadgeXIcon,
  default109 as BaggageClaimIcon,
  default112 as BalloonIcon,
  default111 as BanIcon,
  default113 as BananaIcon,
  default114 as BandageIcon,
  default115 as BanknoteArrowDownIcon,
  default116 as BanknoteArrowUpIcon,
  default118 as BanknoteIcon,
  default117 as BanknoteXIcon,
  default1525 as BarChart2Icon,
  default1517 as BarChart3Icon,
  default1513 as BarChart4Icon,
  default1515 as BarChartBigIcon,
  default1507 as BarChartHorizontalBigIcon,
  default1509 as BarChartHorizontalIcon,
  default1519 as BarChartIcon,
  default119 as BarcodeIcon,
  default120 as BarrelIcon,
  default121 as BaselineIcon,
  default122 as BathIcon,
  default123 as BatteryChargingIcon,
  default124 as BatteryFullIcon,
  default129 as BatteryIcon,
  default125 as BatteryLowIcon,
  default126 as BatteryMediumIcon,
  default127 as BatteryPlusIcon,
  default128 as BatteryWarningIcon,
  default130 as BeakerIcon,
  default132 as BeanIcon,
  default131 as BeanOffIcon,
  default133 as BedDoubleIcon,
  default135 as BedIcon,
  default134 as BedSingleIcon,
  default136 as BeefIcon,
  default137 as BeefOffIcon,
  default139 as BeerIcon,
  default138 as BeerOffIcon,
  default140 as BellCheckIcon,
  default141 as BellDotIcon,
  default142 as BellElectricIcon,
  default147 as BellIcon,
  default143 as BellMinusIcon,
  default144 as BellOffIcon,
  default146 as BellPlusIcon,
  default145 as BellRingIcon,
  default1495 as BetweenHorizonalEndIcon,
  default1497 as BetweenHorizonalStartIcon,
  default1494 as BetweenHorizontalEndIcon,
  default1496 as BetweenHorizontalStartIcon,
  default148 as BetweenVerticalEndIcon,
  default149 as BetweenVerticalStartIcon,
  default150 as BicepsFlexedIcon,
  default151 as BikeIcon,
  default152 as BinaryIcon,
  default154 as BinocularsIcon,
  default153 as BiohazardIcon,
  default155 as BirdIcon,
  default156 as BirdhouseIcon,
  default157 as BitcoinIcon,
  default158 as BlendIcon,
  default159 as BlindsIcon,
  default160 as BlocksIcon,
  default161 as BluetoothConnectedIcon,
  default164 as BluetoothIcon,
  default162 as BluetoothOffIcon,
  default163 as BluetoothSearchingIcon,
  default165 as BoldIcon,
  default166 as BoltIcon,
  default167 as BombIcon,
  default168 as BoneIcon,
  default169 as BookAIcon,
  default170 as BookAlertIcon,
  default171 as BookAudioIcon,
  default172 as BookCheckIcon,
  default173 as BookCopyIcon,
  default1498 as BookDashedIcon,
  default174 as BookDownIcon,
  default175 as BookHeadphonesIcon,
  default176 as BookHeartIcon,
  default196 as BookIcon,
  default177 as BookImageIcon,
  default178 as BookKeyIcon,
  default179 as BookLockIcon,
  default180 as BookMarkedIcon,
  default181 as BookMinusIcon,
  default182 as BookOpenCheckIcon,
  default184 as BookOpenIcon,
  default183 as BookOpenTextIcon,
  default185 as BookPlusIcon,
  default186 as BookSearchIcon,
  default1499 as BookTemplateIcon,
  default187 as BookTextIcon,
  default188 as BookTypeIcon,
  default189 as BookUp2Icon,
  default190 as BookUpIcon,
  default191 as BookUserIcon,
  default192 as BookXIcon,
  default193 as BookmarkCheckIcon,
  default199 as BookmarkIcon,
  default194 as BookmarkMinusIcon,
  default195 as BookmarkOffIcon,
  default198 as BookmarkPlusIcon,
  default197 as BookmarkXIcon,
  default200 as BoomBoxIcon,
  default203 as BotIcon,
  default201 as BotMessageSquareIcon,
  default202 as BotOffIcon,
  default204 as BottleWineIcon,
  default205 as BowArrowIcon,
  default206 as BoxIcon,
  default1839 as BoxSelectIcon,
  default207 as BoxesIcon,
  default1500 as BracesIcon,
  default208 as BracketsIcon,
  default209 as BrainCircuitIcon,
  default210 as BrainCogIcon,
  default211 as BrainIcon,
  default212 as BrickWallFireIcon,
  default214 as BrickWallIcon,
  default213 as BrickWallShieldIcon,
  default215 as BriefcaseBusinessIcon,
  default216 as BriefcaseConveyorBeltIcon,
  default218 as BriefcaseIcon,
  default217 as BriefcaseMedicalIcon,
  default219 as BringToFrontIcon,
  default220 as BrushCleaningIcon,
  default221 as BrushIcon,
  default222 as BubblesIcon,
  default225 as BugIcon,
  default223 as BugOffIcon,
  default224 as BugPlayIcon,
  default226 as Building2Icon,
  default227 as BuildingIcon,
  default228 as BusFrontIcon,
  default229 as BusIcon,
  default230 as CableCarIcon,
  default232 as CableIcon,
  default233 as CakeIcon,
  default231 as CakeSliceIcon,
  default234 as CalculatorIcon,
  default235 as Calendar1Icon,
  default236 as CalendarArrowDownIcon,
  default237 as CalendarArrowUpIcon,
  default238 as CalendarCheck2Icon,
  default239 as CalendarCheckIcon,
  default240 as CalendarClockIcon,
  default241 as CalendarCogIcon,
  default242 as CalendarDaysIcon,
  default244 as CalendarFoldIcon,
  default243 as CalendarHeartIcon,
  default255 as CalendarIcon,
  default245 as CalendarMinus2Icon,
  default246 as CalendarMinusIcon,
  default247 as CalendarOffIcon,
  default248 as CalendarPlus2Icon,
  default249 as CalendarPlusIcon,
  default250 as CalendarRangeIcon,
  default252 as CalendarSearchIcon,
  default251 as CalendarSyncIcon,
  default253 as CalendarX2Icon,
  default254 as CalendarXIcon,
  default256 as CalendarsIcon,
  default258 as CameraIcon,
  default257 as CameraOffIcon,
  default1511 as CandlestickChartIcon,
  default259 as CandyCaneIcon,
  default261 as CandyIcon,
  default260 as CandyOffIcon,
  default262 as CannabisIcon,
  default263 as CannabisOffIcon,
  default1502 as CaptionsIcon,
  default266 as CaptionsOffIcon,
  default264 as CarFrontIcon,
  default269 as CarIcon,
  default265 as CarTaxiFrontIcon,
  default267 as CaravanIcon,
  default268 as CardSimIcon,
  default270 as CarrotIcon,
  default273 as CaseLowerIcon,
  default271 as CaseSensitiveIcon,
  default272 as CaseUpperIcon,
  default274 as CassetteTapeIcon,
  default275 as CastIcon,
  default276 as CastleIcon,
  default277 as CatIcon,
  default279 as CctvIcon,
  default278 as CctvOffIcon,
  default1504 as ChartAreaIcon,
  default1506 as ChartBarBigIcon,
  default280 as ChartBarDecreasingIcon,
  default1508 as ChartBarIcon,
  default281 as ChartBarIncreasingIcon,
  default282 as ChartBarStackedIcon,
  default1510 as ChartCandlestickIcon,
  default1514 as ChartColumnBigIcon,
  default283 as ChartColumnDecreasingIcon,
  default1516 as ChartColumnIcon,
  default1512 as ChartColumnIncreasingIcon,
  default284 as ChartColumnStackedIcon,
  default285 as ChartGanttIcon,
  default1520 as ChartLineIcon,
  default286 as ChartNetworkIcon,
  default287 as ChartNoAxesColumnDecreasingIcon,
  default1524 as ChartNoAxesColumnIcon,
  default1518 as ChartNoAxesColumnIncreasingIcon,
  default288 as ChartNoAxesCombinedIcon,
  default1522 as ChartNoAxesGanttIcon,
  default1526 as ChartPieIcon,
  default1528 as ChartScatterIcon,
  default289 as ChartSplineIcon,
  default290 as CheckCheckIcon,
  default1551 as CheckCircle2Icon,
  default1549 as CheckCircleIcon,
  default292 as CheckIcon,
  default291 as CheckLineIcon,
  default1822 as CheckSquare2Icon,
  default1820 as CheckSquareIcon,
  default294 as ChefHatIcon,
  default293 as CherryIcon,
  default295 as ChessBishopIcon,
  default296 as ChessKingIcon,
  default297 as ChessKnightIcon,
  default298 as ChessPawnIcon,
  default299 as ChessQueenIcon,
  default300 as ChessRookIcon,
  default1555 as ChevronDownCircleIcon,
  default301 as ChevronDownIcon,
  default1818 as ChevronDownSquareIcon,
  default302 as ChevronFirstIcon,
  default305 as ChevronLastIcon,
  default1553 as ChevronLeftCircleIcon,
  default303 as ChevronLeftIcon,
  default1824 as ChevronLeftSquareIcon,
  default1557 as ChevronRightCircleIcon,
  default304 as ChevronRightIcon,
  default1830 as ChevronRightSquareIcon,
  default1559 as ChevronUpCircleIcon,
  default306 as ChevronUpIcon,
  default1826 as ChevronUpSquareIcon,
  default308 as ChevronsDownIcon,
  default307 as ChevronsDownUpIcon,
  default311 as ChevronsLeftIcon,
  default309 as ChevronsLeftRightEllipsisIcon,
  default310 as ChevronsLeftRightIcon,
  default314 as ChevronsRightIcon,
  default312 as ChevronsRightLeftIcon,
  default313 as ChevronsUpDownIcon,
  default316 as ChevronsUpIcon,
  default315 as ChurchIcon,
  default318 as CigaretteIcon,
  default317 as CigaretteOffIcon,
  default1530 as CircleAlertIcon,
  default1532 as CircleArrowDownIcon,
  default1534 as CircleArrowLeftIcon,
  default1536 as CircleArrowOutDownLeftIcon,
  default1538 as CircleArrowOutDownRightIcon,
  default1542 as CircleArrowOutUpLeftIcon,
  default1540 as CircleArrowOutUpRightIcon,
  default1544 as CircleArrowRightIcon,
  default1546 as CircleArrowUpIcon,
  default1548 as CircleCheckBigIcon,
  default1550 as CircleCheckIcon,
  default1554 as CircleChevronDownIcon,
  default1552 as CircleChevronLeftIcon,
  default1556 as CircleChevronRightIcon,
  default1558 as CircleChevronUpIcon,
  default319 as CircleDashedIcon,
  default1560 as CircleDivideIcon,
  default320 as CircleDollarSignIcon,
  default322 as CircleDotDashedIcon,
  default323 as CircleDotIcon,
  default321 as CircleEllipsisIcon,
  default327 as CircleEqualIcon,
  default324 as CircleFadingArrowUpIcon,
  default325 as CircleFadingPlusIcon,
  default1562 as CircleGaugeIcon,
  default1582 as CircleHelpIcon,
  default333 as CircleIcon,
  default1564 as CircleMinusIcon,
  default326 as CircleOffIcon,
  default1570 as CircleParkingIcon,
  default1566 as CircleParkingOffIcon,
  default1568 as CirclePauseIcon,
  default1572 as CirclePercentIcon,
  default328 as CirclePileIcon,
  default1578 as CirclePlayIcon,
  default1574 as CirclePlusIcon,
  default329 as CirclePoundSterlingIcon,
  default1576 as CirclePowerIcon,
  default1580 as CircleQuestionMarkIcon,
  default1583 as CircleSlash2Icon,
  default330 as CircleSlashIcon,
  default1584 as CircleSlashedIcon,
  default331 as CircleSmallIcon,
  default332 as CircleStarIcon,
  default1585 as CircleStopIcon,
  default1591 as CircleUserIcon,
  default1587 as CircleUserRoundIcon,
  default1589 as CircleXIcon,
  default335 as CircuitBoardIcon,
  default334 as CitrusIcon,
  default336 as ClapperboardIcon,
  default337 as ClipboardCheckIcon,
  default340 as ClipboardClockIcon,
  default338 as ClipboardCopyIcon,
  default1596 as ClipboardEditIcon,
  default346 as ClipboardIcon,
  default339 as ClipboardListIcon,
  default342 as ClipboardMinusIcon,
  default341 as ClipboardPasteIcon,
  default1595 as ClipboardPenIcon,
  default1593 as ClipboardPenLineIcon,
  default344 as ClipboardPlusIcon,
  default1594 as ClipboardSignatureIcon,
  default343 as ClipboardTypeIcon,
  default345 as ClipboardXIcon,
  default349 as Clock10Icon,
  default348 as Clock11Icon,
  default350 as Clock12Icon,
  default347 as Clock1Icon,
  default351 as Clock2Icon,
  default352 as Clock3Icon,
  default353 as Clock4Icon,
  default354 as Clock5Icon,
  default355 as Clock6Icon,
  default356 as Clock7Icon,
  default357 as Clock8Icon,
  default358 as Clock9Icon,
  default359 as ClockAlertIcon,
  default361 as ClockArrowDownIcon,
  default360 as ClockArrowUpIcon,
  default362 as ClockCheckIcon,
  default363 as ClockFadingIcon,
  default365 as ClockIcon,
  default364 as ClockPlusIcon,
  default366 as ClosedCaptionIcon,
  default368 as CloudAlertIcon,
  default367 as CloudBackupIcon,
  default371 as CloudCheckIcon,
  default370 as CloudCogIcon,
  default1597 as CloudDownloadIcon,
  default369 as CloudDrizzleIcon,
  default372 as CloudFogIcon,
  default373 as CloudHailIcon,
  default384 as CloudIcon,
  default374 as CloudLightningIcon,
  default376 as CloudMoonIcon,
  default375 as CloudMoonRainIcon,
  default377 as CloudOffIcon,
  default381 as CloudRainIcon,
  default378 as CloudRainWindIcon,
  default379 as CloudSnowIcon,
  default382 as CloudSunIcon,
  default380 as CloudSunRainIcon,
  default383 as CloudSyncIcon,
  default1599 as CloudUploadIcon,
  default385 as CloudyIcon,
  default386 as CloverIcon,
  default387 as ClubIcon,
  default1602 as Code2Icon,
  default392 as CodeIcon,
  default1828 as CodeSquareIcon,
  default1601 as CodeXmlIcon,
  default388 as CoffeeIcon,
  default389 as CogIcon,
  default390 as CoinsIcon,
  default1603 as Columns2Icon,
  default1605 as Columns3CogIcon,
  default1608 as Columns3Icon,
  default395 as Columns4Icon,
  default1604 as ColumnsIcon,
  default1606 as ColumnsSettingsIcon,
  default391 as CombineIcon,
  default393 as CommandIcon,
  default394 as CompassIcon,
  default396 as ComponentIcon,
  default399 as ComputerIcon,
  default397 as ConciergeBellIcon,
  default398 as ConeIcon,
  default400 as ConstructionIcon,
  default1611 as Contact2Icon,
  default401 as ContactIcon,
  default1610 as ContactRoundIcon,
  default402 as ContainerIcon,
  default403 as ContrastIcon,
  default404 as CookieIcon,
  default405 as CookingPotIcon,
  default406 as CopyCheckIcon,
  default411 as CopyIcon,
  default407 as CopyMinusIcon,
  default408 as CopyPlusIcon,
  default409 as CopySlashIcon,
  default410 as CopyXIcon,
  default412 as CopyleftIcon,
  default413 as CopyrightIcon,
  default414 as CornerDownLeftIcon,
  default415 as CornerDownRightIcon,
  default416 as CornerLeftDownIcon,
  default417 as CornerLeftUpIcon,
  default418 as CornerRightDownIcon,
  default419 as CornerRightUpIcon,
  default420 as CornerUpLeftIcon,
  default421 as CornerUpRightIcon,
  default422 as CpuIcon,
  default423 as CreativeCommonsIcon,
  default424 as CreditCardIcon,
  default425 as CroissantIcon,
  default426 as CropIcon,
  default427 as CrossIcon,
  default428 as CrosshairIcon,
  default429 as CrownIcon,
  default430 as CuboidIcon,
  default431 as CupSodaIcon,
  default1501 as CurlyBracesIcon,
  default432 as CurrencyIcon,
  default433 as CylinderIcon,
  default434 as DamIcon,
  default435 as DatabaseBackupIcon,
  default441 as DatabaseIcon,
  default436 as DatabaseSearchIcon,
  default437 as DatabaseZapIcon,
  default438 as DecimalsArrowLeftIcon,
  default439 as DecimalsArrowRightIcon,
  default440 as DeleteIcon,
  default442 as DessertIcon,
  default443 as DiameterIcon,
  default447 as DiamondIcon,
  default444 as DiamondMinusIcon,
  default1612 as DiamondPercentIcon,
  default445 as DiamondPlusIcon,
  default448 as Dice1Icon,
  default446 as Dice2Icon,
  default449 as Dice3Icon,
  default450 as Dice4Icon,
  default451 as Dice5Icon,
  default452 as Dice6Icon,
  default453 as DicesIcon,
  default454 as DiffIcon,
  default455 as Disc2Icon,
  default456 as Disc3Icon,
  default459 as DiscAlbumIcon,
  default457 as DiscIcon,
  default1561 as DivideCircleIcon,
  default458 as DivideIcon,
  default1841 as DivideSquareIcon,
  default461 as DnaIcon,
  default460 as DnaOffIcon,
  default462 as DockIcon,
  default463 as DogIcon,
  default464 as DollarSignIcon,
  default465 as DonutIcon,
  default467 as DoorClosedIcon,
  default466 as DoorClosedLockedIcon,
  default468 as DoorOpenIcon,
  default469 as DotIcon,
  default1843 as DotSquareIcon,
  default1598 as DownloadCloudIcon,
  default470 as DownloadIcon,
  default471 as DraftingCompassIcon,
  default472 as DramaIcon,
  default473 as DrillIcon,
  default474 as DroneIcon,
  default476 as DropletIcon,
  default475 as DropletOffIcon,
  default477 as DropletsIcon,
  default478 as DrumIcon,
  default479 as DrumstickIcon,
  default480 as DumbbellIcon,
  default482 as EarIcon,
  default481 as EarOffIcon,
  default1616 as EarthIcon,
  default483 as EarthLockIcon,
  default484 as EclipseIcon,
  default1755 as Edit2Icon,
  default1757 as Edit3Icon,
  default1866 as EditIcon,
  default485 as EggFriedIcon,
  default487 as EggIcon,
  default486 as EggOffIcon,
  default488 as EllipseIcon,
  default1618 as EllipsisIcon,
  default1614 as EllipsisVerticalIcon,
  default489 as EqualApproximatelyIcon,
  default491 as EqualIcon,
  default490 as EqualNotIcon,
  default1847 as EqualSquareIcon,
  default492 as EraserIcon,
  default493 as EthernetPortIcon,
  default494 as EuroIcon,
  default495 as EvChargerIcon,
  default496 as ExpandIcon,
  default497 as ExternalLinkIcon,
  default498 as EyeClosedIcon,
  default500 as EyeIcon,
  default499 as EyeOffIcon,
  default501 as FactoryIcon,
  default502 as FanIcon,
  default503 as FastForwardIcon,
  default504 as FeatherIcon,
  default505 as FenceIcon,
  default506 as FerrisWheelIcon,
  default507 as FileArchiveIcon,
  default1646 as FileAudio2Icon,
  default1645 as FileAudioIcon,
  default1621 as FileAxis3DIcon,
  default1620 as FileAxis3dIcon,
  default1623 as FileBadge2Icon,
  default1622 as FileBadgeIcon,
  default1631 as FileBarChart2Icon,
  default1629 as FileBarChartIcon,
  default508 as FileBoxIcon,
  default1624 as FileBracesCornerIcon,
  default1626 as FileBracesIcon,
  default1630 as FileChartColumnIcon,
  default1628 as FileChartColumnIncreasingIcon,
  default1634 as FileChartLineIcon,
  default1632 as FileChartPieIcon,
  default1641 as FileCheck2Icon,
  default1640 as FileCheckCornerIcon,
  default510 as FileCheckIcon,
  default509 as FileClockIcon,
  default1637 as FileCode2Icon,
  default1636 as FileCodeCornerIcon,
  default511 as FileCodeIcon,
  default1639 as FileCog2Icon,
  default1638 as FileCogIcon,
  default513 as FileDiffIcon,
  default512 as FileDigitIcon,
  default514 as FileDownIcon,
  default1656 as FileEditIcon,
  default1642 as FileExclamationPointIcon,
  default1644 as FileHeadphoneIcon,
  default515 as FileHeartIcon,
  default535 as FileIcon,
  default516 as FileImageIcon,
  default517 as FileInputIcon,
  default1625 as FileJson2Icon,
  default1627 as FileJsonIcon,
  default1652 as FileKey2Icon,
  default1651 as FileKeyIcon,
  default1635 as FileLineChartIcon,
  default1648 as FileLock2Icon,
  default1647 as FileLockIcon,
  default1650 as FileMinus2Icon,
  default1649 as FileMinusCornerIcon,
  default518 as FileMinusIcon,
  default519 as FileMusicIcon,
  default520 as FileOutputIcon,
  default1655 as FilePenIcon,
  default1653 as FilePenLineIcon,
  default1633 as FilePieChartIcon,
  default1657 as FilePlayIcon,
  default1660 as FilePlus2Icon,
  default1659 as FilePlusCornerIcon,
  default521 as FilePlusIcon,
  default1664 as FileQuestionIcon,
  default1663 as FileQuestionMarkIcon,
  default522 as FileScanIcon,
  default1662 as FileSearch2Icon,
  default1661 as FileSearchCornerIcon,
  default523 as FileSearchIcon,
  default1665 as FileSignalIcon,
  default1654 as FileSignatureIcon,
  default524 as FileSlidersIcon,
  default526 as FileSpreadsheetIcon,
  default525 as FileStackIcon,
  default527 as FileSymlinkIcon,
  default528 as FileTerminalIcon,
  default531 as FileTextIcon,
  default1668 as FileType2Icon,
  default1667 as FileTypeCornerIcon,
  default529 as FileTypeIcon,
  default530 as FileUpIcon,
  default533 as FileUserIcon,
  default1670 as FileVideo2Icon,
  default1669 as FileVideoCameraIcon,
  default1658 as FileVideoIcon,
  default1666 as FileVolume2Icon,
  default532 as FileVolumeIcon,
  default1643 as FileWarningIcon,
  default1672 as FileX2Icon,
  default1671 as FileXCornerIcon,
  default534 as FileXIcon,
  default536 as FilesIcon,
  default537 as FilmIcon,
  default1680 as FilterIcon,
  default1678 as FilterXIcon,
  default1465 as FingerprintIcon,
  default1464 as FingerprintPatternIcon,
  default539 as FireExtinguisherIcon,
  default541 as FishIcon,
  default540 as FishOffIcon,
  default538 as FishSymbolIcon,
  default543 as FishingHookIcon,
  default542 as FishingRodIcon,
  default547 as FlagIcon,
  default544 as FlagOffIcon,
  default545 as FlagTriangleLeftIcon,
  default546 as FlagTriangleRightIcon,
  default549 as FlameIcon,
  default548 as FlameKindlingIcon,
  default552 as FlashlightIcon,
  default550 as FlashlightOffIcon,
  default551 as FlaskConicalIcon,
  default553 as FlaskConicalOffIcon,
  default554 as FlaskRoundIcon,
  default555 as FlipHorizontal2Icon,
  default1811 as FlipHorizontalIcon,
  default556 as FlipVertical2Icon,
  default1813 as FlipVerticalIcon,
  default557 as Flower2Icon,
  default558 as FlowerIcon,
  default560 as FocusIcon,
  default559 as FoldHorizontalIcon,
  default561 as FoldVerticalIcon,
  default562 as FolderArchiveIcon,
  default563 as FolderCheckIcon,
  default564 as FolderClockIcon,
  default565 as FolderClosedIcon,
  default566 as FolderCodeIcon,
  default1674 as FolderCog2Icon,
  default1673 as FolderCogIcon,
  default567 as FolderDotIcon,
  default568 as FolderDownIcon,
  default1676 as FolderEditIcon,
  default569 as FolderGit2Icon,
  default570 as FolderGitIcon,
  default572 as FolderHeartIcon,
  default589 as FolderIcon,
  default571 as FolderInputIcon,
  default573 as FolderKanbanIcon,
  default574 as FolderKeyIcon,
  default577 as FolderLockIcon,
  default575 as FolderMinusIcon,
  default576 as FolderOpenDotIcon,
  default578 as FolderOpenIcon,
  default579 as FolderOutputIcon,
  default1675 as FolderPenIcon,
  default580 as FolderPlusIcon,
  default581 as FolderRootIcon,
  default582 as FolderSearch2Icon,
  default583 as FolderSearchIcon,
  default584 as FolderSymlinkIcon,
  default585 as FolderSyncIcon,
  default586 as FolderTreeIcon,
  default588 as FolderUpIcon,
  default587 as FolderXIcon,
  default590 as FoldersIcon,
  default591 as FootprintsIcon,
  default1938 as ForkKnifeCrossedIcon,
  default1940 as ForkKnifeIcon,
  default592 as ForkliftIcon,
  default593 as FormIcon,
  default1759 as FormInputIcon,
  default594 as ForwardIcon,
  default595 as FrameIcon,
  default596 as FrownIcon,
  default597 as FuelIcon,
  default598 as FullscreenIcon,
  default1845 as FunctionSquareIcon,
  default1679 as FunnelIcon,
  default599 as FunnelPlusIcon,
  default1677 as FunnelXIcon,
  default600 as GalleryHorizontalEndIcon,
  default601 as GalleryHorizontalIcon,
  default603 as GalleryThumbnailsIcon,
  default604 as GalleryVerticalEndIcon,
  default602 as GalleryVerticalIcon,
  default605 as Gamepad2Icon,
  default606 as GamepadDirectionalIcon,
  default607 as GamepadIcon,
  default1523 as GanttChartIcon,
  default1815 as GanttChartSquareIcon,
  default1563 as GaugeCircleIcon,
  default608 as GaugeIcon,
  default609 as GavelIcon,
  default610 as GemIcon,
  default611 as GeorgianLariIcon,
  default612 as GhostIcon,
  default613 as GiftIcon,
  default616 as GitBranchIcon,
  default615 as GitBranchMinusIcon,
  default614 as GitBranchPlusIcon,
  default1681 as GitCommitHorizontalIcon,
  default1682 as GitCommitIcon,
  default617 as GitCommitVerticalIcon,
  default618 as GitCompareArrowsIcon,
  default619 as GitCompareIcon,
  default620 as GitForkIcon,
  default621 as GitGraphIcon,
  default622 as GitMergeConflictIcon,
  default624 as GitMergeIcon,
  default623 as GitPullRequestArrowIcon,
  default626 as GitPullRequestClosedIcon,
  default625 as GitPullRequestCreateArrowIcon,
  default627 as GitPullRequestCreateIcon,
  default628 as GitPullRequestDraftIcon,
  default629 as GitPullRequestIcon,
  default630 as GlassWaterIcon,
  default631 as GlassesIcon,
  default1617 as Globe2Icon,
  default635 as GlobeIcon,
  default632 as GlobeLockIcon,
  default633 as GlobeOffIcon,
  default634 as GlobeXIcon,
  default636 as GoalIcon,
  default637 as GpuIcon,
  default1697 as GrabIcon,
  default638 as GraduationCapIcon,
  default639 as GrapeIcon,
  default1684 as Grid2X2CheckIcon,
  default1688 as Grid2X2Icon,
  default1686 as Grid2X2PlusIcon,
  default1690 as Grid2X2XIcon,
  default1683 as Grid2x2CheckIcon,
  default1687 as Grid2x2Icon,
  default1685 as Grid2x2PlusIcon,
  default1689 as Grid2x2XIcon,
  default1693 as Grid3X3Icon,
  default640 as Grid3x2Icon,
  default1691 as Grid3x3Icon,
  default1692 as GridIcon,
  default641 as GripHorizontalIcon,
  default642 as GripIcon,
  default644 as GripVerticalIcon,
  default643 as GroupIcon,
  default645 as GuitarIcon,
  default646 as HamIcon,
  default649 as HamburgerIcon,
  default647 as HammerIcon,
  default648 as HandCoinsIcon,
  default650 as HandFistIcon,
  default1696 as HandGrabIcon,
  default651 as HandHeartIcon,
  default1694 as HandHelpingIcon,
  default654 as HandIcon,
  default652 as HandMetalIcon,
  default653 as HandPlatterIcon,
  default656 as HandbagIcon,
  default655 as HandshakeIcon,
  default657 as HardDriveDownloadIcon,
  default659 as HardDriveIcon,
  default658 as HardDriveUploadIcon,
  default660 as HardHatIcon,
  default661 as HashIcon,
  default663 as HatGlassesIcon,
  default664 as HazeIcon,
  default662 as HdIcon,
  default666 as HdmiPortIcon,
  default665 as Heading1Icon,
  default667 as Heading2Icon,
  default669 as Heading3Icon,
  default668 as Heading4Icon,
  default670 as Heading5Icon,
  default671 as Heading6Icon,
  default672 as HeadingIcon,
  default673 as HeadphoneOffIcon,
  default674 as HeadphonesIcon,
  default675 as HeadsetIcon,
  default676 as HeartCrackIcon,
  default677 as HeartHandshakeIcon,
  default683 as HeartIcon,
  default678 as HeartMinusIcon,
  default679 as HeartOffIcon,
  default680 as HeartPlusIcon,
  default681 as HeartPulseIcon,
  default682 as HeartXIcon,
  default685 as HeaterIcon,
  default684 as HelicopterIcon,
  default1581 as HelpCircleIcon,
  default1695 as HelpingHandIcon,
  default686 as HexagonIcon,
  default687 as HighlighterIcon,
  default688 as HistoryIcon,
  default1699 as HomeIcon,
  default690 as HopIcon,
  default689 as HopOffIcon,
  default691 as HospitalIcon,
  default692 as HotelIcon,
  default693 as HourglassIcon,
  default694 as HouseHeartIcon,
  default1698 as HouseIcon,
  default697 as HousePlugIcon,
  default695 as HousePlusIcon,
  default696 as HouseWifiIcon,
  default1701 as IceCream2Icon,
  default1700 as IceCreamBowlIcon,
  default1702 as IceCreamConeIcon,
  default1703 as IceCreamIcon,
  default699 as IdCardIcon,
  default698 as IdCardLanyardIcon,
  default700 as ImageDownIcon,
  default707 as ImageIcon,
  default701 as ImageMinusIcon,
  default702 as ImageOffIcon,
  default703 as ImagePlayIcon,
  default704 as ImagePlusIcon,
  default705 as ImageUpIcon,
  default706 as ImageUpscaleIcon,
  default709 as ImagesIcon,
  default708 as ImportIcon,
  default710 as InboxIcon,
  default1710 as IndentDecreaseIcon,
  default1712 as IndentIcon,
  default1713 as IndentIncreaseIcon,
  default711 as IndianRupeeIcon,
  default712 as InfinityIcon,
  default713 as InfoIcon,
  default1859 as InspectIcon,
  default715 as InspectionPanelIcon,
  default714 as ItalicIcon,
  default716 as IterationCcwIcon,
  default717 as IterationCwIcon,
  default719 as JapaneseYenIcon,
  default718 as JoystickIcon,
  default721 as KanbanIcon,
  default1832 as KanbanSquareDashedIcon,
  default1849 as KanbanSquareIcon,
  default720 as KayakIcon,
  default724 as KeyIcon,
  default723 as KeyRoundIcon,
  default722 as KeySquareIcon,
  default727 as KeyboardIcon,
  default725 as KeyboardMusicIcon,
  default726 as KeyboardOffIcon,
  default728 as LampCeilingIcon,
  default731 as LampDeskIcon,
  default729 as LampFloorIcon,
  default733 as LampIcon,
  default730 as LampWallDownIcon,
  default732 as LampWallUpIcon,
  default734 as LandPlotIcon,
  default735 as LandmarkIcon,
  default736 as LanguagesIcon,
  default1705 as Laptop2Icon,
  default738 as LaptopIcon,
  default737 as LaptopMinimalCheckIcon,
  default1704 as LaptopMinimalIcon,
  default740 as LassoIcon,
  default739 as LassoSelectIcon,
  default741 as LaughIcon,
  default742 as Layers2Icon,
  default1707 as Layers3Icon,
  default1706 as LayersIcon,
  default743 as LayersMinusIcon,
  default744 as LayersPlusIcon,
  default746 as LayoutDashboardIcon,
  default745 as LayoutGridIcon,
  default1753 as LayoutIcon,
  default747 as LayoutListIcon,
  default748 as LayoutPanelLeftIcon,
  default749 as LayoutPanelTopIcon,
  default750 as LayoutTemplateIcon,
  default751 as LeafIcon,
  default753 as LeafyGreenIcon,
  default752 as LecternIcon,
  default754 as LensConcaveIcon,
  default756 as LensConvexIcon,
  default1910 as LetterTextIcon,
  default755 as LibraryBigIcon,
  default757 as LibraryIcon,
  default1851 as LibrarySquareIcon,
  default758 as LifeBuoyIcon,
  default759 as LigatureIcon,
  default761 as LightbulbIcon,
  default760 as LightbulbOffIcon,
  default1521 as LineChartIcon,
  default762 as LineDotRightHorizontalIcon,
  default763 as LineSquiggleIcon,
  default764 as LineStyleIcon,
  default766 as Link2Icon,
  default765 as Link2OffIcon,
  default767 as LinkIcon,
  default768 as ListCheckIcon,
  default771 as ListChecksIcon,
  default769 as ListChevronsDownUpIcon,
  default770 as ListChevronsUpDownIcon,
  default772 as ListCollapseIcon,
  default773 as ListEndIcon,
  default775 as ListFilterIcon,
  default774 as ListFilterPlusIcon,
  default786 as ListIcon,
  default1708 as ListIndentDecreaseIcon,
  default1711 as ListIndentIncreaseIcon,
  default776 as ListMinusIcon,
  default777 as ListMusicIcon,
  default778 as ListOrderedIcon,
  default779 as ListPlusIcon,
  default780 as ListRestartIcon,
  default781 as ListStartIcon,
  default783 as ListTodoIcon,
  default782 as ListTreeIcon,
  default784 as ListVideoIcon,
  default785 as ListXIcon,
  default1717 as Loader2Icon,
  default1716 as LoaderCircleIcon,
  default788 as LoaderIcon,
  default787 as LoaderPinwheelIcon,
  default789 as LocateFixedIcon,
  default791 as LocateIcon,
  default790 as LocateOffIcon,
  default1723 as LocationEditIcon,
  default793 as LockIcon,
  default792 as LockKeyholeIcon,
  default1718 as LockKeyholeOpenIcon,
  default1714 as LockOpenIcon,
  default794 as LogInIcon,
  default795 as LogOutIcon,
  default797 as LogsIcon,
  default796 as LollipopIcon,
  default798 as LuggageIcon,
  default1853 as MSquareIcon,
  default799 as MagnetIcon,
  default800 as MailCheckIcon,
  default808 as MailIcon,
  default801 as MailMinusIcon,
  default805 as MailOpenIcon,
  default802 as MailPlusIcon,
  default1721 as MailQuestionIcon,
  default1720 as MailQuestionMarkIcon,
  default803 as MailSearchIcon,
  default804 as MailWarningIcon,
  default806 as MailXIcon,
  default807 as MailboxIcon,
  default809 as MailsIcon,
  default825 as MapIcon,
  default811 as MapMinusIcon,
  default812 as MapPinCheckIcon,
  default810 as MapPinCheckInsideIcon,
  default813 as MapPinHouseIcon,
  default822 as MapPinIcon,
  default814 as MapPinMinusIcon,
  default815 as MapPinMinusInsideIcon,
  default816 as MapPinOffIcon,
  default1722 as MapPinPenIcon,
  default818 as MapPinPlusIcon,
  default817 as MapPinPlusInsideIcon,
  default819 as MapPinSearchIcon,
  default821 as MapPinXIcon,
  default820 as MapPinXInsideIcon,
  default823 as MapPinnedIcon,
  default824 as MapPlusIcon,
  default827 as MarsIcon,
  default826 as MarsStrokeIcon,
  default828 as MartiniIcon,
  default829 as Maximize2Icon,
  default831 as MaximizeIcon,
  default832 as MedalIcon,
  default833 as MegaphoneIcon,
  default830 as MegaphoneOffIcon,
  default834 as MehIcon,
  default835 as MemoryStickIcon,
  default837 as MenuIcon,
  default1855 as MenuSquareIcon,
  default836 as MergeIcon,
  default838 as MessageCircleCheckIcon,
  default839 as MessageCircleCodeIcon,
  default840 as MessageCircleDashedIcon,
  default841 as MessageCircleHeartIcon,
  default849 as MessageCircleIcon,
  default842 as MessageCircleMoreIcon,
  default843 as MessageCircleOffIcon,
  default844 as MessageCirclePlusIcon,
  default1725 as MessageCircleQuestionIcon,
  default1724 as MessageCircleQuestionMarkIcon,
  default845 as MessageCircleReplyIcon,
  default846 as MessageCircleWarningIcon,
  default847 as MessageCircleXIcon,
  default848 as MessageSquareCheckIcon,
  default850 as MessageSquareCodeIcon,
  default852 as MessageSquareDashedIcon,
  default851 as MessageSquareDiffIcon,
  default853 as MessageSquareDotIcon,
  default854 as MessageSquareHeartIcon,
  default865 as MessageSquareIcon,
  default855 as MessageSquareLockIcon,
  default856 as MessageSquareMoreIcon,
  default857 as MessageSquareOffIcon,
  default858 as MessageSquarePlusIcon,
  default859 as MessageSquareQuoteIcon,
  default860 as MessageSquareReplyIcon,
  default861 as MessageSquareShareIcon,
  default862 as MessageSquareTextIcon,
  default863 as MessageSquareWarningIcon,
  default864 as MessageSquareXIcon,
  default866 as MessagesSquareIcon,
  default867 as MetronomeIcon,
  default1727 as Mic2Icon,
  default869 as MicIcon,
  default868 as MicOffIcon,
  default1726 as MicVocalIcon,
  default870 as MicrochipIcon,
  default871 as MicroscopeIcon,
  default872 as MicrowaveIcon,
  default873 as MilestoneIcon,
  default875 as MilkIcon,
  default874 as MilkOffIcon,
  default876 as Minimize2Icon,
  default877 as MinimizeIcon,
  default1565 as MinusCircleIcon,
  default878 as MinusIcon,
  default1857 as MinusSquareIcon,
  default879 as MirrorRectangularIcon,
  default880 as MirrorRoundIcon,
  default881 as MonitorCheckIcon,
  default882 as MonitorCloudIcon,
  default883 as MonitorCogIcon,
  default884 as MonitorDotIcon,
  default885 as MonitorDownIcon,
  default894 as MonitorIcon,
  default886 as MonitorOffIcon,
  default889 as MonitorPauseIcon,
  default887 as MonitorPlayIcon,
  default888 as MonitorSmartphoneIcon,
  default890 as MonitorSpeakerIcon,
  default892 as MonitorStopIcon,
  default891 as MonitorUpIcon,
  default893 as MonitorXIcon,
  default896 as MoonIcon,
  default895 as MoonStarIcon,
  default1619 as MoreHorizontalIcon,
  default1615 as MoreVerticalIcon,
  default897 as MotorbikeIcon,
  default898 as MountainIcon,
  default899 as MountainSnowIcon,
  default909 as MouseIcon,
  default900 as MouseLeftIcon,
  default901 as MouseOffIcon,
  default903 as MousePointer2Icon,
  default902 as MousePointer2OffIcon,
  default904 as MousePointerBanIcon,
  default905 as MousePointerClickIcon,
  default906 as MousePointerIcon,
  default1834 as MousePointerSquareDashedIcon,
  default907 as MouseRightIcon,
  default1729 as Move3DIcon,
  default1728 as Move3dIcon,
  default908 as MoveDiagonal2Icon,
  default910 as MoveDiagonalIcon,
  default913 as MoveDownIcon,
  default911 as MoveDownLeftIcon,
  default912 as MoveDownRightIcon,
  default914 as MoveHorizontalIcon,
  default921 as MoveIcon,
  default915 as MoveLeftIcon,
  default916 as MoveRightIcon,
  default919 as MoveUpIcon,
  default917 as MoveUpLeftIcon,
  default918 as MoveUpRightIcon,
  default920 as MoveVerticalIcon,
  default922 as Music2Icon,
  default923 as Music3Icon,
  default924 as Music4Icon,
  default927 as MusicIcon,
  default925 as Navigation2Icon,
  default926 as Navigation2OffIcon,
  default929 as NavigationIcon,
  default928 as NavigationOffIcon,
  default931 as NetworkIcon,
  default930 as NewspaperIcon,
  default932 as NfcIcon,
  default933 as NonBinaryIcon,
  default937 as NotebookIcon,
  default934 as NotebookPenIcon,
  default935 as NotebookTabsIcon,
  default936 as NotebookTextIcon,
  default938 as NotepadTextDashedIcon,
  default939 as NotepadTextIcon,
  default942 as NutIcon,
  default940 as NutOffIcon,
  default1730 as OctagonAlertIcon,
  default1060 as OctagonIcon,
  default941 as OctagonMinusIcon,
  default1732 as OctagonPauseIcon,
  default1734 as OctagonXIcon,
  default944 as OmegaIcon,
  default943 as OptionIcon,
  default946 as OrbitIcon,
  default945 as OrigamiIcon,
  default1709 as OutdentIcon,
  default947 as Package2Icon,
  default948 as PackageCheckIcon,
  default956 as PackageIcon,
  default949 as PackageMinusIcon,
  default950 as PackageOpenIcon,
  default951 as PackagePlusIcon,
  default952 as PackageSearchIcon,
  default954 as PackageXIcon,
  default953 as PaintBucketIcon,
  default955 as PaintRollerIcon,
  default1741 as Paintbrush2Icon,
  default957 as PaintbrushIcon,
  default1740 as PaintbrushVerticalIcon,
  default958 as PaletteIcon,
  default1916 as PalmtreeIcon,
  default959 as PandaIcon,
  default962 as PanelBottomCloseIcon,
  default1736 as PanelBottomDashedIcon,
  default961 as PanelBottomIcon,
  default1737 as PanelBottomInactiveIcon,
  default960 as PanelBottomOpenIcon,
  default1742 as PanelLeftCloseIcon,
  default1738 as PanelLeftDashedIcon,
  default1744 as PanelLeftIcon,
  default1739 as PanelLeftInactiveIcon,
  default1746 as PanelLeftOpenIcon,
  default963 as PanelLeftRightDashedIcon,
  default964 as PanelRightCloseIcon,
  default1748 as PanelRightDashedIcon,
  default966 as PanelRightIcon,
  default1749 as PanelRightInactiveIcon,
  default965 as PanelRightOpenIcon,
  default967 as PanelTopBottomDashedIcon,
  default968 as PanelTopCloseIcon,
  default1750 as PanelTopDashedIcon,
  default970 as PanelTopIcon,
  default1751 as PanelTopInactiveIcon,
  default969 as PanelTopOpenIcon,
  default972 as PanelsLeftBottomIcon,
  default1609 as PanelsLeftRightIcon,
  default971 as PanelsRightBottomIcon,
  default1767 as PanelsTopBottomIcon,
  default1752 as PanelsTopLeftIcon,
  default973 as PaperclipIcon,
  default974 as ParenthesesIcon,
  default1571 as ParkingCircleIcon,
  default1567 as ParkingCircleOffIcon,
  default975 as ParkingMeterIcon,
  default1861 as ParkingSquareIcon,
  default1869 as ParkingSquareOffIcon,
  default976 as PartyPopperIcon,
  default1569 as PauseCircleIcon,
  default978 as PauseIcon,
  default1733 as PauseOctagonIcon,
  default977 as PawPrintIcon,
  default979 as PcCaseIcon,
  default1865 as PenBoxIcon,
  default1754 as PenIcon,
  default1756 as PenLineIcon,
  default980 as PenOffIcon,
  default1867 as PenSquareIcon,
  default982 as PenToolIcon,
  default984 as PencilIcon,
  default981 as PencilLineIcon,
  default986 as PencilOffIcon,
  default983 as PencilRulerIcon,
  default985 as PentagonIcon,
  default1573 as PercentCircleIcon,
  default1613 as PercentDiamondIcon,
  default987 as PercentIcon,
  default1863 as PercentSquareIcon,
  default990 as PersonStandingIcon,
  default988 as PhilippinePesoIcon,
  default989 as PhoneCallIcon,
  default991 as PhoneForwardedIcon,
  default996 as PhoneIcon,
  default992 as PhoneIncomingIcon,
  default993 as PhoneMissedIcon,
  default994 as PhoneOffIcon,
  default995 as PhoneOutgoingIcon,
  default997 as PiIcon,
  default1871 as PiSquareIcon,
  default998 as PianoIcon,
  default999 as PickaxeIcon,
  default1000 as PictureInPicture2Icon,
  default1001 as PictureInPictureIcon,
  default1527 as PieChartIcon,
  default1002 as PiggyBankIcon,
  default1005 as PilcrowIcon,
  default1003 as PilcrowLeftIcon,
  default1004 as PilcrowRightIcon,
  default1873 as PilcrowSquareIcon,
  default1006 as PillBottleIcon,
  default1007 as PillIcon,
  default1008 as PinIcon,
  default1012 as PinOffIcon,
  default1009 as PipetteIcon,
  default1010 as PizzaIcon,
  default1014 as PlaneIcon,
  default1011 as PlaneLandingIcon,
  default1013 as PlaneTakeoffIcon,
  default1579 as PlayCircleIcon,
  default1017 as PlayIcon,
  default1875 as PlaySquareIcon,
  default1015 as Plug2Icon,
  default1016 as PlugIcon,
  default1761 as PlugZap2Icon,
  default1760 as PlugZapIcon,
  default1575 as PlusCircleIcon,
  default1018 as PlusIcon,
  default1877 as PlusSquareIcon,
  default1019 as PocketKnifeIcon,
  default1020 as PodcastIcon,
  default1022 as PointerIcon,
  default1021 as PointerOffIcon,
  default1023 as PopcornIcon,
  default1024 as PopsicleIcon,
  default1025 as PoundSterlingIcon,
  default1577 as PowerCircleIcon,
  default1031 as PowerIcon,
  default1026 as PowerOffIcon,
  default1883 as PowerSquareIcon,
  default1027 as PresentationIcon,
  default1028 as PrinterCheckIcon,
  default1033 as PrinterIcon,
  default1029 as PrinterXIcon,
  default1030 as ProjectorIcon,
  default1032 as ProportionsIcon,
  default1037 as PuzzleIcon,
  default1034 as PyramidIcon,
  default1035 as QrCodeIcon,
  default1036 as QuoteIcon,
  default1038 as RabbitIcon,
  default1039 as RadarIcon,
  default1042 as RadiationIcon,
  default1040 as RadicalIcon,
  default1045 as RadioIcon,
  default1041 as RadioOffIcon,
  default1043 as RadioReceiverIcon,
  default1044 as RadioTowerIcon,
  default1046 as RadiusIcon,
  default1047 as RainbowIcon,
  default1048 as RatIcon,
  default1049 as RatioIcon,
  default1050 as ReceiptCentIcon,
  default1051 as ReceiptEuroIcon,
  default1059 as ReceiptIcon,
  default1052 as ReceiptIndianRupeeIcon,
  default1053 as ReceiptJapaneseYenIcon,
  default1054 as ReceiptPoundSterlingIcon,
  default1055 as ReceiptRussianRubleIcon,
  default1056 as ReceiptSwissFrancIcon,
  default1057 as ReceiptTextIcon,
  default1058 as ReceiptTurkishLiraIcon,
  default1062 as RectangleCircleIcon,
  default1758 as RectangleEllipsisIcon,
  default1061 as RectangleGogglesIcon,
  default1063 as RectangleHorizontalIcon,
  default1064 as RectangleVerticalIcon,
  default1065 as RecycleIcon,
  default1066 as Redo2Icon,
  default1067 as RedoDotIcon,
  default1068 as RedoIcon,
  default1070 as RefreshCcwDotIcon,
  default1069 as RefreshCcwIcon,
  default1072 as RefreshCwIcon,
  default1071 as RefreshCwOffIcon,
  default1073 as RefrigeratorIcon,
  default1074 as RegexIcon,
  default1075 as RemoveFormattingIcon,
  default1076 as Repeat1Icon,
  default1077 as Repeat2Icon,
  default1078 as RepeatIcon,
  default1079 as ReplaceAllIcon,
  default1081 as ReplaceIcon,
  default1080 as ReplyAllIcon,
  default1082 as ReplyIcon,
  default1083 as RewindIcon,
  default1084 as RibbonIcon,
  default1085 as RoadIcon,
  default1086 as RocketIcon,
  default1087 as RockingChairIcon,
  default1088 as RollerCoasterIcon,
  default1089 as RoseIcon,
  default1763 as Rotate3DIcon,
  default1762 as Rotate3dIcon,
  default1092 as RotateCcwIcon,
  default1090 as RotateCcwKeyIcon,
  default1091 as RotateCcwSquareIcon,
  default1093 as RotateCwIcon,
  default1094 as RotateCwSquareIcon,
  default1096 as RouteIcon,
  default1095 as RouteOffIcon,
  default1097 as RouterIcon,
  default1764 as Rows2Icon,
  default1766 as Rows3Icon,
  default1098 as Rows4Icon,
  default1765 as RowsIcon,
  default1101 as RssIcon,
  default1099 as RulerDimensionLineIcon,
  default1100 as RulerIcon,
  default1102 as RussianRubleIcon,
  default1103 as SailboatIcon,
  default1104 as SaladIcon,
  default1105 as SandwichIcon,
  default1106 as SatelliteDishIcon,
  default1107 as SatelliteIcon,
  default1108 as SaudiRiyalIcon,
  default1109 as SaveAllIcon,
  default1111 as SaveIcon,
  default1110 as SaveOffIcon,
  default1769 as Scale3DIcon,
  default1768 as Scale3dIcon,
  default1112 as ScaleIcon,
  default1113 as ScalingIcon,
  default1114 as ScanBarcodeIcon,
  default1115 as ScanEyeIcon,
  default1116 as ScanFaceIcon,
  default1117 as ScanHeartIcon,
  default1122 as ScanIcon,
  default1118 as ScanLineIcon,
  default1119 as ScanQrCodeIcon,
  default1120 as ScanSearchIcon,
  default1121 as ScanTextIcon,
  default1529 as ScatterChartIcon,
  default1922 as School2Icon,
  default1123 as SchoolIcon,
  default1125 as ScissorsIcon,
  default1124 as ScissorsLineDashedIcon,
  default1807 as ScissorsSquareDashedBottomIcon,
  default1879 as ScissorsSquareIcon,
  default1126 as ScooterIcon,
  default1129 as ScreenShareIcon,
  default1127 as ScreenShareOffIcon,
  default1130 as ScrollIcon,
  default1128 as ScrollTextIcon,
  default1131 as SearchAlertIcon,
  default1132 as SearchCheckIcon,
  default1133 as SearchCodeIcon,
  default1136 as SearchIcon,
  default1134 as SearchSlashIcon,
  default1135 as SearchXIcon,
  default1138 as SectionIcon,
  default1771 as SendHorizonalIcon,
  default1770 as SendHorizontalIcon,
  default1139 as SendIcon,
  default1137 as SendToBackIcon,
  default1140 as SeparatorHorizontalIcon,
  default1142 as SeparatorVerticalIcon,
  default1141 as ServerCogIcon,
  default1143 as ServerCrashIcon,
  default1145 as ServerIcon,
  default1144 as ServerOffIcon,
  default1146 as Settings2Icon,
  default1147 as SettingsIcon,
  default1149 as ShapesIcon,
  default1150 as Share2Icon,
  default1148 as ShareIcon,
  default1151 as SheetIcon,
  default1152 as ShellIcon,
  default1153 as ShelvingUnitIcon,
  default1154 as ShieldAlertIcon,
  default1155 as ShieldBanIcon,
  default1156 as ShieldCheckIcon,
  default1775 as ShieldCloseIcon,
  default1157 as ShieldCogCornerIcon,
  default1158 as ShieldCogIcon,
  default1159 as ShieldEllipsisIcon,
  default1160 as ShieldHalfIcon,
  default1165 as ShieldIcon,
  default1161 as ShieldMinusIcon,
  default1162 as ShieldOffIcon,
  default1163 as ShieldPlusIcon,
  default1773 as ShieldQuestionIcon,
  default1772 as ShieldQuestionMarkIcon,
  default1164 as ShieldUserIcon,
  default1774 as ShieldXIcon,
  default1167 as ShipIcon,
  default1166 as ShipWheelIcon,
  default1168 as ShirtIcon,
  default1169 as ShoppingBagIcon,
  default1170 as ShoppingBasketIcon,
  default1171 as ShoppingCartIcon,
  default1173 as ShovelIcon,
  default1172 as ShowerHeadIcon,
  default1174 as ShredderIcon,
  default1175 as ShrimpIcon,
  default1176 as ShrinkIcon,
  default1177 as ShrubIcon,
  default1178 as ShuffleIcon,
  default1743 as SidebarCloseIcon,
  default1745 as SidebarIcon,
  default1747 as SidebarOpenIcon,
  default1179 as SigmaIcon,
  default1881 as SigmaSquareIcon,
  default1180 as SignalHighIcon,
  default1187 as SignalIcon,
  default1181 as SignalLowIcon,
  default1182 as SignalMediumIcon,
  default1183 as SignalZeroIcon,
  default1184 as SignatureIcon,
  default1185 as SignpostBigIcon,
  default1186 as SignpostIcon,
  default1188 as SirenIcon,
  default1189 as SkipBackIcon,
  default1190 as SkipForwardIcon,
  default1191 as SkullIcon,
  default1192 as SlashIcon,
  default1885 as SlashSquareIcon,
  default1193 as SliceIcon,
  default1194 as SlidersHorizontalIcon,
  default1777 as SlidersIcon,
  default1776 as SlidersVerticalIcon,
  default1195 as SmartphoneChargingIcon,
  default1197 as SmartphoneIcon,
  default1196 as SmartphoneNfcIcon,
  default1199 as SmileIcon,
  default1198 as SmilePlusIcon,
  default1200 as SnailIcon,
  default1203 as SnowflakeIcon,
  default1201 as SoapDispenserDropletIcon,
  default1202 as SofaIcon,
  default1204 as SolarPanelIcon,
  default1485 as SortAscIcon,
  default1479 as SortDescIcon,
  default1205 as SoupIcon,
  default1208 as SpaceIcon,
  default1206 as SpadeIcon,
  default1207 as SparkleIcon,
  default1778 as SparklesIcon,
  default1209 as SpeakerIcon,
  default1210 as SpeechIcon,
  default1211 as SpellCheck2Icon,
  default1212 as SpellCheckIcon,
  default1214 as SplineIcon,
  default1213 as SplinePointerIcon,
  default1215 as SplitIcon,
  default1887 as SplitSquareHorizontalIcon,
  default1889 as SplitSquareVerticalIcon,
  default1216 as SpoolIcon,
  default1217 as SportShoeIcon,
  default1218 as SpotlightIcon,
  default1219 as SprayCanIcon,
  default1220 as SproutIcon,
  default1780 as SquareActivityIcon,
  default1786 as SquareArrowDownIcon,
  default1782 as SquareArrowDownLeftIcon,
  default1784 as SquareArrowDownRightIcon,
  default1788 as SquareArrowLeftIcon,
  default1792 as SquareArrowOutDownLeftIcon,
  default1790 as SquareArrowOutDownRightIcon,
  default1794 as SquareArrowOutUpLeftIcon,
  default1796 as SquareArrowOutUpRightIcon,
  default1221 as SquareArrowRightEnterIcon,
  default1222 as SquareArrowRightExitIcon,
  default1798 as SquareArrowRightIcon,
  default1804 as SquareArrowUpIcon,
  default1800 as SquareArrowUpLeftIcon,
  default1802 as SquareArrowUpRightIcon,
  default1808 as SquareAsteriskIcon,
  default1806 as SquareBottomDashedScissorsIcon,
  default1810 as SquareCenterlineDashedHorizontalIcon,
  default1812 as SquareCenterlineDashedVerticalIcon,
  default1814 as SquareChartGanttIcon,
  default1819 as SquareCheckBigIcon,
  default1821 as SquareCheckIcon,
  default1817 as SquareChevronDownIcon,
  default1823 as SquareChevronLeftIcon,
  default1829 as SquareChevronRightIcon,
  default1825 as SquareChevronUpIcon,
  default1827 as SquareCodeIcon,
  default1223 as SquareDashedBottomCodeIcon,
  default1224 as SquareDashedBottomIcon,
  default1838 as SquareDashedIcon,
  default1831 as SquareDashedKanbanIcon,
  default1833 as SquareDashedMousePointerIcon,
  default1835 as SquareDashedTextIcon,
  default1225 as SquareDashedTopSolidIcon,
  default1840 as SquareDivideIcon,
  default1842 as SquareDotIcon,
  default1846 as SquareEqualIcon,
  default1844 as SquareFunctionIcon,
  default1816 as SquareGanttChartIcon,
  default1233 as SquareIcon,
  default1848 as SquareKanbanIcon,
  default1850 as SquareLibraryIcon,
  default1852 as SquareMIcon,
  default1854 as SquareMenuIcon,
  default1856 as SquareMinusIcon,
  default1858 as SquareMousePointerIcon,
  default1860 as SquareParkingIcon,
  default1868 as SquareParkingOffIcon,
  default1226 as SquarePauseIcon,
  default1864 as SquarePenIcon,
  default1862 as SquarePercentIcon,
  default1870 as SquarePiIcon,
  default1872 as SquarePilcrowIcon,
  default1874 as SquarePlayIcon,
  default1876 as SquarePlusIcon,
  default1882 as SquarePowerIcon,
  default1227 as SquareRadicalIcon,
  default1228 as SquareRoundCornerIcon,
  default1878 as SquareScissorsIcon,
  default1880 as SquareSigmaIcon,
  default1884 as SquareSlashIcon,
  default1886 as SquareSplitHorizontalIcon,
  default1888 as SquareSplitVerticalIcon,
  default1229 as SquareSquareIcon,
  default1230 as SquareStackIcon,
  default1231 as SquareStarIcon,
  default1232 as SquareStopIcon,
  default1890 as SquareTerminalIcon,
  default1894 as SquareUserIcon,
  default1892 as SquareUserRoundIcon,
  default1896 as SquareXIcon,
  default1234 as SquaresExcludeIcon,
  default1235 as SquaresIntersectIcon,
  default1236 as SquaresSubtractIcon,
  default1237 as SquaresUniteIcon,
  default1238 as SquircleDashedIcon,
  default1239 as SquircleIcon,
  default1240 as SquirrelIcon,
  default1241 as StampIcon,
  default1242 as StarHalfIcon,
  default1245 as StarIcon,
  default1243 as StarOffIcon,
  default1779 as StarsIcon,
  default1244 as StepBackIcon,
  default1246 as StepForwardIcon,
  default1248 as StethoscopeIcon,
  default1247 as StickerIcon,
  default1249 as StickyNoteIcon,
  default1252 as StoneIcon,
  default1586 as StopCircleIcon,
  default1250 as StoreIcon,
  default1251 as StretchHorizontalIcon,
  default1253 as StretchVerticalIcon,
  default1254 as StrikethroughIcon,
  default1255 as SubscriptIcon,
  default1503 as SubtitlesIcon,
  default1256 as SunDimIcon,
  default1260 as SunIcon,
  default1257 as SunMediumIcon,
  default1258 as SunMoonIcon,
  default1259 as SunSnowIcon,
  default1261 as SunriseIcon,
  default1262 as SunsetIcon,
  default1263 as SuperscriptIcon,
  default1264 as SwatchBookIcon,
  default1265 as SwissFrancIcon,
  default1266 as SwitchCameraIcon,
  default1267 as SwordIcon,
  default1268 as SwordsIcon,
  default1269 as SyringeIcon,
  default1271 as Table2Icon,
  default1270 as TableCellsMergeIcon,
  default1272 as TableCellsSplitIcon,
  default1273 as TableColumnsSplitIcon,
  default1607 as TableConfigIcon,
  default1277 as TableIcon,
  default1274 as TableOfContentsIcon,
  default1275 as TablePropertiesIcon,
  default1276 as TableRowsSplitIcon,
  default1280 as TabletIcon,
  default1278 as TabletSmartphoneIcon,
  default1279 as TabletsIcon,
  default1281 as TagIcon,
  default1283 as TagsIcon,
  default1282 as Tally1Icon,
  default1284 as Tally2Icon,
  default1285 as Tally3Icon,
  default1286 as Tally4Icon,
  default1287 as Tally5Icon,
  default1288 as TangentIcon,
  default1289 as TargetIcon,
  default1290 as TelescopeIcon,
  default1294 as TentIcon,
  default1291 as TentTreeIcon,
  default1292 as TerminalIcon,
  default1891 as TerminalSquareIcon,
  default1899 as TestTube2Icon,
  default1898 as TestTubeDiagonalIcon,
  default1293 as TestTubeIcon,
  default1295 as TestTubesIcon,
  default1900 as TextAlignCenterIcon,
  default1902 as TextAlignEndIcon,
  default1904 as TextAlignJustifyIcon,
  default1906 as TextAlignStartIcon,
  default1297 as TextCursorIcon,
  default1296 as TextCursorInputIcon,
  default1907 as TextIcon,
  default1909 as TextInitialIcon,
  default1298 as TextQuoteIcon,
  default1299 as TextSearchIcon,
  default1837 as TextSelectIcon,
  default1836 as TextSelectionIcon,
  default1911 as TextWrapIcon,
  default1300 as TheaterIcon,
  default1303 as ThermometerIcon,
  default1301 as ThermometerSnowflakeIcon,
  default1302 as ThermometerSunIcon,
  default1304 as ThumbsDownIcon,
  default1305 as ThumbsUpIcon,
  default1306 as TicketCheckIcon,
  default1311 as TicketIcon,
  default1307 as TicketMinusIcon,
  default1308 as TicketPercentIcon,
  default1309 as TicketPlusIcon,
  default1310 as TicketSlashIcon,
  default1312 as TicketXIcon,
  default1314 as TicketsIcon,
  default1313 as TicketsPlaneIcon,
  default1315 as TimelineIcon,
  default1317 as TimerIcon,
  default1316 as TimerOffIcon,
  default1318 as TimerResetIcon,
  default1319 as ToggleLeftIcon,
  default1320 as ToggleRightIcon,
  default1321 as ToiletIcon,
  default1322 as ToolCaseIcon,
  default1323 as ToolboxIcon,
  default1324 as TornadoIcon,
  default1326 as TorusIcon,
  default1327 as TouchpadIcon,
  default1325 as TouchpadOffIcon,
  default1328 as TowelRackIcon,
  default1329 as TowerControlIcon,
  default1330 as ToyBrickIcon,
  default1331 as TractorIcon,
  default1333 as TrafficConeIcon,
  default1334 as TrainFrontIcon,
  default1332 as TrainFrontTunnelIcon,
  default1914 as TrainIcon,
  default1335 as TrainTrackIcon,
  default1913 as TramFrontIcon,
  default1336 as TransgenderIcon,
  default1337 as Trash2Icon,
  default1338 as TrashIcon,
  default1339 as TreeDeciduousIcon,
  default1915 as TreePalmIcon,
  default1340 as TreePineIcon,
  default1341 as TreesIcon,
  default1342 as TrendingDownIcon,
  default1343 as TrendingUpDownIcon,
  default1344 as TrendingUpIcon,
  default1917 as TriangleAlertIcon,
  default1345 as TriangleDashedIcon,
  default1347 as TriangleIcon,
  default1346 as TriangleRightIcon,
  default1348 as TrophyIcon,
  default1349 as TruckElectricIcon,
  default1351 as TruckIcon,
  default1350 as TurkishLiraIcon,
  default1352 as TurntableIcon,
  default1353 as TurtleIcon,
  default1920 as Tv2Icon,
  default1355 as TvIcon,
  default1919 as TvMinimalIcon,
  default1354 as TvMinimalPlayIcon,
  default1357 as TypeIcon,
  default1356 as TypeOutlineIcon,
  default1359 as UmbrellaIcon,
  default1358 as UmbrellaOffIcon,
  default1360 as UnderlineIcon,
  default1361 as Undo2Icon,
  default1362 as UndoDotIcon,
  default1363 as UndoIcon,
  default1364 as UnfoldHorizontalIcon,
  default1365 as UnfoldVerticalIcon,
  default1367 as UngroupIcon,
  default1921 as UniversityIcon,
  default1366 as Unlink2Icon,
  default1368 as UnlinkIcon,
  default1715 as UnlockIcon,
  default1719 as UnlockKeyholeIcon,
  default1369 as UnplugIcon,
  default1600 as UploadCloudIcon,
  default1370 as UploadIcon,
  default1373 as UsbIcon,
  default1934 as User2Icon,
  default1924 as UserCheck2Icon,
  default1371 as UserCheckIcon,
  default1588 as UserCircle2Icon,
  default1592 as UserCircleIcon,
  default1926 as UserCog2Icon,
  default1372 as UserCogIcon,
  default1385 as UserIcon,
  default1375 as UserKeyIcon,
  default1374 as UserLockIcon,
  default1928 as UserMinus2Icon,
  default1376 as UserMinusIcon,
  default1377 as UserPenIcon,
  default1930 as UserPlus2Icon,
  default1378 as UserPlusIcon,
  default1923 as UserRoundCheckIcon,
  default1925 as UserRoundCogIcon,
  default1933 as UserRoundIcon,
  default1379 as UserRoundKeyIcon,
  default1927 as UserRoundMinusIcon,
  default1380 as UserRoundPenIcon,
  default1929 as UserRoundPlusIcon,
  default1381 as UserRoundSearchIcon,
  default1931 as UserRoundXIcon,
  default1382 as UserSearchIcon,
  default1893 as UserSquare2Icon,
  default1895 as UserSquareIcon,
  default1383 as UserStarIcon,
  default1932 as UserX2Icon,
  default1384 as UserXIcon,
  default1936 as Users2Icon,
  default1386 as UsersIcon,
  default1935 as UsersRoundIcon,
  default1937 as UtensilsCrossedIcon,
  default1939 as UtensilsIcon,
  default1388 as UtilityPoleIcon,
  default1387 as VanIcon,
  default1389 as VariableIcon,
  default1390 as VaultIcon,
  default1391 as VectorSquareIcon,
  default1392 as VeganIcon,
  default1393 as VenetianMaskIcon,
  default1395 as VenusAndMarsIcon,
  default1394 as VenusIcon,
  default1491 as VerifiedIcon,
  default1397 as VibrateIcon,
  default1396 as VibrateOffIcon,
  default1399 as VideoIcon,
  default1398 as VideoOffIcon,
  default1400 as VideotapeIcon,
  default1401 as ViewIcon,
  default1403 as VoicemailIcon,
  default1402 as VolleyballIcon,
  default1404 as Volume1Icon,
  default1405 as Volume2Icon,
  default1408 as VolumeIcon,
  default1406 as VolumeOffIcon,
  default1407 as VolumeXIcon,
  default1409 as VoteIcon,
  default1942 as Wallet2Icon,
  default1410 as WalletCardsIcon,
  default1411 as WalletIcon,
  default1941 as WalletMinimalIcon,
  default1412 as WallpaperIcon,
  default1944 as Wand2Icon,
  default1413 as WandIcon,
  default1943 as WandSparklesIcon,
  default1414 as WarehouseIcon,
  default1415 as WashingMachineIcon,
  default1416 as WatchIcon,
  default1417 as WavesArrowDownIcon,
  default1418 as WavesArrowUpIcon,
  default1421 as WavesIcon,
  default1419 as WavesLadderIcon,
  default1420 as WaypointsIcon,
  default1422 as WebcamIcon,
  default1424 as WebhookIcon,
  default1423 as WebhookOffIcon,
  default1426 as WeightIcon,
  default1425 as WeightTildeIcon,
  default1427 as WheatIcon,
  default1428 as WheatOffIcon,
  default1429 as WholeWordIcon,
  default1430 as WifiCogIcon,
  default1431 as WifiHighIcon,
  default1437 as WifiIcon,
  default1432 as WifiLowIcon,
  default1433 as WifiOffIcon,
  default1434 as WifiPenIcon,
  default1435 as WifiSyncIcon,
  default1436 as WifiZeroIcon,
  default1438 as WindArrowDownIcon,
  default1439 as WindIcon,
  default1441 as WineIcon,
  default1440 as WineOffIcon,
  default1442 as WorkflowIcon,
  default1444 as WormIcon,
  default1912 as WrapTextIcon,
  default1445 as WrenchIcon,
  default1590 as XCircleIcon,
  default1443 as XIcon,
  default1446 as XLineTopIcon,
  default1735 as XOctagonIcon,
  default1897 as XSquareIcon,
  default1447 as ZapIcon,
  default1448 as ZapOffIcon,
  default1449 as ZodiacAquariusIcon,
  default1450 as ZodiacAriesIcon,
  default1451 as ZodiacCancerIcon,
  default1452 as ZodiacCapricornIcon,
  default1453 as ZodiacGeminiIcon,
  default1454 as ZodiacLeoIcon,
  default1455 as ZodiacLibraIcon,
  default1456 as ZodiacOphiuchusIcon,
  default1458 as ZodiacPiscesIcon,
  default1457 as ZodiacSagittariusIcon,
  default1459 as ZodiacScorpioIcon,
  default1461 as ZodiacTaurusIcon,
  default1462 as ZodiacVirgoIcon,
  default1460 as ZoomInIcon,
  default1463 as ZoomOutIcon
};
