// @tanstack/solid-router-devtools/dist/esm/TanStackRouterDevtools.js
import { template, use } from "solid-js/web";
import { useRouter, useRouterState } from "@tanstack/solid-router";
import { TanStackRouterDevtoolsCore } from "@tanstack/router-devtools-core";
import { createEffect, createSignal, onCleanup, onMount } from "solid-js";
//#region src/TanStackRouterDevtools.tsx
var _tmpl$ = /* @__PURE__ */ template(`<div>`);
var TanStackRouterDevtools = (props) => {
	const activeRouter = props.router ?? useRouter();
	const activeRouterState = useRouterState({ router: activeRouter });
	const usedProps = {
		...props,
		router: activeRouter,
		routerState: activeRouterState
	};
	let devToolRef;
	const [devtools] = createSignal(new TanStackRouterDevtoolsCore(usedProps));
	createEffect(() => {
		devtools().setRouter(usedProps.router);
	});
	createEffect(() => {
		devtools().setRouterState(usedProps.routerState);
	});
	createEffect(() => {
		devtools().setOptions({
			initialIsOpen: usedProps.initialIsOpen,
			panelProps: usedProps.panelProps,
			closeButtonProps: usedProps.closeButtonProps,
			toggleButtonProps: usedProps.toggleButtonProps,
			position: usedProps.position,
			containerElement: usedProps.containerElement,
			shadowDOMTarget: usedProps.shadowDOMTarget
		});
	});
	onMount(() => {
		if (devToolRef) {
			devtools().mount(devToolRef);
			onCleanup(() => {
				devtools().unmount();
			});
		}
	});
	return (() => {
		var _el$ = _tmpl$();
		var _ref$ = devToolRef;
		typeof _ref$ === "function" ? use(_ref$, _el$) : devToolRef = _el$;
		return _el$;
	})();
};
//#endregion
export { TanStackRouterDevtools };
