// @tanstack/solid-router/dist/cjs/headContentUtils.cjs
const require_runtime = require("./_virtual/_rolldown/runtime.cjs");
const require_useRouter = require("./useRouter.cjs");
let _tanstack_router_core = require("@tanstack/router-core");
let solid_js = require("solid-js");
solid_js = require_runtime.__toESM(solid_js);
//#region src/headContentUtils.tsx
/**
* Build the list of head/link/meta/script tags to render for active matches.
* Used internally by `HeadContent`.
*/
var useTags = (assetCrossOrigin) => {
	const router = require_useRouter.useRouter();
	const nonce = router.options.ssr?.nonce;
	const activeMatches = solid_js.createMemo(() => router.stores.matches.get());
	const routeMeta = solid_js.createMemo(() => activeMatches().map((match) => match.meta).filter(Boolean));
	const meta = solid_js.createMemo(() => {
		const resultMeta = [];
		const metaByAttribute = {};
		let title;
		const routeMetasArray = routeMeta();
		for (let i = routeMetasArray.length - 1; i >= 0; i--) {
			const metas = routeMetasArray[i];
			for (let j = metas.length - 1; j >= 0; j--) {
				const m = metas[j];
				if (!m) continue;
				if (m.title) {
					if (!title) title = {
						tag: "title",
						children: m.title
					};
				} else if ("script:ld+json" in m) try {
					const json = JSON.stringify(m["script:ld+json"]);
					resultMeta.push({
						tag: "script",
						attrs: { type: "application/ld+json" },
						children: (0, _tanstack_router_core.escapeHtml)(json)
					});
				} catch {}
				else {
					const attribute = m.name ?? m.property;
					if (attribute) if (metaByAttribute[attribute]) continue;
					else metaByAttribute[attribute] = true;
					resultMeta.push({
						tag: "meta",
						attrs: {
							...m,
							nonce
						}
					});
				}
			}
		}
		if (title) resultMeta.push(title);
		if (router.options.ssr?.nonce) resultMeta.push({
			tag: "meta",
			attrs: {
				property: "csp-nonce",
				content: router.options.ssr.nonce
			}
		});
		resultMeta.reverse();
		return resultMeta;
	});
	const links = solid_js.createMemo(() => {
		const matches = activeMatches();
		const constructed = matches.map((match) => match.links).filter(Boolean).flat(1).map((link) => ({
			tag: "link",
			attrs: {
				...link,
				nonce
			}
		}));
		const manifest = router.ssr?.manifest;
		const assets = matches.map((match) => manifest?.routes[match.routeId]?.assets ?? []).filter(Boolean).flat(1).flatMap((asset) => {
			if (asset.tag === "link") {
				if ((0, _tanstack_router_core.isInlinableStylesheet)(manifest, asset)) return [];
				return [{
					tag: "link",
					attrs: {
						...asset.attrs,
						crossOrigin: (0, _tanstack_router_core.getAssetCrossOrigin)(assetCrossOrigin, "stylesheet") ?? asset.attrs?.crossOrigin,
						nonce
					}
				}];
			}
			if (asset.tag === "style") return [{
				tag: "style",
				attrs: {
					...asset.attrs,
					nonce
				},
				children: asset.children,
				...asset.inlineCss ? { inlineCss: true } : {}
			}];
			return [];
		});
		return [...constructed, ...assets];
	});
	const preloadLinks = solid_js.createMemo(() => {
		const matches = activeMatches();
		const preloadLinks = [];
		matches.map((match) => router.looseRoutesById[match.routeId]).forEach((route) => router.ssr?.manifest?.routes[route.id]?.preloads?.filter(Boolean).forEach((preload) => {
			const preloadLink = (0, _tanstack_router_core.resolveManifestAssetLink)(preload);
			preloadLinks.push({
				tag: "link",
				attrs: {
					rel: "modulepreload",
					href: preloadLink.href,
					crossOrigin: (0, _tanstack_router_core.getAssetCrossOrigin)(assetCrossOrigin, "modulepreload") ?? preloadLink.crossOrigin,
					nonce
				}
			});
		}));
		return preloadLinks;
	});
	const styles = solid_js.createMemo(() => activeMatches().map((match) => match.styles).flat(1).filter(Boolean).map(({ children, ...style }) => ({
		tag: "style",
		attrs: {
			...style,
			nonce
		},
		children
	})));
	const headScripts = solid_js.createMemo(() => activeMatches().map((match) => match.headScripts).flat(1).filter(Boolean).map(({ children, ...script }) => ({
		tag: "script",
		attrs: {
			...script,
			nonce
		},
		children
	})));
	return solid_js.createMemo((prev) => {
		const next = uniqBy([
			...meta(),
			...preloadLinks(),
			...links(),
			...styles(),
			...headScripts()
		], (d) => {
			return JSON.stringify(d);
		});
		if (prev === void 0) return next;
		return replaceEqualTags(prev, next);
	});
};
function replaceEqualTags(prev, next) {
	const prevByKey = /* @__PURE__ */ new Map();
	for (const tag of prev) prevByKey.set(JSON.stringify(tag), tag);
	let isEqual = prev.length === next.length;
	const result = next.map((tag, index) => {
		const existing = prevByKey.get(JSON.stringify(tag));
		if (existing) {
			if (existing !== prev[index]) isEqual = false;
			return existing;
		}
		isEqual = false;
		return tag;
	});
	return isEqual ? prev : result;
}
function uniqBy(arr, fn) {
	const seen = /* @__PURE__ */ new Set();
	return arr.filter((item) => {
		const key = fn(item);
		if (seen.has(key)) return false;
		seen.add(key);
		return true;
	});
}
//#endregion
exports.useTags = useTags;
