// @tanstack/solid-router/dist/cjs/routerStores.cjs
const require_runtime = require("./_virtual/_rolldown/runtime.cjs");
let _tanstack_router_core = require("@tanstack/router-core");
let solid_js = require("solid-js");
solid_js = require_runtime.__toESM(solid_js);
let _tanstack_router_core_isServer = require("@tanstack/router-core/isServer");
//#region src/routerStores.ts
function initRouterStores(stores, createReadonlyStore) {
	stores.childMatchIdByRouteId = createReadonlyStore(() => {
		const ids = stores.matchesId.get();
		const obj = {};
		for (let i = 0; i < ids.length - 1; i++) {
			const parentStore = stores.matchStores.get(ids[i]);
			if (parentStore?.routeId) obj[parentStore.routeId] = ids[i + 1];
		}
		return obj;
	});
	stores.pendingRouteIds = createReadonlyStore(() => {
		const ids = stores.pendingIds.get();
		const obj = {};
		for (const id of ids) {
			const store = stores.pendingMatchStores.get(id);
			if (store?.routeId) obj[store.routeId] = true;
		}
		return obj;
	});
}
function createSolidMutableStore(initialValue) {
	const [signal, setSignal] = solid_js.createSignal(initialValue);
	return {
		get: signal,
		set: setSignal
	};
}
var finalizationRegistry = null;
if (typeof globalThis !== "undefined" && "FinalizationRegistry" in globalThis) finalizationRegistry = new FinalizationRegistry((cb) => cb());
function createSolidReadonlyStore(read) {
	let dispose;
	const store = { get: solid_js.createRoot((d) => {
		dispose = d;
		return solid_js.createMemo(read);
	}) };
	finalizationRegistry?.register(store, dispose);
	return store;
}
var getStoreFactory = (opts) => {
	if (_tanstack_router_core_isServer.isServer ?? opts.isServer) return {
		createMutableStore: _tanstack_router_core.createNonReactiveMutableStore,
		createReadonlyStore: _tanstack_router_core.createNonReactiveReadonlyStore,
		batch: (fn) => fn(),
		init: (stores) => initRouterStores(stores, _tanstack_router_core.createNonReactiveReadonlyStore)
	};
	return {
		createMutableStore: createSolidMutableStore,
		createReadonlyStore: createSolidReadonlyStore,
		batch: solid_js.batch,
		init: (stores) => initRouterStores(stores, createSolidReadonlyStore)
	};
};
//#endregion
exports.getStoreFactory = getStoreFactory;
