// @tanstack/solid-router/dist/cjs/route.cjs
require("./_virtual/_rolldown/runtime.cjs");
const require_useRouter = require("./useRouter.cjs");
const require_link = require("./link.cjs");
const require_useMatch = require("./useMatch.cjs");
const require_useLoaderData = require("./useLoaderData.cjs");
const require_useLoaderDeps = require("./useLoaderDeps.cjs");
const require_useParams = require("./useParams.cjs");
const require_useSearch = require("./useSearch.cjs");
const require_useNavigate = require("./useNavigate.cjs");
const require_useRouteContext = require("./useRouteContext.cjs");
let _tanstack_router_core = require("@tanstack/router-core");
let solid_js_web = require("solid-js/web");
//#region src/route.tsx
function getRouteApi(id) {
	return new RouteApi({ id });
}
var RouteApi = class extends _tanstack_router_core.BaseRouteApi {
	/**
	* @deprecated Use the `getRouteApi` function instead.
	*/
	constructor({ id }) {
		super({ id });
		this.useMatch = (opts) => {
			return require_useMatch.useMatch({
				select: opts?.select,
				from: this.id
			});
		};
		this.useRouteContext = (opts) => {
			return require_useRouteContext.useRouteContext({
				...opts,
				from: this.id
			});
		};
		this.useSearch = (opts) => {
			return require_useSearch.useSearch({
				select: opts?.select,
				from: this.id
			});
		};
		this.useParams = (opts) => {
			return require_useParams.useParams({
				select: opts?.select,
				from: this.id
			});
		};
		this.useLoaderDeps = (opts) => {
			return require_useLoaderDeps.useLoaderDeps({
				...opts,
				from: this.id,
				strict: false
			});
		};
		this.useLoaderData = (opts) => {
			return require_useLoaderData.useLoaderData({
				...opts,
				from: this.id,
				strict: false
			});
		};
		this.useNavigate = () => {
			return require_useNavigate.useNavigate({ from: require_useRouter.useRouter().routesById[this.id].fullPath });
		};
		this.notFound = (opts) => {
			return (0, _tanstack_router_core.notFound)({
				routeId: this.id,
				...opts
			});
		};
		this.Link = ((props) => {
			const fullPath = require_useRouter.useRouter().routesById[this.id].fullPath;
			return (0, solid_js_web.createComponent)(require_link.Link, (0, solid_js_web.mergeProps)({ from: fullPath }, props));
		});
	}
};
var Route = class extends _tanstack_router_core.BaseRoute {
	/**
	* @deprecated Use the `createRoute` function instead.
	*/
	constructor(options) {
		super(options);
		this.useMatch = (opts) => {
			return require_useMatch.useMatch({
				select: opts?.select,
				from: this.id
			});
		};
		this.useRouteContext = (opts) => {
			return require_useRouteContext.useRouteContext({
				...opts,
				from: this.id
			});
		};
		this.useSearch = (opts) => {
			return require_useSearch.useSearch({
				select: opts?.select,
				from: this.id
			});
		};
		this.useParams = (opts) => {
			return require_useParams.useParams({
				select: opts?.select,
				from: this.id
			});
		};
		this.useLoaderDeps = (opts) => {
			return require_useLoaderDeps.useLoaderDeps({
				...opts,
				from: this.id
			});
		};
		this.useLoaderData = (opts) => {
			return require_useLoaderData.useLoaderData({
				...opts,
				from: this.id
			});
		};
		this.useNavigate = () => {
			return require_useNavigate.useNavigate({ from: this.fullPath });
		};
		this.Link = ((props) => {
			const _self$ = this;
			return (0, solid_js_web.createComponent)(require_link.Link, (0, solid_js_web.mergeProps)({ get from() {
				return _self$.fullPath;
			} }, props));
		});
	}
};
function createRoute(options) {
	return new Route(options);
}
function createRootRouteWithContext() {
	return (options) => {
		return createRootRoute(options);
	};
}
/**
* @deprecated Use the `createRootRouteWithContext` function instead.
*/
var rootRouteWithContext = createRootRouteWithContext;
var RootRoute = class extends _tanstack_router_core.BaseRootRoute {
	/**
	* @deprecated `RootRoute` is now an internal implementation detail. Use `createRootRoute()` instead.
	*/
	constructor(options) {
		super(options);
		this.useMatch = (opts) => {
			return require_useMatch.useMatch({
				select: opts?.select,
				from: this.id
			});
		};
		this.useRouteContext = (opts) => {
			return require_useRouteContext.useRouteContext({
				...opts,
				from: this.id
			});
		};
		this.useSearch = (opts) => {
			return require_useSearch.useSearch({
				select: opts?.select,
				from: this.id
			});
		};
		this.useParams = (opts) => {
			return require_useParams.useParams({
				select: opts?.select,
				from: this.id
			});
		};
		this.useLoaderDeps = (opts) => {
			return require_useLoaderDeps.useLoaderDeps({
				...opts,
				from: this.id
			});
		};
		this.useLoaderData = (opts) => {
			return require_useLoaderData.useLoaderData({
				...opts,
				from: this.id
			});
		};
		this.useNavigate = () => {
			return require_useNavigate.useNavigate({ from: this.fullPath });
		};
		this.Link = ((props) => {
			const _self$2 = this;
			return (0, solid_js_web.createComponent)(require_link.Link, (0, solid_js_web.mergeProps)({ get from() {
				return _self$2.fullPath;
			} }, props));
		});
	}
};
function createRouteMask(opts) {
	return opts;
}
var NotFoundRoute = class extends Route {
	constructor(options) {
		super({
			...options,
			id: "404"
		});
	}
};
function createRootRoute(options) {
	return new RootRoute(options);
}
//#endregion
exports.NotFoundRoute = NotFoundRoute;
exports.RootRoute = RootRoute;
exports.Route = Route;
exports.RouteApi = RouteApi;
exports.createRootRoute = createRootRoute;
exports.createRootRouteWithContext = createRootRouteWithContext;
exports.createRoute = createRoute;
exports.createRouteMask = createRouteMask;
exports.getRouteApi = getRouteApi;
exports.rootRouteWithContext = rootRouteWithContext;
